{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "2261787c",
   "metadata": {},
   "source": [
    "## Cleanup citation files\n",
    "Trim the various bib files."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "bcfba470",
   "metadata": {},
   "outputs": [],
   "source": [
    "bibfilepath = \"../ms_files/umap_popgen_review.bib\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "6c5dbe59",
   "metadata": {},
   "outputs": [],
   "source": [
    "file = open(bibfilepath)\n",
    "\n",
    "lines = file.readlines()\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "337e49cf",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['\\n', '@article{wei_graph_2019,\\n', '\\ttitle = {Graph {Embedding} {Method} {Based} genetical trajectory {Reveals} {Migration} {History} {Among} {East} {Asians}},\\n', '\\tcopyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/870253v1},\\n', '\\tdoi = {10.1101/870253},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}ABSTRACT{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}An important issue in human population genetics is the ancestry. By extracting the ancestral information retained in the single nucleotide polymorphism (SNP) of genomic DNA, the history of migration and reproduction of the population can be reconstructed. Since the SNP data of population are multidimensional, their dimensionality reduction can demonstrate their potential internal connections. In this study, the graph and structure learning based Graph Embedding method commonly used in single cell mRNA sequencing was applied to human population genetics research to decrease the data dimension. As a result, the human population trajectory of East Asia based on 1000 Genomes Project was reconstructed to discover the inseparable relationship between the Chinese population and other East Asian populations. These results are visualized from various ancestry calculators such as E11 and K12B. Finally, the unique SNPs along the psudotime of trajectory were found by differential analysis. Bioprocess enrichment analysis was also used to reveal that the genes of these SNPs may be related to neurological diseases. These results will lay the data foundation for precision medicine.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Wei, Zhuang and Chang, Ching-Wen and Luo, Van and Bian, Beilei and Ding, Xuewei},\\n', '\\tmonth = dec,\\n', '\\tyear = {2019},\\n', '\\tpages = {870253}\\n', '}\\n', '\\n', '@article{simon_local_2019,\\n', '\\ttitle = {Local introgression at two spatial scales in mosaic hybrid zones of mussels},\\n', '\\tcopyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/818559v1},\\n', '\\tdoi = {10.1101/818559},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}When the ranges of closely-related lineages are large, and overlapping, we can often study introgression at many “replicated” contacts, with different locations and spatial scales. Here we analysed multiple contact zones of the \\\\textit{M. edulis} complex of marine mussel species, which represent a mosaic distribution of heterogeneously differentiated, semi-isolated genomes. Our aim was to contrast ongoing introgression at the heart of hybrid zones, with past introgression between similar parental populations, at increasing distance from the contact. Using a panel of ancestry-informative SNPs derived from a previous genomic study, we first confirm, with a broader sampling, that local introgression, affecting one but not all of the populations compared, is both widespread and heterogeneous across the genome. Some outlier loci show patterns of complete introgression in certain populations, and an absence of introgression in others. Genomic cline analyses reveal a globally high concordance among loci at a local scale, albeit with signals of asymmetric introgression at a few loci. Enhanced local introgression at specific loci is consistent with the early transfer of adaptive variants after contact, possibly including asymmetric bi-stable variants, or less loaded alleles. Given the mosaic structure of the \\\\textit{M. edulis} complex, with a succession of genetic barriers to gene flow, variants with enhanced introgression through one barrier can be trapped, maybe transiently, at the next barrier, confining introgression locally. This makes the \\\\textit{Mytilus} complex an ideal model of the heterogeneous porosity of species barriers.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Simon, Alexis and Fraïsse, Christelle and Ayari, Tahani El and Liautard-Haag, Cathy and Strelkov, Petr and Welch, John J. and Bierne, Nicolas},\\n', '\\tmonth = oct,\\n', '\\tyear = {2019},\\n', '\\tpages = {818559}\\n', '}\\n', '\\n', '@article{dorrity_dimensionality_2020,\\n', '\\ttitle = {Dimensionality reduction by {UMAP} to visualize physical and genetic interactions},\\n', '\\tvolume = {11},\\n', '\\tcopyright = {2020 The Author(s)},\\n', '\\tissn = {2041-1723},\\n', '\\turl = {https://www.nature.com/articles/s41467-020-15351-4},\\n', '\\tdoi = {10.1038/s41467-020-15351-4},\\n', '\\tabstract = {Dimensionality reduction is often used to visualize complex expression profiling data. Here, we use the Uniform Manifold Approximation and Projection (UMAP) method on published transcript profiles of 1484 single gene deletions of Saccharomyces cerevisiae. Proximity in low-dimensional UMAP space identifies groups of genes that correspond to protein complexes and pathways, and finds novel protein interactions, even within well-characterized complexes. This approach is more sensitive than previous methods and should be broadly useful as additional transcriptome datasets become available for other organisms.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {1},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Nature Communications},\\n', '\\tauthor = {Dorrity, Michael W. and Saunders, Lauren M. and Queitsch, Christine and Fields, Stanley and Trapnell, Cole},\\n', '\\tmonth = mar,\\n', '\\tyear = {2020},\\n', '\\tpages = {1537}\\n', '}\\n', '\\n', '@article{yelmen_creating_2019,\\n', '\\ttitle = {Creating {Artificial} {Human} {Genomes} {Using} {Generative} {Models}},\\n', '\\tcopyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/769091v2},\\n', '\\tdoi = {10.1101/769091},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}Generative models have shown breakthroughs in a wide spectrum of domains due to recent advancements in machine learning algorithms and increased computational power. Despite these impressive achievements, the ability of generative models to create realistic synthetic data is still under-exploited in genetics and absent from population genetics.{\\\\textless}/p{\\\\textgreater}{\\\\textless}p{\\\\textgreater}Yet a known limitation of this field is the reduced access to many genetic databases due to concerns about violations of individual privacy, although they would provide a rich resource for data mining and integration towards advancing genetic studies. In this study, we demonstrated that deep generative adversarial networks (GANs) and restricted Boltzmann machines (RBMs) can be trained to learn the high dimensional distributions of real genomic datasets and create high quality artificial genomes (AGs) with none to little privacy loss. To illustrate the promising outcomes of our method, we showed that (i) imputation quality for low frequency alleles can be improved by augmenting reference panels with AGs, (ii) scores obtained from selection tests on AGs and real genomes are highly correlated and (iii) AGs can inherit genotype-phenotype associations. AGs have the potential to become valuable assets in genetic studies by providing high quality anonymous substitutes for private databases.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Yelmen, Burak and Decelle, Aurélien and Ongaro, Linda and Marnetto, Davide and Tallec, Corentin and Montinaro, Francesco and Furtlehner, Cyril and Pagani, Luca and Jay, Flora},\\n', '\\tmonth = oct,\\n', '\\tyear = {2019},\\n', '\\tpages = {769091}\\n', '}\\n', '\\n', '@article{schmidt_population_2020,\\n', '\\ttitle = {Population genomics of two invasive mosquitoes ({Aedes} aegypti and {Aedes} albopictus) from the {Indo}-{Pacific}},\\n', '\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/2020.03.15.993055v3},\\n', '\\tdoi = {10.1101/2020.03.15.993055},\\n', '\\tabstract = {{\\\\textless}p{\\\\textgreater}The arbovirus vectors Aedes aegypti (yellow fever mosquito) and Ae. albopictus (Asian tiger mosquito) are both common throughout the Indo-Pacific region, where 70\\\\% of global dengue transmission occurs. For Ae. aegypti all Indo-Pacific populations are invasive, having spread from an initial native range of Africa, while for Ae. albopictus the Indo-Pacific includes invasive populations and those from the native range: putatively, India to Japan to Southeast Asia. This study analyses the population genomics of 480 of these mosquitoes sampled from 27 locations in the Indo-Pacific. We investigated patterns of genome-wide genetic differentiation to compare pathways of invasion and ongoing gene flow in both species, and to compare invasive and native-range populations of Ae. albopictus. We also tested landscape genomic hypotheses that genetic differentiation would increase with geographical distance and be lower between locations with high connectivity to human transportation routes, the primary means of dispersal at these scales. We found that genetic distances were generally higher in Ae. aegypti, with Pacific populations the most highly differentiated. The most differentiated Ae. albopictus populations were in Vanuatu, Indonesia and Sri Lanka, the latter two representing potential native-range populations and potential cryptic subspeciation respectively. Genetic distances in Ae. aegypti increased with geographical distance, while in Ae. albopictus they decreased with higher connectivity to human transportation routes. Contrary to the situation in Ae. aegypti, we found evidence of long-distance Ae. albopictus colonisation events, including colonisation of Mauritius from East Asia and of Fiji from Southeast Asia. These direct genomic comparisons indicate likely differences in dispersal ecology in these species, despite their broadly sympatric distributions and similar use of human transport to disperse. Our findings will assist biosecurity operations to trace the source of invasive material and for biocontrol operations that benefit from matching genetic backgrounds of released and local populations.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Schmidt, Thomas L. and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R. and Hoffmann, Ary A.},\\n', '\\tmonth = may,\\n', '\\tyear = {2020},\\n', '\\tpages = {2020.03.15.993055}\\n', '}\\n', '\\n', '@article{schmidt2020population,\\n', '  title={Population genomics of two invasive mosquitoes (Aedes aegypti and Aedes albopictus) from the Indo-Pacific},\\n', '  author={Schmidt, Thomas L and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R and Hoffmann, Ary A},\\n', '  journal={bioRxiv},\\n', '  year={2020},\\n', '  publisher={Cold Spring Harbor Laboratory}\\n', '}\\n', '\\n', '@article{zhang_generalized_2020,\\n', '\\ttitle = {A {Generalized} {Robust} {Allele}-based {Genetic} {Association} {Test}},\\n', \"\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission.},\\n\", '\\turl = {https://www.biorxiv.org/content/10.1101/2020.03.12.989004v1},\\n', '\\tdoi = {10.1101/2020.03.12.989004},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}The allele-based association test or the allelic test, comparing allele frequency difference between case and control groups, is locally most powerful. However, the classical allelic test is limited in applications because it is sensitive to the Hardy–Weinberg equilibrium (HWE) assumption, not applicable to continuous traits, and not easy to account for covariate effects or sample correlation. To develop a generalized robust allelic test, we propose a unifying regression model with individual allele as the response variable. We show that the score test statistic derived from this novel regression framework contains a correction factor that explicitly adjusts for the departure from HWE and encompasses the classical allelic test as a special case. When the trait of interest is continuous, the corresponding allelic test evaluates a weighted difference between individual-level allele frequency estimate and sample estimate where the weight is proportional to an individual’s trait value, and the test remains valid under Y-dependent sampling. Finally, the proposed method allows for joint allele-based association analyses of multiple (continuous or binary) phenotypes, in the presence of covariates, sample correlation and population heterogeneity. To support our analytical findings, we provide empirical evidence from both simulation and application studies.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Zhang, Lin and Sun, Lei},\\n', '\\tmonth = mar,\\n', '\\tyear = {2020},\\n', '\\tpages = {2020.03.12.989004}\\n', '}\\n', '\\n', '@article{mcinnes_umap_2018,\\n', '\\ttitle = {{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection} for {Dimension} {Reduction}},\\n', '\\tshorttitle = {{UMAP}},\\n', '\\turl = {http://arxiv.org/abs/1802.03426},\\n', '\\tabstract = {UMAP (Uniform Manifold Approximation and Projection) is a novel manifold learning technique for dimension reduction. UMAP is constructed from a theoretical framework based in Riemannian geometry and algebraic topology. The result is a practical scalable algorithm that applies to real world data. The UMAP algorithm is competitive with t-SNE for visualization quality, and arguably preserves more of the global structure with superior run time performance. Furthermore, UMAP has no computational restrictions on embedding dimension, making it viable as a general purpose dimension reduction technique for machine learning.},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {arXiv},\\n', '\\tauthor = {McInnes, Leland and Healy, John and Melville, James},\\n', '\\tmonth = dec,\\n', '\\tyear = {2018}\\n', '}\\n', '\\n', '@article{sakaue_dimensionality_2020,\\n', '\\ttitle = {Dimensionality reduction reveals fine-scale structure in the {Japanese} population with consequences for polygenic risk prediction},\\n', '\\tvolume = {11},\\n', '\\tcopyright = {2020 The Author(s)},\\n', '\\tissn = {2041-1723},\\n', '\\turl = {https://www.nature.com/articles/s41467-020-15194-z},\\n', '\\tdoi = {10.1038/s41467-020-15194-z},\\n', '\\tabstract = {The diversity in our genome is crucial to understanding the demographic history of worldwide populations. However, we have yet to know whether subtle genetic differences within a population can be disentangled, or whether they have an impact on complex traits. Here we apply dimensionality reduction methods (PCA, t-SNE, PCA-t-SNE, UMAP, and PCA-UMAP) to biobank-derived genomic data of a Japanese population (n\\u2009=\\u2009169,719). Dimensionality reduction reveals fine-scale population structure, conspicuously differentiating adjacent insular subpopulations. We further enluciate the demographic landscape of these Japanese subpopulations using population genetics analyses. Finally, we perform phenome-wide polygenic risk score (PRS) analyses on 67 complex traits. Differences in PRS between the deconvoluted subpopulations are not always concordant with those in the observed phenotypes, suggesting that the PRS differences might reflect biases from the uncorrected structure, in a trait-dependent manner. This study suggests that such an uncorrected structure can be a potential pitfall in the clinical application of PRS.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {1},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Nature Communications},\\n', '\\tauthor = {Sakaue, Saori and Hirata, Jun and Kanai, Masahiro and Suzuki, Ken and Akiyama, Masato and Lai Too, Chun and Arayssi, Thurayya and Hammoudeh, Mohammed and Al Emadi, Samar and Masri, Basel K. and Halabi, Hussein and Badsha, Humeira and Uthman, Imad W. and Saxena, Richa and Padyukov, Leonid and Hirata, Makoto and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Okada, Yukinori},\\n', '\\tmonth = mar,\\n', '\\tyear = {2020},\\n', '\\tpages = {1569}\\n', '}\\n', '\\n', '@article{karczewski_mutational_2020,\\n', '\\ttitle = {The mutational constraint spectrum quantified from variation in 141,456 humans},\\n', '\\tvolume = {581},\\n', '\\tcopyright = {2020 The Author(s)},\\n', '\\tissn = {1476-4687},\\n', '\\turl = {https://www.nature.com/articles/s41586-020-2308-7},\\n', '\\tdoi = {10.1038/s41586-020-2308-7},\\n', '\\tabstract = {Genetic variants that inactivate protein-coding genes are a powerful source of information about the phenotypic consequences of gene disruption: genes that are crucial for the function of an organism will be depleted of such variants in natural populations, whereas non-essential genes will tolerate their accumulation. However, predicted loss-of-function variants are enriched for annotation errors, and tend to be found at extremely low frequencies, so their analysis requires careful variant annotation and very large sample sizes1. Here we describe the aggregation of 125,748 exomes and 15,708 genomes from human sequencing studies into the Genome Aggregation Database (gnomAD). We identify 443,769 high-confidence predicted loss-of-function variants in this cohort after filtering for artefacts caused by sequencing and annotation errors. Using an improved model of human mutation rates, we classify human protein-coding genes along a spectrum that represents tolerance to inactivation, validate this classification using data from model organisms and engineered human cells, and show that it can be used to improve the power of gene discovery for both common and rare diseases.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {7809},\\n', '\\turldate = {2020-06-03},\\n', '\\tjournal = {Nature},\\n', '\\tauthor = {Karczewski, Konrad J. and Francioli, Laurent C. and Tiao, Grace and Cummings, Beryl B. and Alföldi, Jessica and Wang, Qingbo and Collins, Ryan L. and Laricchia, Kristen M. and Ganna, Andrea and Birnbaum, Daniel P. and Gauthier, Laura D. and Brand, Harrison and Solomonson, Matthew and Watts, Nicholas A. and Rhodes, Daniel and Singer-Berk, Moriel and England, Eleina M. and Seaby, Eleanor G. and Kosmicki, Jack A. and Walters, Raymond K. and Tashman, Katherine and Farjoun, Yossi and Banks, Eric and Poterba, Timothy and Wang, Arcturus and Seed, Cotton and Whiffin, Nicola and Chong, Jessica X. and Samocha, Kaitlin E. and Pierce-Hoffman, Emma and Zappala, Zachary and O’Donnell-Luria, Anne H. and Minikel, Eric Vallabh and Weisburd, Ben and Lek, Monkol and Ware, James S. and Vittal, Christopher and Armean, Irina M. and Bergelson, Louis and Cibulskis, Kristian and Connolly, Kristen M. and Covarrubias, Miguel and Donnelly, Stacey and Ferriera, Steven and Gabriel, Stacey and Gentry, Jeff and Gupta, Namrata and Jeandet, Thibault and Kaplan, Diane and Llanwarne, Christopher and Munshi, Ruchi and Novod, Sam and Petrillo, Nikelle and Roazen, David and Ruano-Rubio, Valentin and Saltzman, Andrea and Schleicher, Molly and Soto, Jose and Tibbetts, Kathleen and Tolonen, Charlotte and Wade, Gordon and Talkowski, Michael E. and Neale, Benjamin M. and Daly, Mark J. and MacArthur, Daniel G.},\\n', '\\tmonth = may,\\n', '\\tyear = {2020},\\n', '\\tpages = {434--443}\\n', '}\\n', '\\n', '@article{tonkin-hill_fast_2019,\\n', '\\ttitle = {Fast hierarchical {Bayesian} analysis of population structure},\\n', '\\tvolume = {47},\\n', '\\tissn = {0305-1048},\\n', '\\turl = {https://academic.oup.com/nar/article/47/11/5539/5488018},\\n', '\\tdoi = {10.1093/nar/gkz361},\\n', '\\tabstract = {Abstract.  We present fastbaps, a fast solution to the genetic clustering problem. Fastbaps rapidly identifies an approximate fit to a Dirichlet process mixture},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {11},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Nucleic Acids Research},\\n', '\\tauthor = {Tonkin-Hill, Gerry and Lees, John A. and Bentley, Stephen D. and Frost, Simon D. W. and Corander, Jukka},\\n', '\\tmonth = jun,\\n', '\\tyear = {2019},\\n', '\\tpages = {5539--5549}\\n', '}\\n', '\\n', '@article{hunter-zinck_genotyping_2020,\\n', '\\ttitle = {Genotyping {Array} {Design} and {Data} {Quality} {Control} in the {Million} {Veteran} {Program}},\\n', '\\tvolume = {106},\\n', '\\tissn = {0002-9297},\\n', '\\turl = {http://www.sciencedirect.com/science/article/pii/S000292972030080X},\\n', '\\tdoi = {10.1016/j.ajhg.2020.03.004},\\n', '\\tabstract = {The Million Veteran Program (MVP), initiated by the Department of Veterans Affairs (VA), aims to collect biosamples with consent from at least one million veterans. Presently, blood samples have been collected from over 800,000 enrolled participants. The size and diversity of the MVP cohort, as well as the availability of extensive VA electronic health records, make it a promising resource for precision medicine. MVP is conducting array-based genotyping to provide a genome-wide scan of the entire cohort, in parallel with whole-genome sequencing, methylation, and other ‘omics assays. Here, we present the design and performance of the MVP 1.0 custom Axiom array, which was designed and developed as a single assay to be used across the multi-ethnic MVP cohort. A unified genetic quality-control analysis was developed and conducted on an initial tranche of 485,856 individuals, leading to a high-quality dataset of 459,777 unique individuals. 668,418 genetic markers passed quality control and showed high-quality genotypes not only on common variants but also on rare variants. We confirmed that, with non-European individuals making up nearly 30\\\\%, MVP’s substantial ancestral diversity surpasses that of other large biobanks. We also demonstrated the quality of the MVP dataset by replicating established genetic associations with height in European Americans and African Americans ancestries. This current dataset has been made available to approved MVP researchers for genome-wide association studies and other downstream analyses. Further data releases will be available for analysis as recruitment at the VA continues and the cohort expands both in size and diversity.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {4},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {The American Journal of Human Genetics},\\n', '\\tauthor = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Gong, Xin and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju},\\n', '\\tmonth = apr,\\n', '\\tyear = {2020},\\n', '\\tkeywords = {GWAS, biobank, genetic ancestry, clinical variants, genetic relatedness, genotype data, Million Veteran Program, quality control, SNP array design, VA},\\n', '\\tpages = {535--548}\\n', '}\\n', '\\n', '@article{hunter-zinck_measuring_2020,\\n', '\\ttitle = {Measuring genetic variation in the multi-ethnic {Million} {Veteran} {Program} ({MVP})},\\n', '\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/2020.01.06.896613v1},\\n', '\\tdoi = {10.1101/2020.01.06.896613},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}The Million Veteran Program (MVP), initiated by the Department of Veterans Affairs (VA), aims to collect consented biosamples from at least one million Veterans. Presently, blood samples have been collected from over 800,000 enrolled participants. The size and diversity of the MVP cohort, as well as the availability of extensive VA electronic health records make it a promising resource for precision medicine. MVP is conducting array-based genotyping to provide genome-wide scan of the entire cohort, in parallel with whole genome sequencing, methylation, and other omics assays. Here, we present the design and performance of MVP 1.0 custom Axiom$^{\\\\textrm{®}}$ array, which was designed and developed as a single assay to be used across the multi-ethnic MVP cohort. A unified genetic quality control analysis was developed and conducted on an initial tranche of 485,856 individuals leading to a high-quality dataset of 459,777 unique individuals. 668,418 genetic markers passed quality control and showed high quality genotypes not only on common variants but also on rare variants. We confirmed the substantial ancestral diversity of MVP with nearly 30\\\\% non-European individuals, surpassing other large biobanks. We also demonstrated the quality of the MVP dataset by replicating established genetic associations with height in European Americans and African Americans ancestries. This current data set has been made available to approved MVP researchers for genome-wide association studies and other downstream analyses. Further data releases will be available for analysis as recruitment at the VA continues and the cohort expands both in size and diversity.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju and Program, on behalf of the VA Million Veteran},\\n', '\\tmonth = jan,\\n', '\\tyear = {2020},\\n', '\\tpages = {2020.01.06.896613}\\n', '}\\n', '\\n', '@article{dai_population_2020,\\n', '\\ttitle = {Population {Histories} of the {United} {States} {Revealed} through {Fine}-{Scale} {Migration} and {Haplotype} {Analysis}},\\n', '\\tvolume = {106},\\n', '\\tissn = {00029297},\\n', '\\turl = {https://linkinghub.elsevier.com/retrieve/pii/S0002929720300446},\\n', '\\tdoi = {10.1016/j.ajhg.2020.02.002},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {3},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {The American Journal of Human Genetics},\\n', '\\tauthor = {Dai, Chengzhen L. and Vazifeh, Mohammad M. and Yeang, Chen-Hsiang and Tachet, Remi and Wells, R. Spencer and Vilar, Miguel G. and Daly, Mark J. and Ratti, Carlo and Martin, Alicia R.},\\n', '\\tmonth = Mar,\\n', '\\tyear = {2020},\\n', '\\tpages = {371--388}\\n', '}\\n', '\\n', '@article{sanchez-barreiro_historical_2020,\\n', '\\ttitle = {Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros ({Ceratotherium} simum)},\\n', '\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/2020.05.10.086686v1},\\n', '\\tdoi = {10.1101/2020.05.10.086686},\\n', '\\tabstract = {{\\\\textless}p{\\\\textgreater}Large vertebrates are extremely sensitive to anthropogenic pressure, and their populations are declining fast. The white rhinoceros (Ceratotherium simum) is a paradigmatic case: this African megaherbivore suffered a remarkable population reduction in the last 150 years due to human activities. The two white rhinoceros subspecies, the northern (NWR) and the southern white rhinoceros (SWR), however, underwent opposite fates: the NWR vanished quickly after the onset of the decline, while the SWR recovered after a severe bottleneck. Such demographic events are predicted to have an erosive effect at the genomic level, in connection with the extirpation of diversity, and increased genetic drift and inbreeding. However there is currently little empirical data available that allows us to directly reconstruct the subtleties of such processes in light of distinct demographic histories. Therefore to assess these effects, we generated a whole-genome, temporal dataset consisting of 52 re-sequenced white rhinoceros genomes, that represents both subspecies at two time windows: before and during/after the bottleneck. Our data not only reveals previously unknown population substructure within both subspecies, but allowed us to quantify the genomic erosion undergone by both, with post-bottleneck white rhinoceroses harbouring significantly fewer heterozygous sites, and showing higher inbreeding coefficients than pre-bottleneck individuals. Moreover, the effective population size suffered a decrease of two and three orders of magnitude in the NWR and SWR respectively, due to the recent bottleneck. Our data therefore provides much needed empirical support for theoretical predictions about the genomic consequences of shrinking populations, information that is relevant for understanding the process of population extinction. Furthermore, our findings have the potential to inform management approaches for the conservation of the remaining white rhinoceroses.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Sánchez-Barreiro, Fátima and Gopalakrishnan, Shyam and Ramos-Madrigal, Jazmín and Westbury, Michael V. and Manuel, Marc de and Margaryan, Ashot and Ciucani, Marta M. and Vieira, Filipe G. and Patramanis, Yannis and Kalthoff, Daniela C. and Timmons, Zena and Sicheritz-Pontén, Thomas and Dalén, Love and Ryder, Oliver A. and Zhang, Guojie and Marquès-Bonet, Tomás and Moodley, Yoshan and Gilbert, M. Thomas P.},\\n', '\\tmonth = may,\\n', '\\tyear = {2020},\\n', '\\tpages = {2020.05.10.086686}\\n', '}\\n', '\\n', '@article{almarri_population_2020,\\n', '\\ttitle = {Population {Structure}, {Stratification} and {Introgression} of {Human} {Structural} {Variation}},\\n', '\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/746172v2},\\n', '\\tdoi = {10.1101/746172},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}Structural variants contribute substantially to genetic diversity and are important evolutionarily and medically, yet are still understudied. Here, we present a comprehensive analysis of deletions, duplications, insertions, inversions and non-reference unique insertions in the Human Genome Diversity Project (HGDP-CEPH) panel, a high-coverage dataset of 911 samples from 54 diverse worldwide populations. We identify in total 126,018 structural variants (25,588 \\\\&lt;100 bp in size), of which 78\\\\% are novel. Some reach high frequency and are private to continental groups or even individual populations, including a deletion in the maltase-glucoamylase gene \\\\textit{MGAM} involved in starch digestion, in the South American Karitiana and a deletion in the Central African Mbuti in \\\\textit{SIGLEC5,} potentially leading to immune hyperactivity. We discover a dynamic range of copy number expansions and find cases of regionally-restricted runaway duplications, for example, 18 copies near the olfactory receptor \\\\textit{OR7D2} in East Asia and in the clinically-relevant \\\\textit{HCAR2} in Central Asia. We identify highly-stratified putatively introgressed variants from Neanderthals or Denisovans, some of which, like a deletion within \\\\textit{AQR} in Papuans, are almost fixed in individual populations. Finally, by \\\\textit{de novo} assembly of 25 genomes using linked-read sequencing we discover 1631 breakpoint-resolved unique insertions, in aggregate accounting for 1.9 Mb of sequence absent from the GRCh38 reference. These insertions show population structure and some reside in functional regions, illustrating the limitation of a single human reference and the need for high-quality genomes from diverse populations to fully discover and understand human genetic variation.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Almarri, Mohamed A. and Bergström, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S. and Chen, Yuan and Hurles, Matthew E. and Tyler-Smith, Chris and Xue, Yali},\\n', '\\tmonth = jan,\\n', '\\tyear = {2020},\\n', '\\tpages = {746172}\\n', '}\\n', '\\n', '@article{almarri2020population,\\n', '  title={Population Structure, Stratification, and Introgression of Human Structural Variation},\\n', '  author={Almarri, Mohamed A and Bergstr{\\\\\"o}m, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S and Chen, Yuan and Hurles, Matthew E and Tyler-Smith, Chris and Xue, Yali},\\n', '  journal={Cell},\\n', '  year={2020},\\n', '  publisher={Elsevier}\\n', '}\\n', '\\n', '@article{belbin_towards_2019,\\n', '\\ttitle = {Towards a fine-scale population health monitoring system},\\n', '\\tcopyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/780668v1},\\n', '\\tdoi = {10.1101/780668},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}Understanding population health disparities is an essential component of equitable precision health efforts. Epidemiology research often relies on definitions of race and ethnicity, but these population labels may not adequately capture disease burdens specific to sub-populations. Here we propose a framework for repurposing data from Electronic Health Records (EHRs) in concert with genomic data to explore enrichment of disease within sub-populations. Using data from a diverse biobank in New York City, we genetically identified 17 sub-populations, and noted the presence of genetic founder effects in 7. By then linking community membership to the EHR, we were able to identify over 600 health outcomes that were statistically enriched within a specific population, with many representing known associations, and many others being novel. This work reinforces the utility of linking genomic data to EHRs, and provides a framework towards fine-scale monitoring of population health.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},\\n', '\\tmonth = sep,\\n', '\\tyear = {2019},\\n', '\\tpages = {780668}\\n', '}\\n', '\\n', '@article{margaryan_population_2019,\\n', '\\ttitle = {Population genomics of the {Viking} world},\\n', '\\tcopyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/703405v1},\\n', '\\tdoi = {10.1101/703405},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}The Viking maritime expansion from Scandinavia (Denmark, Norway, and Sweden) marks one of the swiftest and most far-flung cultural transformations in global history. During this time (c. 750 to 1050 CE), the Vikings reached most of western Eurasia, Greenland, and North America, and left a cultural legacy that persists till today. To understand the genetic structure and influence of the Viking expansion, we sequenced the genomes of 442 ancient humans from across Europe and Greenland ranging from the Bronze Age (c. 2400 BC) to the early Modern period (c. 1600 CE), with particular emphasis on the Viking Age. We find that the period preceding the Viking Age was accompanied by foreign gene flow into Scandinavia from the south and east: spreading from Denmark and eastern Sweden to the rest of Scandinavia. Despite the close linguistic similarities of modern Scandinavian languages, we observe genetic structure within Scandinavia, suggesting that regional population differences were already present 1,000 years ago. We find evidence for a majority of Danish Viking presence in England, Swedish Viking presence in the Baltic, and Norwegian Viking presence in Ireland, Iceland, and Greenland. Additionally, we see substantial foreign European ancestry entering Scandinavia during the Viking Age. We also find that several of the members of the only archaeologically well-attested Viking expedition were close family members. By comparing Viking Scandinavian genomes with present-day Scandinavian genomes, we find that pigmentation-associated loci have undergone strong population differentiation during the last millennia. Finally, we are able to trace the allele frequency dynamics of positively selected loci with unprecedented detail, including the lactase persistence allele and various alleles associated with the immune response. We conclude that the Viking diaspora was characterized by substantial foreign engagement: distinct Viking populations influenced the genomic makeup of different regions of Europe, while Scandinavia also experienced increased contact with the rest of the continent.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Margaryan, Ashot and Lawson, Daniel and Sikora, Martin and Racimo, Fernando and Rasmussen, Simon and Moltke, Ida and Cassidy, Lara and Jørsboe, Emil and Ingason, Andrés and Pedersen, Mikkel and Korneliussen, Thorfinn and Wilhelmson, Helene and Buś, Magdalena and Damgaard, Peter de Barros and Martiniano, Rui and Renaud, Gabriel and Bhérer, Claude and Moreno-Mayar, J. Víctor and Fotakis, Anna and Allen, Marie and Molak, Martyna and Cappellini, Enrico and Scorrano, Gabriele and Buzhilova, Alexandra and Fox, Allison and Albrechtsen, Anders and Schütz, Berit and Skar, Birgitte and Arcini, Caroline and Falys, Ceri and Jonson, Charlotte Hedenstierna and Błaszczyk, Dariusz and Pezhemsky, Denis and Turner-Walker, Gordon and Gestsdóttir, Hildur and Lundstrøm, Inge and Gustin, Ingrid and Mainland, Ingrid and Potekhina, Inna and Muntoni, Italo and Cheng, Jade and Stenderup, Jesper and Ma, Jilong and Gibson, Julie and Peets, Jüri and Gustafsson, Jörgen and Iversen, Katrine and Simpson, Linzi and Strand, Lisa and Loe, Louise and Sikora, Maeve and Florek, Marek and Vretemark, Maria and Redknap, Mark and Bajka, Monika and Pushkina, Tamara and Søvsø, Morten and Grigoreva, Natalia and Christensen, Tom and Kastholm, Ole and Uldum, Otto and Favia, Pasquale and Holck, Per and Allmäe, Raili and Sten, Sabine and Arge, Símun and Ellingvåg, Sturla and Moiseyev, Vayacheslav and Bogdanowicz, Wiesław and Magnusson, Yvonne and Orlando, Ludovic and Bradley, Daniel and Jørkov, Marie Louise and Arneborg, Jette and Lynnerup, Niels and Price, Neil and Gilbert, M. Thomas and Allentoft, Morten and Bill, Jan and Sindbæk, Søren and Hedeager, Lotte and Kristiansen, Kristian and Nielsen, Rasmus and Werge, Thomas and Willerslev, Eske},\\n', '\\tmonth = jul,\\n', '\\tyear = {2019},\\n', '\\tpages = {703405}\\n', '}\\n', '\\n', '@article{consortium_genome_2020,\\n', '\\ttitle = {Genome variation and population structure among 1,142 mosquitoes of the {African} malaria vector species {Anopheles} gambiae and {Anopheles} coluzzii},\\n', '\\tcopyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/},\\n', '\\turl = {https://www.biorxiv.org/content/10.1101/864314v2},\\n', '\\tdoi = {10.1101/864314},\\n', '\\tabstract = {{\\\\textless}h3{\\\\textgreater}Abstract{\\\\textless}/h3{\\\\textgreater} {\\\\textless}p{\\\\textgreater}Mosquito control remains a central pillar of efforts to reduce malaria burden in sub-Saharan Africa. However, insecticide resistance is entrenched in malaria vector populations, and countries with high malaria burden face a daunting challenge to sustain malaria control with a limited set of surveillance and intervention tools. Here we report on the second phase of a project to build an open resource of high quality data on genome variation among natural populations of the major African malaria vector species \\\\textit{Anopheles gambiae} and \\\\textit{Anopheles coluzzii}. We analysed whole genomes of 1,142 individual mosquitoes sampled from the wild in 13 African countries, and a further 234 individuals comprising parents and progeny of 11 lab crosses. The data resource includes high confidence single nucleotide polymorphism (SNP) calls at 57 million variable sites, genome-wide copy number variation (CNV) calls, and haplotypes phased at biallelic SNPs. We used these data to analyse genetic population structure, and characterise genetic diversity within and between populations. We also illustrate the utility of these data by investigating species differences in isolation by distance, genetic variation within proposed gene drive target sequences, and patterns of resistance to pyrethroid insecticides. This data resource provides a foundation for developing new operational systems for molecular surveillance, and for accelerating research and development of new vector control tools.{\\\\textless}/p{\\\\textgreater}},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {bioRxiv},\\n', '\\tauthor = {Consortium, The Anopheles gambiae 1000 Genomes and Clarkson, Chris S. and Miles, Alistair and Harding, Nicholas J. and Lucas, Eric R. and Battey, C. J. and Amaya-Romero, Jorge Edouardo and Cano, Jorge and Diabate, Abdoulaye and Constant, Edi and Nwakanma, Davis C. and Jawara, Musa and Essandoh, John and Dinis, Joao and Goff, Gilbert Le and Robert, Vincent and Troco, Arlete D. and Costantini, Carlo and Rohatgi, Kyanne R. and Elissa, Nohal and Coulibaly, Boubacar and Midega, Janet and Mbogo, Charles and Mawejje, Henry D. and Stalker, Jim and Rockett, Kirk A. and Drury, Eleanor and Mead, Daniel and Jeffreys, Anna E. and Hubbart, Christina and Rowlands, Kate and Isaacs, Alison T. and Jyothi, Dushyanth and Malangone, Cinzia and Kamali, Maryam and Henrichs, Christa and Simpson, Victoria and Ayala, Diego and Besansky, Nora J. and Burt, Austin and Caputo, Beniamino and Torre, Alessandra della and Fontaine, Michael and Godfray, H. Charles J. and Hahn, Matthew W. and Kern, Andrew D. and Lawniczak, Mara K. N. and O’Loughlin, Samantha and Pinto, Joao and Riehle, Michelle M. and Sharakhov, Igor and Schrider, Daniel R. and Vernick, Kenneth D. and White, Bradley J. and Donnelly, Martin J. and Kwiatkowski, Dominic P.},\\n', '\\tmonth = feb,\\n', '\\tyear = {2020},\\n', '\\tpages = {864314}\\n', '}\\n', '\\n', '@article{greenbaum_network-based_2019,\\n', '\\ttitle = {Network-based hierarchical population structure analysis for large genomic data sets},\\n', '\\tvolume = {29},\\n', '\\tissn = {1088-9051, 1549-5469},\\n', '\\turl = {http://genome.cshlp.org/content/29/12/2020},\\n', '\\tdoi = {10.1101/gr.250092.119},\\n', '\\tabstract = {Analysis of population structure in natural populations using genetic data is a common practice in ecological and evolutionary studies. With large genomic data sets of populations now appearing more frequently across the taxonomic spectrum, it is becoming increasingly possible to reveal many hierarchical levels of structure, including fine-scale genetic clusters. To analyze these data sets, methods need to be appropriately suited to the challenges of extracting multilevel structure from whole-genome data. Here, we present a network-based approach for constructing population structure representations from genetic data. The use of community-detection algorithms from network theory generates a natural hierarchical perspective on the representation that the method produces. The method is computationally efficient, and it requires relatively few assumptions regarding the biological processes that underlie the data. We show the approach by analyzing population structure in the model plant species Arabidopsis thaliana and in human populations. These examples illustrate how network-based approaches for population structure analysis are well-suited to extracting valuable ecological and evolutionary information in the era of large genomic data sets.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {12},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Genome {R}esearch},\\n', '\\tauthor = {Greenbaum, Gili and Rubin, Amir and Templeton, Alan R. and Rosenberg, Noah A.},\\n', '\\tmonth = jan,\\n', '\\tyear = {2019},\\n', '\\tpmid = {31694865},\\n', '\\tpages = {2020--2033}\\n', '}\\n', '\\n', '@article{yamamoto_genetic_2020,\\n', '\\ttitle = {Genetic and phenotypic landscape of the mitochondrial genome in the {Japanese} population},\\n', '\\tvolume = {3},\\n', '\\tcopyright = {2020 The Author(s)},\\n', '\\tissn = {2399-3642},\\n', '\\turl = {https://www.nature.com/articles/s42003-020-0812-9},\\n', '\\tdoi = {10.1038/s42003-020-0812-9},\\n', '\\tabstract = {The genetic landscape of mitochondrial DNA (mtDNA) has been elusive. By analyzing mtDNA using the whole genome sequence (WGS) of Japanese individuals (n\\u2009=\\u20091928), we identified 2023 mtDNA variants and high-resolution haplogroups. Frequency spectra of the haplogroups were population-specific and were heterogeneous among geographic regions within Japan. Application of machine learning methods could finely classify the subjects corresponding to the high-digit mtDNA sub-haplogroups. mtDNA had distinct genetic structures from that of nuclear DNA (nDNA), characterized by no distance-dependent linkage disequilibrium decay, sparse tagging of common variants, and the existence of common haplotypes spanning the entire mtDNA. We did not detect any evidence of mtDNA–nDNA (or mtDNA copy number–nDNA) genotype associations. Together with WGS-based mtDNA variant imputation, we conducted a phenome-wide association study of 147,437 Japanese individuals with 99 clinical phenotypes. We observed pleiotropy of mtDNA genetic risk on the five late-onset human complex traits including creatine kinase (P\\u2009=\\u20091.7\\u2009×\\u200910−12).},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {1},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Communications Biology},\\n', '\\tauthor = {Yamamoto, Kenichi and Sakaue, Saori and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Ozono, Keiichi and Momozawa, Yukihide and Okada, Yukinori},\\n', '\\tmonth = mar,\\n', '\\tyear = {2020},\\n', '\\tpages = {1--11}\\n', '}\\n', '\\n', '@article{derkarabetian_demonstration_2019,\\n', '\\ttitle = {A demonstration of unsupervised machine learning in species delimitation},\\n', '\\tvolume = {139},\\n', '\\tissn = {1055-7903},\\n', '\\turl = {http://www.sciencedirect.com/science/article/pii/S1055790319301721},\\n', '\\tdoi = {10.1016/j.ympev.2019.106562},\\n', '\\tabstract = {One major challenge to delimiting species with genetic data is successfully differentiating population structure from species-level divergence, an issue exacerbated in taxa inhabiting naturally fragmented habitats. Many fields of science are now using machine learning, and in evolutionary biology supervised machine learning has recently been used to infer species boundaries. These supervised methods require training data with associated labels. Conversely, unsupervised machine learning (UML) uses inherent data structure and does not require user-specified training labels, potentially providing more objectivity in species delimitation. In the context of integrative taxonomy, we demonstrate the utility of three UML approaches (random forests, variational autoencoders, t-distributed stochastic neighbor embedding) for species delimitation in an arachnid taxon with high population genetic structure (Opiliones, Laniatores, Metanonychus). We find that UML approaches successfully cluster samples according to species-level divergences and not high levels of population structure, while model-based validation methods severely over-split putative species. UML offers intuitive data visualization in two-dimensional space, the ability to accommodate various data types, and has potential in many areas of systematic and evolutionary biology. We argue that machine learning methods are ideally suited for species delimitation and may perform well in many natural systems and across taxa with diverse biological characteristics.},\\n', '\\tlanguage = {en},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Molecular Phylogenetics and Evolution},\\n', '\\tauthor = {Derkarabetian, Shahan and Castillo, Stephanie and Koo, Peter K. and Ovchinnikov, Sergey and Hedin, Marshal},\\n', '\\tmonth = oct,\\n', '\\tyear = {2019},\\n', '\\tkeywords = {t-SNE, Integrative taxonomy, Opiliones, Random forest, Ultraconserved elements, Variational autoencoders},\\n', '\\tpages = {106562}\\n', '}\\n', '\\n', '@article{diaz-papkovich_umap_2019,\\n', '\\ttitle = {{UMAP} reveals cryptic population structure and phenotype heterogeneity in large genomic cohorts},\\n', '\\tvolume = {15},\\n', '\\tissn = {1553-7390},\\n', '\\turl = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853336/},\\n', '\\tdoi = {10.1371/journal.pgen.1008432},\\n', '\\tabstract = {Human populations feature both discrete and continuous patterns of variation. Current analysis approaches struggle to jointly identify these patterns because of modelling assumptions, mathematical constraints, or numerical challenges. Here we apply uniform manifold approximation and projection (UMAP), a non-linear dimension reduction tool, to three well-studied genotype datasets and discover overlooked subpopulations within the American Hispanic population, fine-scale relationships between geography, genotypes, and phenotypes in the UK population, and cryptic structure in the Thousand Genomes Project data. This approach is well-suited to the influx of large and diverse data and opens new lines of inquiry in population-scale datasets., The demographic history of human populations features varying geographic and social barriers to mating. Over time, these barriers have led to varying levels of genetic relatedness among individuals. This population structure is informative about human history, and can have a significant impact on studies of medical genetics. Because population structure depends on myriad demographic, ecological, and social forces, a priori visualization is useful to identify subtle patterns of population structure. We use a dimension reduction method—UMAP—to visualize population structure in three genomic datasets and find previously unobserved patterns, revealing fine-scale population structure and illustrating differences between groups in traits such as white blood cell count, height, and FEV1, a measure of lung function. Using UMAP is computationally efficient and can identify fine-scale population structure in large population datasets. We find it particularly useful to reveal phenotypic variation among genetically related populations, and recommend it is a complement to principal component analysis in primary data visualization.},\\n', '\\tnumber = {11},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {PLoS Genetics},\\n', '\\tauthor = {Diaz-Papkovich, Alex and Anderson-Trocmé, Luke and Ben-Eghan, Chief and Gravel, Simon},\\n', '\\tmonth = nov,\\n', '\\tyear = {2019},\\n', '\\tpmid = {31675358},\\n', '\\tpmcid = {PMC6853336}\\n', '}\\n', '\\n', '@article{li_past_2020,\\n', '\\ttitle = {Past demography and local adaptation in forest trees : {Insights} from natural populations and breeding programs of {Norway} spruce},\\n', '\\tshorttitle = {Past demography and local adaptation in forest trees},\\n', '\\turl = {http://urn.kb.se/resolve?urn=urn:nbn:se:uu:diva-403450},\\n', '\\tabstract = {DiVA portal is a finding tool for research publications and student theses written at the following 49 universities and research institutions.},\\n', '\\tlanguage = {eng},\\n', '\\turldate = {2020-05-21},\\n', '\\tauthor = {Li, Lili},\\n', '\\tyear = {2020}\\n', '}\\n', '\\n', '@article{becht_dimensionality_2019,\\n', '\\ttitle = {Dimensionality reduction for visualizing single-cell data using {UMAP}},\\n', '\\tvolume = {37},\\n', '\\tcopyright = {2018 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.},\\n', '\\tissn = {1546-1696},\\n', '\\turl = {https://www.nature.com/articles/nbt.4314},\\n', '\\tdoi = {10.1038/nbt.4314},\\n', '\\tabstract = {A benchmarking analysis on single-cell RNA-seq and mass cytometry data reveals the best-performing technique for dimensionality reduction.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {1},\\n', '\\turldate = {2020-05-21},\\n', '\\tjournal = {Nature Biotechnology},\\n', '\\tauthor = {Becht, Etienne and McInnes, Leland and Healy, John and Dutertre, Charles-Antoine and Kwok, Immanuel W. H. and Ng, Lai Guan and Ginhoux, Florent and Newell, Evan W.},\\n', '\\tmonth = jan,\\n', '\\tyear = {2019},\\n', '\\tpages = {38--44}\\n', '}\\n', '\\n', '@article{kobak_art_2019,\\n', '\\ttitle = {The art of using t-{SNE} for single-cell transcriptomics},\\n', '\\tvolume = {10},\\n', '\\tcopyright = {2019 The Author(s)},\\n', '\\tissn = {2041-1723},\\n', '\\turl = {https://www.nature.com/articles/s41467-019-13056-x},\\n', '\\tdoi = {10.1038/s41467-019-13056-x},\\n', '\\tabstract = {Single-cell transcriptomics yields ever growing data sets containing RNA expression levels for thousands of genes from up to millions of cells. Common data analysis pipelines include a dimensionality reduction step for visualising the data in two dimensions, most frequently performed using t-distributed stochastic neighbour embedding (t-SNE). It excels at revealing local structure in high-dimensional data, but naive applications often suffer from severe shortcomings, e.g. the global structure of the data is not represented accurately. Here we describe how to circumvent such pitfalls, and develop a protocol for creating more faithful t-SNE visualisations. It includes PCA initialisation, a high learning rate, and multi-scale similarity kernels; for very large data sets, we additionally use exaggeration and downsampling-based initialisation. We use published single-cell RNA-seq data sets to demonstrate that this protocol yields superior results compared to the naive application of t-SNE.},\\n', '\\tlanguage = {en},\\n', '\\tnumber = {1},\\n', '\\turldate = {2020-06-02},\\n', '\\tjournal = {Nature Communications},\\n', '\\tauthor = {Kobak, Dmitry and Berens, Philipp},\\n', '\\tmonth = nov,\\n', '\\tyear = {2019},\\n', '\\tpages = {5416}\\n', '}\\n', '\\n', '@article{spear2020recent,\\n', '  title={Recent fluctuations in Mexican American genomes have altered the genetic architecture of biomedical traits},\\n', '  author={Spear, Melissa L and Diaz-Papkovich, Alex and Ziv, Elad and Gravel, Simon and Torgerson, Dara G and Hernandez, Ryan},\\n', '  journal={bioRxiv},\\n', '  year={2020},\\n', '  publisher={Cold Spring Harbor Laboratory}\\n', '}\\n', '\\n', '@article{patterson2006population,\\n', '  title={Population structure and eigenanalysis},\\n', '  author={Patterson, Nick and Price, Alkes L and Reich, David},\\n', '  journal={PLoS Genetics},\\n', '  volume={2},\\n', '  number={12},\\n', '  pages={e190},\\n', '  year={2006},\\n', '  publisher={Public Library of Science},\\n', '  doi={10.1371/journal.pgen.0020190}\\n', '}\\n', '\\n', '@article{10002015global,\\n', '  title={A global reference for human genetic variation},\\n', '  author={1000 Genomes Project Consortium and others},\\n', '  journal={Nature},\\n', '  volume={526},\\n', '  number={7571},\\n', '  pages={68--74},\\n', '  year={2015},\\n', '  publisher={Nature Publishing Group}\\n', '}\\n', '\\n', '@article{cann2002human,\\n', '  title={A human genome diversity cell line panel},\\n', '  author={Cann, Howard M and De Toma, Claudia and Cazes, Lucien and Legrand, Marie-Fernande and Morel, Valerie and Piouffre, Laurence and Bodmer, Julia and Bodmer, Walter F and Bonne-Tamir, Batsheva and Cambon-Thomsen, Anne and others},\\n', '  journal={Science},\\n', '  volume={296},\\n', '  number={5566},\\n', '  pages={261--262},\\n', '  year={2002},\\n', '  publisher={American Association for the Advancement of Science}\\n', '}\\n', '\\n', '@article{matise2011next,\\n', '  title={The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study},\\n', '  author={Matise, Tara C and Ambite, Jose Luis and Buyske, Steven and Carlson, Christopher S and Cole, Shelley A and Crawford, Dana C and Haiman, Christopher A and Heiss, Gerardo and Kooperberg, Charles and Marchand, Loic Le and others},\\n', '  journal={American journal of epidemiology},\\n', '  volume={174},\\n', '  number={7},\\n', '  pages={849--859},\\n', '  year={2011},\\n', '  publisher={Oxford University Press}\\n', '}\\n', '\\n', '@article{nelson2008population,\\n', '  title={The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research},\\n', \"  author={Nelson, Matthew R and Bryc, Katarzyna and King, Karen S and Indap, Amit and Boyko, Adam R and Novembre, John and Briley, Linda P and Maruyama, Yuka and Waterworth, Dawn M and Waeber, G{\\\\'e}rard and others},\\n\", '  journal={The American Journal of Human Genetics},\\n', '  volume={83},\\n', '  number={3},\\n', '  pages={347--358},\\n', '  year={2008},\\n', '  publisher={Elsevier}\\n', '}\\n', '\\n', '@article{juster1995overview,\\n', '  title={An overview of the Health and Retirement Study},\\n', '  author={Juster, F Thomas and Suzman, Richard},\\n', '  journal={Journal of Human Resources},\\n', '  pages={S7--S56},\\n', '  year={1995},\\n', '  publisher={JSTOR}\\n', '}\\n', '\\n', '@article{sudlow2015uk,\\n', '  title={UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age},\\n', '  author={Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and others},\\n', '  journal={Plos med},\\n', '  volume={12},\\n', '  number={3},\\n', '  pages={e1001779},\\n', '  year={2015},\\n', '  publisher={Public Library of Science}\\n', '}\\n', '\\n', '@article{sorlie2010design,\\n', '  title={Design and implementation of the Hispanic community health study/study of Latinos},\\n', \"  author={Sorlie, Paul D and Avil{\\\\'e}s-Santa, Larissa M and Wassertheil-Smoller, Sylvia and Kaplan, Robert C and Daviglus, Martha L and Giachello, Aida L and Schneiderman, Neil and Raij, Leopoldo and Talavera, Gregory and Allison, Matthew and others},\\n\", '  journal={Annals of epidemiology},\\n', '  volume={20},\\n', '  number={8},\\n', '  pages={629--641},\\n', '  year={2010},\\n', '  publisher={Elsevier}\\n', '}\\n', '\\n', '@article{nagai2017overview,\\n', '  title={Overview of the {BioBank Japan Project}: study design and profile},\\n', '  author={Nagai, Akiko and Hirata, Makoto and Kamatani, Yoichiro and Muto, Kaori and Matsuda, Koichi and Kiyohara, Yutaka and Ninomiya, Toshiharu and Tamakoshi, Akiko and Yamagata, Zentaro and Mushiroda, Taisei and others},\\n', '  journal={Journal of epidemiology},\\n', '  volume={27},\\n', '  number={Supplement\\\\_III},\\n', '  pages={S2--S8},\\n', '  year={2017},\\n', '  publisher={Japan Epidemiological Association}\\n', '}\\n', '\\n', '@article{lawson2012inference,\\n', '  title={Inference of population structure using dense haplotype data},\\n', '  author={Lawson, Daniel John and Hellenthal, Garrett and Myers, Simon and Falush, Daniel},\\n', '  journal={PLoS Genetics},\\n', '  volume={8},\\n', '  number={1},\\n', '  pages={e1002453},\\n', '  year={2012},\\n', '  publisher={Public Library of Science}\\n', '}\\n', '\\n', '@article{alexander2009fast,\\n', '  title={Fast model-based estimation of ancestry in unrelated individuals},\\n', '  author={Alexander, David H and Novembre, John and Lange, Kenneth},\\n', '  journal={Genome {R}esearch},\\n', '  volume={19},\\n', '  number={9},\\n', '  pages={1655--1664},\\n', '  year={2009},\\n', '  publisher={Cold Spring Harbor Lab}\\n', '}\\n', '\\n', '@Article{Hunter2007,\\n', '  Author    = {Hunter, J. D.},\\n', '  Title     = {Matplotlib: A 2D graphics environment},\\n', '  Journal   = {Computing In Science \\\\& Engineering},\\n', '  Volume    = {9},\\n', '  Number    = {3},\\n', '  Pages     = {90--95},\\n', '  abstract  = {Matplotlib is a 2D graphics package used for Python\\n', '  for application development, interactive scripting, and\\n', '  publication-quality image generation across user\\n', '  interfaces and operating systems.},\\n', '  publisher = {IEEE COMPUTER SOC},\\n', '  doi       = {10.1109/MCSE.2007.55},\\n', '  year      = 2007\\n', '}\\n', '\\n', '@article{scikit-learn,\\n', ' title={Scikit-learn: Machine Learning in {P}ython},\\n', ' author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.\\n', '         and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.\\n', '         and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and\\n', '         Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},\\n', ' journal={Journal of Machine Learning Research},\\n', ' volume={12},\\n', ' pages={2825--2830},\\n', ' year={2011}\\n', '}\\n', '\\n', '@article{kerminen2019geographic,\\n', '  title={Geographic variation and bias in the polygenic scores of complex diseases and traits in {F}inland},\\n', '  author={Kerminen, Sini and Martin, Alicia R and Koskela, Jukka and Ruotsalainen, Sanni E and Havulinna, Aki S and Surakka, Ida and Palotie, Aarno and Perola, Markus and Salomaa, Veikko and Daly, Mark J and others},\\n', '  journal={The American Journal of Human Genetics},\\n', '  volume={104},\\n', '  number={6},\\n', '  pages={1169--1181},\\n', '  year={2019},\\n', '  publisher={Elsevier}\\n', '}\\n', '\\n', '@article{berg2019reduced,\\n', '  title={Reduced signal for polygenic adaptation of height in {UK} {B}iobank},\\n', '  author={Berg, Jeremy J and Harpak, Arbel and Sinnott-Armstrong, Nasa and Joergensen, Anja Moltke and Mostafavi, Hakhamanesh and Field, Yair and Boyle, Evan August and Zhang, Xinjun and Racimo, Fernando and Pritchard, Jonathan K and others},\\n', '  journal={e{L}ife},\\n', '  volume={8},\\n', '  pages={e39725},\\n', '  year={2019},\\n', '  publisher={eLife Sciences Publications Limited}\\n', '}\\n', '\\n', '@article{sohail2019polygenic,\\n', '  title={Polygenic adaptation on height is overestimated due to uncorrected stratification in genome-wide association studies},\\n', '  author={Sohail, Mashaal and Maier, Robert M and Ganna, Andrea and Bloemendal, Alex and Martin, Alicia R and Turchin, Michael C and Chiang, Charleston WK and Hirschhorn, Joel and Daly, Mark J and Patterson, Nick and others},\\n', '  journal={e{L}ife},\\n', '  volume={8},\\n', '  pages={e39702},\\n', '  year={2019},\\n', '  publisher={eLife Sciences Publications Limited}\\n', '}\\n', '\\n', '@article{mcvean2009genealogical,\\n', '  title={A genealogical interpretation of principal components analysis},\\n', '  author={McVean, Gil},\\n', '  journal={PLoS Genetics},\\n', '  volume={5},\\n', '  number={10},\\n', '  pages={e1000686},\\n', '  year={2009},\\n', '  publisher={Public Library of Science},\\n', '  doi = {10.1371/journal.pgen.1000686}\\n', '}\\n', '\\n', '@article{moon2019visualizing,\\n', '  title={Visualizing structure and transitions in high-dimensional biological data},\\n', '  author={Moon, Kevin R and van Dijk, David and Wang, Zheng and Gigante, Scott and Burkhardt, Daniel B and Chen, William S and Yim, Kristina and van den Elzen, Antonia and Hirn, Matthew J and Coifman, Ronald R and others},\\n', '  journal={Nature {B}iotechnology},\\n', '  volume={37},\\n', '  number={12},\\n', '  pages={1482--1492},\\n', '  year={2019},\\n', '  publisher={Nature Publishing Group}\\n', '}\\n', '\\n', '@article{micheletti_genetic_2020,\\n', '\\ttitle = {Genetic {Consequences} of the {Transatlantic} {Slave} {Trade} in the {Americas}},\\n', '\\tvolume = {0},\\n', '\\tissn = {0002-9297, 1537-6605},\\n', '\\turl = {https://www.cell.com/ajhg/abstract/S0002-9297(20)30200-7},\\n', '\\tdoi = {10.1016/j.ajhg.2020.06.012},\\n', '\\tlanguage = {English},\\n', '\\tnumber = {0},\\n', '\\tjournal = {The American Journal of Human Genetics},\\n', '\\tauthor = {Micheletti, Steven J. and Bryc, Kasia and Esselmann, Samantha G. Ancona and Freyman, William A. and Moreno, Meghan E. and Poznik, G. David and Shastri, Anjali J. and Agee, M. and Aslibekyan, S. and Auton, A. and Bell, R. and Clark, S. and Das, S. and Elson, S. and Fletez-Brant, K. and Fontanillas, P. and Gandhi, P. and Heilbron, K. and Hicks, B. and Hinds, D. and Huber, K. and Jewett, E. and Jiang, Y. and Kleinman, A. and Lin, K. and Litterman, N. and McCreight, J. and McIntyre, M. and McManus, K. and Mozaffari, S. and Nandakumar, P. and Noblin, L. and Northover, C. and O’Connell, J. and Petrakovitz, A. and Pitts, S. and Shelton, J. and Shringarpure, S. and Tian, C. and Tung, J. and Tunney, R. and Vacic, V. and Wang, X. and Zare, A. and Beleza, Sandra and Mountain, Joanna L.},\\n', '\\tmonth = jul,\\n', '\\tyear = {2020}\\n', '}\\n', '\\n', '@book{holmes2018modern,\\n', '  title={Modern statistics for modern biology},\\n', '  author={Holmes, Susan and Huber, Wolfgang},\\n', '  year={2018},\\n', '  chapter={Introduction},\\n', '  publisher={Cambridge University Press}\\n', '}\\n', '\\n', '@misc{genizon,\\n', 'title = {Genizon Biobank},\\n', \"publisher = {Genome Qu\\\\'{e}bec},\\n\", 'year={2020},\\n', 'url={http://www.genomequebec.com/genizon-biobank/}\\n', '}\\n', '\\n', '@article{mcinnes2018software,\\n', '  title={{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection}},\\n', '  author={McInnes, Leland and Healy, John and Saul, Nathaniel and Grossberger, Lukas},\\n', '  journal={The Journal of Open Source Software},\\n', '  volume={3},\\n', '  number={29},\\n', '  pages={861},\\n', '  year={2018}\\n', '}\\n', '\\n', '@article{mathieson2020ancestry,\\n', '  title={What is ancestry?},\\n', '  author={Mathieson, Iain and Scally, Aylwyn},\\n', '  journal={PLoS Genetics},\\n', '  volume={16},\\n', '  number={3},\\n', '  pages={e1008624},\\n', '  year={2020},\\n', '  publisher={Public Library of Science San Francisco, CA USA}\\n', '}\\n', '\\n', '@article{maaten2008visualizing,\\n', '  title={Visualizing data using t-SNE},\\n', '  author={Maaten, Laurens van der and Hinton, Geoffrey},\\n', '  journal={Journal of machine learning research},\\n', '  volume={9},\\n', '  number={Nov},\\n', '  pages={2579--2605},\\n', '  year={2008}\\n', '}\\n', '\\n', '@article{belkin2003laplacian,\\n', '  title={Laplacian eigenmaps for dimensionality reduction and data representation},\\n', '  author={Belkin, Mikhail and Niyogi, Partha},\\n', '  journal={Neural computation},\\n', '  volume={15},\\n', '  number={6},\\n', '  pages={1373--1396},\\n', '  year={2003},\\n', '  publisher={MIT Press}\\n', '}']\n"
     ]
    }
   ],
   "source": [
    "print(lines)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "4a0a0031",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "@article{wei_graph_2019,\n",
      "\n",
      "title = {Graph {Embedding} {Method} {Based} genetical trajectory {Reveals} {Migration} {History} {Among} {East} {Asians}},\n",
      "\n",
      "copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/870253v1},\n",
      "\n",
      "doi = {10.1101/870253},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}ABSTRACT{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}An important issue in human population genetics is the ancestry. By extracting the ancestral information retained in the single nucleotide polymorphism (SNP) of genomic DNA, the history of migration and reproduction of the population can be reconstructed. Since the SNP data of population are multidimensional, their dimensionality reduction can demonstrate their potential internal connections. In this study, the graph and structure learning based Graph Embedding method commonly used in single cell mRNA sequencing was applied to human population genetics research to decrease the data dimension. As a result, the human population trajectory of East Asia based on 1000 Genomes Project was reconstructed to discover the inseparable relationship between the Chinese population and other East Asian populations. These results are visualized from various ancestry calculators such as E11 and K12B. Finally, the unique SNPs along the psudotime of trajectory were found by differential analysis. Bioprocess enrichment analysis was also used to reveal that the genes of these SNPs may be related to neurological diseases. These results will lay the data foundation for precision medicine.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Wei, Zhuang and Chang, Ching-Wen and Luo, Van and Bian, Beilei and Ding, Xuewei},\n",
      "\n",
      "month = dec,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {870253}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{simon_local_2019,\n",
      "\n",
      "title = {Local introgression at two spatial scales in mosaic hybrid zones of mussels},\n",
      "\n",
      "copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution 4.0 International), CC BY 4.0, as described at http://creativecommons.org/licenses/by/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/818559v1},\n",
      "\n",
      "doi = {10.1101/818559},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}When the ranges of closely-related lineages are large, and overlapping, we can often study introgression at many “replicated” contacts, with different locations and spatial scales. Here we analysed multiple contact zones of the \\textit{M. edulis} complex of marine mussel species, which represent a mosaic distribution of heterogeneously differentiated, semi-isolated genomes. Our aim was to contrast ongoing introgression at the heart of hybrid zones, with past introgression between similar parental populations, at increasing distance from the contact. Using a panel of ancestry-informative SNPs derived from a previous genomic study, we first confirm, with a broader sampling, that local introgression, affecting one but not all of the populations compared, is both widespread and heterogeneous across the genome. Some outlier loci show patterns of complete introgression in certain populations, and an absence of introgression in others. Genomic cline analyses reveal a globally high concordance among loci at a local scale, albeit with signals of asymmetric introgression at a few loci. Enhanced local introgression at specific loci is consistent with the early transfer of adaptive variants after contact, possibly including asymmetric bi-stable variants, or less loaded alleles. Given the mosaic structure of the \\textit{M. edulis} complex, with a succession of genetic barriers to gene flow, variants with enhanced introgression through one barrier can be trapped, maybe transiently, at the next barrier, confining introgression locally. This makes the \\textit{Mytilus} complex an ideal model of the heterogeneous porosity of species barriers.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Simon, Alexis and Fraïsse, Christelle and Ayari, Tahani El and Liautard-Haag, Cathy and Strelkov, Petr and Welch, John J. and Bierne, Nicolas},\n",
      "\n",
      "month = oct,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {818559}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{dorrity_dimensionality_2020,\n",
      "\n",
      "title = {Dimensionality reduction by {UMAP} to visualize physical and genetic interactions},\n",
      "\n",
      "volume = {11},\n",
      "\n",
      "copyright = {2020 The Author(s)},\n",
      "\n",
      "issn = {2041-1723},\n",
      "\n",
      "url = {https://www.nature.com/articles/s41467-020-15351-4},\n",
      "\n",
      "doi = {10.1038/s41467-020-15351-4},\n",
      "\n",
      "abstract = {Dimensionality reduction is often used to visualize complex expression profiling data. Here, we use the Uniform Manifold Approximation and Projection (UMAP) method on published transcript profiles of 1484 single gene deletions of Saccharomyces cerevisiae. Proximity in low-dimensional UMAP space identifies groups of genes that correspond to protein complexes and pathways, and finds novel protein interactions, even within well-characterized complexes. This approach is more sensitive than previous methods and should be broadly useful as additional transcriptome datasets become available for other organisms.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {1},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Nature Communications},\n",
      "\n",
      "author = {Dorrity, Michael W. and Saunders, Lauren M. and Queitsch, Christine and Fields, Stanley and Trapnell, Cole},\n",
      "\n",
      "month = mar,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {1537}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{yelmen_creating_2019,\n",
      "\n",
      "title = {Creating {Artificial} {Human} {Genomes} {Using} {Generative} {Models}},\n",
      "\n",
      "copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/769091v2},\n",
      "\n",
      "doi = {10.1101/769091},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}Generative models have shown breakthroughs in a wide spectrum of domains due to recent advancements in machine learning algorithms and increased computational power. Despite these impressive achievements, the ability of generative models to create realistic synthetic data is still under-exploited in genetics and absent from population genetics.{\\textless}/p{\\textgreater}{\\textless}p{\\textgreater}Yet a known limitation of this field is the reduced access to many genetic databases due to concerns about violations of individual privacy, although they would provide a rich resource for data mining and integration towards advancing genetic studies. In this study, we demonstrated that deep generative adversarial networks (GANs) and restricted Boltzmann machines (RBMs) can be trained to learn the high dimensional distributions of real genomic datasets and create high quality artificial genomes (AGs) with none to little privacy loss. To illustrate the promising outcomes of our method, we showed that (i) imputation quality for low frequency alleles can be improved by augmenting reference panels with AGs, (ii) scores obtained from selection tests on AGs and real genomes are highly correlated and (iii) AGs can inherit genotype-phenotype associations. AGs have the potential to become valuable assets in genetic studies by providing high quality anonymous substitutes for private databases.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Yelmen, Burak and Decelle, Aurélien and Ongaro, Linda and Marnetto, Davide and Tallec, Corentin and Montinaro, Francesco and Furtlehner, Cyril and Pagani, Luca and Jay, Flora},\n",
      "\n",
      "month = oct,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {769091}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{schmidt_population_2020,\n",
      "\n",
      "title = {Population genomics of two invasive mosquitoes ({Aedes} aegypti and {Aedes} albopictus) from the {Indo}-{Pacific}},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/2020.03.15.993055v3},\n",
      "\n",
      "doi = {10.1101/2020.03.15.993055},\n",
      "\n",
      "abstract = {{\\textless}p{\\textgreater}The arbovirus vectors Aedes aegypti (yellow fever mosquito) and Ae. albopictus (Asian tiger mosquito) are both common throughout the Indo-Pacific region, where 70\\% of global dengue transmission occurs. For Ae. aegypti all Indo-Pacific populations are invasive, having spread from an initial native range of Africa, while for Ae. albopictus the Indo-Pacific includes invasive populations and those from the native range: putatively, India to Japan to Southeast Asia. This study analyses the population genomics of 480 of these mosquitoes sampled from 27 locations in the Indo-Pacific. We investigated patterns of genome-wide genetic differentiation to compare pathways of invasion and ongoing gene flow in both species, and to compare invasive and native-range populations of Ae. albopictus. We also tested landscape genomic hypotheses that genetic differentiation would increase with geographical distance and be lower between locations with high connectivity to human transportation routes, the primary means of dispersal at these scales. We found that genetic distances were generally higher in Ae. aegypti, with Pacific populations the most highly differentiated. The most differentiated Ae. albopictus populations were in Vanuatu, Indonesia and Sri Lanka, the latter two representing potential native-range populations and potential cryptic subspeciation respectively. Genetic distances in Ae. aegypti increased with geographical distance, while in Ae. albopictus they decreased with higher connectivity to human transportation routes. Contrary to the situation in Ae. aegypti, we found evidence of long-distance Ae. albopictus colonisation events, including colonisation of Mauritius from East Asia and of Fiji from Southeast Asia. These direct genomic comparisons indicate likely differences in dispersal ecology in these species, despite their broadly sympatric distributions and similar use of human transport to disperse. Our findings will assist biosecurity operations to trace the source of invasive material and for biocontrol operations that benefit from matching genetic backgrounds of released and local populations.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Schmidt, Thomas L. and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R. and Hoffmann, Ary A.},\n",
      "\n",
      "month = may,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {2020.03.15.993055}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{schmidt2020population,\n",
      "\n",
      "title={Population genomics of two invasive mosquitoes (Aedes aegypti and Aedes albopictus) from the Indo-Pacific},\n",
      "\n",
      "author={Schmidt, Thomas L and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R and Hoffmann, Ary A},\n",
      "\n",
      "journal={bioRxiv},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "publisher={Cold Spring Harbor Laboratory}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{zhang_generalized_2020,\n",
      "\n",
      "title = {A {Generalized} {Robust} {Allele}-based {Genetic} {Association} {Test}},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. The copyright holder for this pre-print is the author. All rights reserved. The material may not be redistributed, re-used or adapted without the author's permission.},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/2020.03.12.989004v1},\n",
      "\n",
      "doi = {10.1101/2020.03.12.989004},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}The allele-based association test or the allelic test, comparing allele frequency difference between case and control groups, is locally most powerful. However, the classical allelic test is limited in applications because it is sensitive to the Hardy–Weinberg equilibrium (HWE) assumption, not applicable to continuous traits, and not easy to account for covariate effects or sample correlation. To develop a generalized robust allelic test, we propose a unifying regression model with individual allele as the response variable. We show that the score test statistic derived from this novel regression framework contains a correction factor that explicitly adjusts for the departure from HWE and encompasses the classical allelic test as a special case. When the trait of interest is continuous, the corresponding allelic test evaluates a weighted difference between individual-level allele frequency estimate and sample estimate where the weight is proportional to an individual’s trait value, and the test remains valid under Y-dependent sampling. Finally, the proposed method allows for joint allele-based association analyses of multiple (continuous or binary) phenotypes, in the presence of covariates, sample correlation and population heterogeneity. To support our analytical findings, we provide empirical evidence from both simulation and application studies.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Zhang, Lin and Sun, Lei},\n",
      "\n",
      "month = mar,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {2020.03.12.989004}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcinnes_umap_2018,\n",
      "\n",
      "title = {{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection} for {Dimension} {Reduction}},\n",
      "\n",
      "shorttitle = {{UMAP}},\n",
      "\n",
      "url = {http://arxiv.org/abs/1802.03426},\n",
      "\n",
      "abstract = {UMAP (Uniform Manifold Approximation and Projection) is a novel manifold learning technique for dimension reduction. UMAP is constructed from a theoretical framework based in Riemannian geometry and algebraic topology. The result is a practical scalable algorithm that applies to real world data. The UMAP algorithm is competitive with t-SNE for visualization quality, and arguably preserves more of the global structure with superior run time performance. Furthermore, UMAP has no computational restrictions on embedding dimension, making it viable as a general purpose dimension reduction technique for machine learning.},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {arXiv},\n",
      "\n",
      "author = {McInnes, Leland and Healy, John and Melville, James},\n",
      "\n",
      "month = dec,\n",
      "\n",
      "year = {2018}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sakaue_dimensionality_2020,\n",
      "\n",
      "title = {Dimensionality reduction reveals fine-scale structure in the {Japanese} population with consequences for polygenic risk prediction},\n",
      "\n",
      "volume = {11},\n",
      "\n",
      "copyright = {2020 The Author(s)},\n",
      "\n",
      "issn = {2041-1723},\n",
      "\n",
      "url = {https://www.nature.com/articles/s41467-020-15194-z},\n",
      "\n",
      "doi = {10.1038/s41467-020-15194-z},\n",
      "\n",
      "abstract = {The diversity in our genome is crucial to understanding the demographic history of worldwide populations. However, we have yet to know whether subtle genetic differences within a population can be disentangled, or whether they have an impact on complex traits. Here we apply dimensionality reduction methods (PCA, t-SNE, PCA-t-SNE, UMAP, and PCA-UMAP) to biobank-derived genomic data of a Japanese population (n = 169,719). Dimensionality reduction reveals fine-scale population structure, conspicuously differentiating adjacent insular subpopulations. We further enluciate the demographic landscape of these Japanese subpopulations using population genetics analyses. Finally, we perform phenome-wide polygenic risk score (PRS) analyses on 67 complex traits. Differences in PRS between the deconvoluted subpopulations are not always concordant with those in the observed phenotypes, suggesting that the PRS differences might reflect biases from the uncorrected structure, in a trait-dependent manner. This study suggests that such an uncorrected structure can be a potential pitfall in the clinical application of PRS.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {1},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Nature Communications},\n",
      "\n",
      "author = {Sakaue, Saori and Hirata, Jun and Kanai, Masahiro and Suzuki, Ken and Akiyama, Masato and Lai Too, Chun and Arayssi, Thurayya and Hammoudeh, Mohammed and Al Emadi, Samar and Masri, Basel K. and Halabi, Hussein and Badsha, Humeira and Uthman, Imad W. and Saxena, Richa and Padyukov, Leonid and Hirata, Makoto and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Okada, Yukinori},\n",
      "\n",
      "month = mar,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {1569}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{karczewski_mutational_2020,\n",
      "\n",
      "title = {The mutational constraint spectrum quantified from variation in 141,456 humans},\n",
      "\n",
      "volume = {581},\n",
      "\n",
      "copyright = {2020 The Author(s)},\n",
      "\n",
      "issn = {1476-4687},\n",
      "\n",
      "url = {https://www.nature.com/articles/s41586-020-2308-7},\n",
      "\n",
      "doi = {10.1038/s41586-020-2308-7},\n",
      "\n",
      "abstract = {Genetic variants that inactivate protein-coding genes are a powerful source of information about the phenotypic consequences of gene disruption: genes that are crucial for the function of an organism will be depleted of such variants in natural populations, whereas non-essential genes will tolerate their accumulation. However, predicted loss-of-function variants are enriched for annotation errors, and tend to be found at extremely low frequencies, so their analysis requires careful variant annotation and very large sample sizes1. Here we describe the aggregation of 125,748 exomes and 15,708 genomes from human sequencing studies into the Genome Aggregation Database (gnomAD). We identify 443,769 high-confidence predicted loss-of-function variants in this cohort after filtering for artefacts caused by sequencing and annotation errors. Using an improved model of human mutation rates, we classify human protein-coding genes along a spectrum that represents tolerance to inactivation, validate this classification using data from model organisms and engineered human cells, and show that it can be used to improve the power of gene discovery for both common and rare diseases.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {7809},\n",
      "\n",
      "urldate = {2020-06-03},\n",
      "\n",
      "journal = {Nature},\n",
      "\n",
      "author = {Karczewski, Konrad J. and Francioli, Laurent C. and Tiao, Grace and Cummings, Beryl B. and Alföldi, Jessica and Wang, Qingbo and Collins, Ryan L. and Laricchia, Kristen M. and Ganna, Andrea and Birnbaum, Daniel P. and Gauthier, Laura D. and Brand, Harrison and Solomonson, Matthew and Watts, Nicholas A. and Rhodes, Daniel and Singer-Berk, Moriel and England, Eleina M. and Seaby, Eleanor G. and Kosmicki, Jack A. and Walters, Raymond K. and Tashman, Katherine and Farjoun, Yossi and Banks, Eric and Poterba, Timothy and Wang, Arcturus and Seed, Cotton and Whiffin, Nicola and Chong, Jessica X. and Samocha, Kaitlin E. and Pierce-Hoffman, Emma and Zappala, Zachary and O’Donnell-Luria, Anne H. and Minikel, Eric Vallabh and Weisburd, Ben and Lek, Monkol and Ware, James S. and Vittal, Christopher and Armean, Irina M. and Bergelson, Louis and Cibulskis, Kristian and Connolly, Kristen M. and Covarrubias, Miguel and Donnelly, Stacey and Ferriera, Steven and Gabriel, Stacey and Gentry, Jeff and Gupta, Namrata and Jeandet, Thibault and Kaplan, Diane and Llanwarne, Christopher and Munshi, Ruchi and Novod, Sam and Petrillo, Nikelle and Roazen, David and Ruano-Rubio, Valentin and Saltzman, Andrea and Schleicher, Molly and Soto, Jose and Tibbetts, Kathleen and Tolonen, Charlotte and Wade, Gordon and Talkowski, Michael E. and Neale, Benjamin M. and Daly, Mark J. and MacArthur, Daniel G.},\n",
      "\n",
      "month = may,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {434--443}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{tonkin-hill_fast_2019,\n",
      "\n",
      "title = {Fast hierarchical {Bayesian} analysis of population structure},\n",
      "\n",
      "volume = {47},\n",
      "\n",
      "issn = {0305-1048},\n",
      "\n",
      "url = {https://academic.oup.com/nar/article/47/11/5539/5488018},\n",
      "\n",
      "doi = {10.1093/nar/gkz361},\n",
      "\n",
      "abstract = {Abstract.  We present fastbaps, a fast solution to the genetic clustering problem. Fastbaps rapidly identifies an approximate fit to a Dirichlet process mixture},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {11},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Nucleic Acids Research},\n",
      "\n",
      "author = {Tonkin-Hill, Gerry and Lees, John A. and Bentley, Stephen D. and Frost, Simon D. W. and Corander, Jukka},\n",
      "\n",
      "month = jun,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {5539--5549}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{hunter-zinck_genotyping_2020,\n",
      "\n",
      "title = {Genotyping {Array} {Design} and {Data} {Quality} {Control} in the {Million} {Veteran} {Program}},\n",
      "\n",
      "volume = {106},\n",
      "\n",
      "issn = {0002-9297},\n",
      "\n",
      "url = {http://www.sciencedirect.com/science/article/pii/S000292972030080X},\n",
      "\n",
      "doi = {10.1016/j.ajhg.2020.03.004},\n",
      "\n",
      "abstract = {The Million Veteran Program (MVP), initiated by the Department of Veterans Affairs (VA), aims to collect biosamples with consent from at least one million veterans. Presently, blood samples have been collected from over 800,000 enrolled participants. The size and diversity of the MVP cohort, as well as the availability of extensive VA electronic health records, make it a promising resource for precision medicine. MVP is conducting array-based genotyping to provide a genome-wide scan of the entire cohort, in parallel with whole-genome sequencing, methylation, and other ‘omics assays. Here, we present the design and performance of the MVP 1.0 custom Axiom array, which was designed and developed as a single assay to be used across the multi-ethnic MVP cohort. A unified genetic quality-control analysis was developed and conducted on an initial tranche of 485,856 individuals, leading to a high-quality dataset of 459,777 unique individuals. 668,418 genetic markers passed quality control and showed high-quality genotypes not only on common variants but also on rare variants. We confirmed that, with non-European individuals making up nearly 30\\%, MVP’s substantial ancestral diversity surpasses that of other large biobanks. We also demonstrated the quality of the MVP dataset by replicating established genetic associations with height in European Americans and African Americans ancestries. This current dataset has been made available to approved MVP researchers for genome-wide association studies and other downstream analyses. Further data releases will be available for analysis as recruitment at the VA continues and the cohort expands both in size and diversity.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {4},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {The American Journal of Human Genetics},\n",
      "\n",
      "author = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Gong, Xin and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju},\n",
      "\n",
      "month = apr,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "keywords = {GWAS, biobank, genetic ancestry, clinical variants, genetic relatedness, genotype data, Million Veteran Program, quality control, SNP array design, VA},\n",
      "\n",
      "pages = {535--548}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{hunter-zinck_measuring_2020,\n",
      "\n",
      "title = {Measuring genetic variation in the multi-ethnic {Million} {Veteran} {Program} ({MVP})},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/2020.01.06.896613v1},\n",
      "\n",
      "doi = {10.1101/2020.01.06.896613},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}The Million Veteran Program (MVP), initiated by the Department of Veterans Affairs (VA), aims to collect consented biosamples from at least one million Veterans. Presently, blood samples have been collected from over 800,000 enrolled participants. The size and diversity of the MVP cohort, as well as the availability of extensive VA electronic health records make it a promising resource for precision medicine. MVP is conducting array-based genotyping to provide genome-wide scan of the entire cohort, in parallel with whole genome sequencing, methylation, and other omics assays. Here, we present the design and performance of MVP 1.0 custom Axiom$^{\\textrm{®}}$ array, which was designed and developed as a single assay to be used across the multi-ethnic MVP cohort. A unified genetic quality control analysis was developed and conducted on an initial tranche of 485,856 individuals leading to a high-quality dataset of 459,777 unique individuals. 668,418 genetic markers passed quality control and showed high quality genotypes not only on common variants but also on rare variants. We confirmed the substantial ancestral diversity of MVP with nearly 30\\% non-European individuals, surpassing other large biobanks. We also demonstrated the quality of the MVP dataset by replicating established genetic associations with height in European Americans and African Americans ancestries. This current data set has been made available to approved MVP researchers for genome-wide association studies and other downstream analyses. Further data releases will be available for analysis as recruitment at the VA continues and the cohort expands both in size and diversity.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju and Program, on behalf of the VA Million Veteran},\n",
      "\n",
      "month = jan,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {2020.01.06.896613}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{dai_population_2020,\n",
      "\n",
      "title = {Population {Histories} of the {United} {States} {Revealed} through {Fine}-{Scale} {Migration} and {Haplotype} {Analysis}},\n",
      "\n",
      "volume = {106},\n",
      "\n",
      "issn = {00029297},\n",
      "\n",
      "url = {https://linkinghub.elsevier.com/retrieve/pii/S0002929720300446},\n",
      "\n",
      "doi = {10.1016/j.ajhg.2020.02.002},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {3},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {The American Journal of Human Genetics},\n",
      "\n",
      "author = {Dai, Chengzhen L. and Vazifeh, Mohammad M. and Yeang, Chen-Hsiang and Tachet, Remi and Wells, R. Spencer and Vilar, Miguel G. and Daly, Mark J. and Ratti, Carlo and Martin, Alicia R.},\n",
      "\n",
      "month = Mar,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {371--388}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sanchez-barreiro_historical_2020,\n",
      "\n",
      "title = {Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros ({Ceratotherium} simum)},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/2020.05.10.086686v1},\n",
      "\n",
      "doi = {10.1101/2020.05.10.086686},\n",
      "\n",
      "abstract = {{\\textless}p{\\textgreater}Large vertebrates are extremely sensitive to anthropogenic pressure, and their populations are declining fast. The white rhinoceros (Ceratotherium simum) is a paradigmatic case: this African megaherbivore suffered a remarkable population reduction in the last 150 years due to human activities. The two white rhinoceros subspecies, the northern (NWR) and the southern white rhinoceros (SWR), however, underwent opposite fates: the NWR vanished quickly after the onset of the decline, while the SWR recovered after a severe bottleneck. Such demographic events are predicted to have an erosive effect at the genomic level, in connection with the extirpation of diversity, and increased genetic drift and inbreeding. However there is currently little empirical data available that allows us to directly reconstruct the subtleties of such processes in light of distinct demographic histories. Therefore to assess these effects, we generated a whole-genome, temporal dataset consisting of 52 re-sequenced white rhinoceros genomes, that represents both subspecies at two time windows: before and during/after the bottleneck. Our data not only reveals previously unknown population substructure within both subspecies, but allowed us to quantify the genomic erosion undergone by both, with post-bottleneck white rhinoceroses harbouring significantly fewer heterozygous sites, and showing higher inbreeding coefficients than pre-bottleneck individuals. Moreover, the effective population size suffered a decrease of two and three orders of magnitude in the NWR and SWR respectively, due to the recent bottleneck. Our data therefore provides much needed empirical support for theoretical predictions about the genomic consequences of shrinking populations, information that is relevant for understanding the process of population extinction. Furthermore, our findings have the potential to inform management approaches for the conservation of the remaining white rhinoceroses.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Sánchez-Barreiro, Fátima and Gopalakrishnan, Shyam and Ramos-Madrigal, Jazmín and Westbury, Michael V. and Manuel, Marc de and Margaryan, Ashot and Ciucani, Marta M. and Vieira, Filipe G. and Patramanis, Yannis and Kalthoff, Daniela C. and Timmons, Zena and Sicheritz-Pontén, Thomas and Dalén, Love and Ryder, Oliver A. and Zhang, Guojie and Marquès-Bonet, Tomás and Moodley, Yoshan and Gilbert, M. Thomas P.},\n",
      "\n",
      "month = may,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {2020.05.10.086686}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{almarri_population_2020,\n",
      "\n",
      "title = {Population {Structure}, {Stratification} and {Introgression} of {Human} {Structural} {Variation}},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/746172v2},\n",
      "\n",
      "doi = {10.1101/746172},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}Structural variants contribute substantially to genetic diversity and are important evolutionarily and medically, yet are still understudied. Here, we present a comprehensive analysis of deletions, duplications, insertions, inversions and non-reference unique insertions in the Human Genome Diversity Project (HGDP-CEPH) panel, a high-coverage dataset of 911 samples from 54 diverse worldwide populations. We identify in total 126,018 structural variants (25,588 \\&lt;100 bp in size), of which 78\\% are novel. Some reach high frequency and are private to continental groups or even individual populations, including a deletion in the maltase-glucoamylase gene \\textit{MGAM} involved in starch digestion, in the South American Karitiana and a deletion in the Central African Mbuti in \\textit{SIGLEC5,} potentially leading to immune hyperactivity. We discover a dynamic range of copy number expansions and find cases of regionally-restricted runaway duplications, for example, 18 copies near the olfactory receptor \\textit{OR7D2} in East Asia and in the clinically-relevant \\textit{HCAR2} in Central Asia. We identify highly-stratified putatively introgressed variants from Neanderthals or Denisovans, some of which, like a deletion within \\textit{AQR} in Papuans, are almost fixed in individual populations. Finally, by \\textit{de novo} assembly of 25 genomes using linked-read sequencing we discover 1631 breakpoint-resolved unique insertions, in aggregate accounting for 1.9 Mb of sequence absent from the GRCh38 reference. These insertions show population structure and some reside in functional regions, illustrating the limitation of a single human reference and the need for high-quality genomes from diverse populations to fully discover and understand human genetic variation.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Almarri, Mohamed A. and Bergström, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S. and Chen, Yuan and Hurles, Matthew E. and Tyler-Smith, Chris and Xue, Yali},\n",
      "\n",
      "month = jan,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {746172}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{almarri2020population,\n",
      "\n",
      "title={Population Structure, Stratification, and Introgression of Human Structural Variation},\n",
      "\n",
      "author={Almarri, Mohamed A and Bergstr{\\\"o}m, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S and Chen, Yuan and Hurles, Matthew E and Tyler-Smith, Chris and Xue, Yali},\n",
      "\n",
      "journal={Cell},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "publisher={Elsevier}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{belbin_towards_2019,\n",
      "\n",
      "title = {Towards a fine-scale population health monitoring system},\n",
      "\n",
      "copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/780668v1},\n",
      "\n",
      "doi = {10.1101/780668},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}Understanding population health disparities is an essential component of equitable precision health efforts. Epidemiology research often relies on definitions of race and ethnicity, but these population labels may not adequately capture disease burdens specific to sub-populations. Here we propose a framework for repurposing data from Electronic Health Records (EHRs) in concert with genomic data to explore enrichment of disease within sub-populations. Using data from a diverse biobank in New York City, we genetically identified 17 sub-populations, and noted the presence of genetic founder effects in 7. By then linking community membership to the EHR, we were able to identify over 600 health outcomes that were statistically enriched within a specific population, with many representing known associations, and many others being novel. This work reinforces the utility of linking genomic data to EHRs, and provides a framework towards fine-scale monitoring of population health.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},\n",
      "\n",
      "month = sep,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {780668}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{margaryan_population_2019,\n",
      "\n",
      "title = {Population genomics of the {Viking} world},\n",
      "\n",
      "copyright = {© 2019, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial-NoDerivs 4.0 International), CC BY-NC-ND 4.0, as described at http://creativecommons.org/licenses/by-nc-nd/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/703405v1},\n",
      "\n",
      "doi = {10.1101/703405},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}The Viking maritime expansion from Scandinavia (Denmark, Norway, and Sweden) marks one of the swiftest and most far-flung cultural transformations in global history. During this time (c. 750 to 1050 CE), the Vikings reached most of western Eurasia, Greenland, and North America, and left a cultural legacy that persists till today. To understand the genetic structure and influence of the Viking expansion, we sequenced the genomes of 442 ancient humans from across Europe and Greenland ranging from the Bronze Age (c. 2400 BC) to the early Modern period (c. 1600 CE), with particular emphasis on the Viking Age. We find that the period preceding the Viking Age was accompanied by foreign gene flow into Scandinavia from the south and east: spreading from Denmark and eastern Sweden to the rest of Scandinavia. Despite the close linguistic similarities of modern Scandinavian languages, we observe genetic structure within Scandinavia, suggesting that regional population differences were already present 1,000 years ago. We find evidence for a majority of Danish Viking presence in England, Swedish Viking presence in the Baltic, and Norwegian Viking presence in Ireland, Iceland, and Greenland. Additionally, we see substantial foreign European ancestry entering Scandinavia during the Viking Age. We also find that several of the members of the only archaeologically well-attested Viking expedition were close family members. By comparing Viking Scandinavian genomes with present-day Scandinavian genomes, we find that pigmentation-associated loci have undergone strong population differentiation during the last millennia. Finally, we are able to trace the allele frequency dynamics of positively selected loci with unprecedented detail, including the lactase persistence allele and various alleles associated with the immune response. We conclude that the Viking diaspora was characterized by substantial foreign engagement: distinct Viking populations influenced the genomic makeup of different regions of Europe, while Scandinavia also experienced increased contact with the rest of the continent.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Margaryan, Ashot and Lawson, Daniel and Sikora, Martin and Racimo, Fernando and Rasmussen, Simon and Moltke, Ida and Cassidy, Lara and Jørsboe, Emil and Ingason, Andrés and Pedersen, Mikkel and Korneliussen, Thorfinn and Wilhelmson, Helene and Buś, Magdalena and Damgaard, Peter de Barros and Martiniano, Rui and Renaud, Gabriel and Bhérer, Claude and Moreno-Mayar, J. Víctor and Fotakis, Anna and Allen, Marie and Molak, Martyna and Cappellini, Enrico and Scorrano, Gabriele and Buzhilova, Alexandra and Fox, Allison and Albrechtsen, Anders and Schütz, Berit and Skar, Birgitte and Arcini, Caroline and Falys, Ceri and Jonson, Charlotte Hedenstierna and Błaszczyk, Dariusz and Pezhemsky, Denis and Turner-Walker, Gordon and Gestsdóttir, Hildur and Lundstrøm, Inge and Gustin, Ingrid and Mainland, Ingrid and Potekhina, Inna and Muntoni, Italo and Cheng, Jade and Stenderup, Jesper and Ma, Jilong and Gibson, Julie and Peets, Jüri and Gustafsson, Jörgen and Iversen, Katrine and Simpson, Linzi and Strand, Lisa and Loe, Louise and Sikora, Maeve and Florek, Marek and Vretemark, Maria and Redknap, Mark and Bajka, Monika and Pushkina, Tamara and Søvsø, Morten and Grigoreva, Natalia and Christensen, Tom and Kastholm, Ole and Uldum, Otto and Favia, Pasquale and Holck, Per and Allmäe, Raili and Sten, Sabine and Arge, Símun and Ellingvåg, Sturla and Moiseyev, Vayacheslav and Bogdanowicz, Wiesław and Magnusson, Yvonne and Orlando, Ludovic and Bradley, Daniel and Jørkov, Marie Louise and Arneborg, Jette and Lynnerup, Niels and Price, Neil and Gilbert, M. Thomas and Allentoft, Morten and Bill, Jan and Sindbæk, Søren and Hedeager, Lotte and Kristiansen, Kristian and Nielsen, Rasmus and Werge, Thomas and Willerslev, Eske},\n",
      "\n",
      "month = jul,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {703405}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{consortium_genome_2020,\n",
      "\n",
      "title = {Genome variation and population structure among 1,142 mosquitoes of the {African} malaria vector species {Anopheles} gambiae and {Anopheles} coluzzii},\n",
      "\n",
      "copyright = {© 2020, Posted by Cold Spring Harbor Laboratory. This pre-print is available under a Creative Commons License (Attribution-NonCommercial 4.0 International), CC BY-NC 4.0, as described at http://creativecommons.org/licenses/by-nc/4.0/},\n",
      "\n",
      "url = {https://www.biorxiv.org/content/10.1101/864314v2},\n",
      "\n",
      "doi = {10.1101/864314},\n",
      "\n",
      "abstract = {{\\textless}h3{\\textgreater}Abstract{\\textless}/h3{\\textgreater} {\\textless}p{\\textgreater}Mosquito control remains a central pillar of efforts to reduce malaria burden in sub-Saharan Africa. However, insecticide resistance is entrenched in malaria vector populations, and countries with high malaria burden face a daunting challenge to sustain malaria control with a limited set of surveillance and intervention tools. Here we report on the second phase of a project to build an open resource of high quality data on genome variation among natural populations of the major African malaria vector species \\textit{Anopheles gambiae} and \\textit{Anopheles coluzzii}. We analysed whole genomes of 1,142 individual mosquitoes sampled from the wild in 13 African countries, and a further 234 individuals comprising parents and progeny of 11 lab crosses. The data resource includes high confidence single nucleotide polymorphism (SNP) calls at 57 million variable sites, genome-wide copy number variation (CNV) calls, and haplotypes phased at biallelic SNPs. We used these data to analyse genetic population structure, and characterise genetic diversity within and between populations. We also illustrate the utility of these data by investigating species differences in isolation by distance, genetic variation within proposed gene drive target sequences, and patterns of resistance to pyrethroid insecticides. This data resource provides a foundation for developing new operational systems for molecular surveillance, and for accelerating research and development of new vector control tools.{\\textless}/p{\\textgreater}},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {bioRxiv},\n",
      "\n",
      "author = {Consortium, The Anopheles gambiae 1000 Genomes and Clarkson, Chris S. and Miles, Alistair and Harding, Nicholas J. and Lucas, Eric R. and Battey, C. J. and Amaya-Romero, Jorge Edouardo and Cano, Jorge and Diabate, Abdoulaye and Constant, Edi and Nwakanma, Davis C. and Jawara, Musa and Essandoh, John and Dinis, Joao and Goff, Gilbert Le and Robert, Vincent and Troco, Arlete D. and Costantini, Carlo and Rohatgi, Kyanne R. and Elissa, Nohal and Coulibaly, Boubacar and Midega, Janet and Mbogo, Charles and Mawejje, Henry D. and Stalker, Jim and Rockett, Kirk A. and Drury, Eleanor and Mead, Daniel and Jeffreys, Anna E. and Hubbart, Christina and Rowlands, Kate and Isaacs, Alison T. and Jyothi, Dushyanth and Malangone, Cinzia and Kamali, Maryam and Henrichs, Christa and Simpson, Victoria and Ayala, Diego and Besansky, Nora J. and Burt, Austin and Caputo, Beniamino and Torre, Alessandra della and Fontaine, Michael and Godfray, H. Charles J. and Hahn, Matthew W. and Kern, Andrew D. and Lawniczak, Mara K. N. and O’Loughlin, Samantha and Pinto, Joao and Riehle, Michelle M. and Sharakhov, Igor and Schrider, Daniel R. and Vernick, Kenneth D. and White, Bradley J. and Donnelly, Martin J. and Kwiatkowski, Dominic P.},\n",
      "\n",
      "month = feb,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {864314}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{greenbaum_network-based_2019,\n",
      "\n",
      "title = {Network-based hierarchical population structure analysis for large genomic data sets},\n",
      "\n",
      "volume = {29},\n",
      "\n",
      "issn = {1088-9051, 1549-5469},\n",
      "\n",
      "url = {http://genome.cshlp.org/content/29/12/2020},\n",
      "\n",
      "doi = {10.1101/gr.250092.119},\n",
      "\n",
      "abstract = {Analysis of population structure in natural populations using genetic data is a common practice in ecological and evolutionary studies. With large genomic data sets of populations now appearing more frequently across the taxonomic spectrum, it is becoming increasingly possible to reveal many hierarchical levels of structure, including fine-scale genetic clusters. To analyze these data sets, methods need to be appropriately suited to the challenges of extracting multilevel structure from whole-genome data. Here, we present a network-based approach for constructing population structure representations from genetic data. The use of community-detection algorithms from network theory generates a natural hierarchical perspective on the representation that the method produces. The method is computationally efficient, and it requires relatively few assumptions regarding the biological processes that underlie the data. We show the approach by analyzing population structure in the model plant species Arabidopsis thaliana and in human populations. These examples illustrate how network-based approaches for population structure analysis are well-suited to extracting valuable ecological and evolutionary information in the era of large genomic data sets.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {12},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Genome {R}esearch},\n",
      "\n",
      "author = {Greenbaum, Gili and Rubin, Amir and Templeton, Alan R. and Rosenberg, Noah A.},\n",
      "\n",
      "month = jan,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pmid = {31694865},\n",
      "\n",
      "pages = {2020--2033}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{yamamoto_genetic_2020,\n",
      "\n",
      "title = {Genetic and phenotypic landscape of the mitochondrial genome in the {Japanese} population},\n",
      "\n",
      "volume = {3},\n",
      "\n",
      "copyright = {2020 The Author(s)},\n",
      "\n",
      "issn = {2399-3642},\n",
      "\n",
      "url = {https://www.nature.com/articles/s42003-020-0812-9},\n",
      "\n",
      "doi = {10.1038/s42003-020-0812-9},\n",
      "\n",
      "abstract = {The genetic landscape of mitochondrial DNA (mtDNA) has been elusive. By analyzing mtDNA using the whole genome sequence (WGS) of Japanese individuals (n = 1928), we identified 2023 mtDNA variants and high-resolution haplogroups. Frequency spectra of the haplogroups were population-specific and were heterogeneous among geographic regions within Japan. Application of machine learning methods could finely classify the subjects corresponding to the high-digit mtDNA sub-haplogroups. mtDNA had distinct genetic structures from that of nuclear DNA (nDNA), characterized by no distance-dependent linkage disequilibrium decay, sparse tagging of common variants, and the existence of common haplotypes spanning the entire mtDNA. We did not detect any evidence of mtDNA–nDNA (or mtDNA copy number–nDNA) genotype associations. Together with WGS-based mtDNA variant imputation, we conducted a phenome-wide association study of 147,437 Japanese individuals with 99 clinical phenotypes. We observed pleiotropy of mtDNA genetic risk on the five late-onset human complex traits including creatine kinase (P = 1.7 × 10−12).},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {1},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Communications Biology},\n",
      "\n",
      "author = {Yamamoto, Kenichi and Sakaue, Saori and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Ozono, Keiichi and Momozawa, Yukihide and Okada, Yukinori},\n",
      "\n",
      "month = mar,\n",
      "\n",
      "year = {2020},\n",
      "\n",
      "pages = {1--11}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{derkarabetian_demonstration_2019,\n",
      "\n",
      "title = {A demonstration of unsupervised machine learning in species delimitation},\n",
      "\n",
      "volume = {139},\n",
      "\n",
      "issn = {1055-7903},\n",
      "\n",
      "url = {http://www.sciencedirect.com/science/article/pii/S1055790319301721},\n",
      "\n",
      "doi = {10.1016/j.ympev.2019.106562},\n",
      "\n",
      "abstract = {One major challenge to delimiting species with genetic data is successfully differentiating population structure from species-level divergence, an issue exacerbated in taxa inhabiting naturally fragmented habitats. Many fields of science are now using machine learning, and in evolutionary biology supervised machine learning has recently been used to infer species boundaries. These supervised methods require training data with associated labels. Conversely, unsupervised machine learning (UML) uses inherent data structure and does not require user-specified training labels, potentially providing more objectivity in species delimitation. In the context of integrative taxonomy, we demonstrate the utility of three UML approaches (random forests, variational autoencoders, t-distributed stochastic neighbor embedding) for species delimitation in an arachnid taxon with high population genetic structure (Opiliones, Laniatores, Metanonychus). We find that UML approaches successfully cluster samples according to species-level divergences and not high levels of population structure, while model-based validation methods severely over-split putative species. UML offers intuitive data visualization in two-dimensional space, the ability to accommodate various data types, and has potential in many areas of systematic and evolutionary biology. We argue that machine learning methods are ideally suited for species delimitation and may perform well in many natural systems and across taxa with diverse biological characteristics.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Molecular Phylogenetics and Evolution},\n",
      "\n",
      "author = {Derkarabetian, Shahan and Castillo, Stephanie and Koo, Peter K. and Ovchinnikov, Sergey and Hedin, Marshal},\n",
      "\n",
      "month = oct,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "keywords = {t-SNE, Integrative taxonomy, Opiliones, Random forest, Ultraconserved elements, Variational autoencoders},\n",
      "\n",
      "pages = {106562}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{diaz-papkovich_umap_2019,\n",
      "\n",
      "title = {{UMAP} reveals cryptic population structure and phenotype heterogeneity in large genomic cohorts},\n",
      "\n",
      "volume = {15},\n",
      "\n",
      "issn = {1553-7390},\n",
      "\n",
      "url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853336/},\n",
      "\n",
      "doi = {10.1371/journal.pgen.1008432},\n",
      "\n",
      "abstract = {Human populations feature both discrete and continuous patterns of variation. Current analysis approaches struggle to jointly identify these patterns because of modelling assumptions, mathematical constraints, or numerical challenges. Here we apply uniform manifold approximation and projection (UMAP), a non-linear dimension reduction tool, to three well-studied genotype datasets and discover overlooked subpopulations within the American Hispanic population, fine-scale relationships between geography, genotypes, and phenotypes in the UK population, and cryptic structure in the Thousand Genomes Project data. This approach is well-suited to the influx of large and diverse data and opens new lines of inquiry in population-scale datasets., The demographic history of human populations features varying geographic and social barriers to mating. Over time, these barriers have led to varying levels of genetic relatedness among individuals. This population structure is informative about human history, and can have a significant impact on studies of medical genetics. Because population structure depends on myriad demographic, ecological, and social forces, a priori visualization is useful to identify subtle patterns of population structure. We use a dimension reduction method—UMAP—to visualize population structure in three genomic datasets and find previously unobserved patterns, revealing fine-scale population structure and illustrating differences between groups in traits such as white blood cell count, height, and FEV1, a measure of lung function. Using UMAP is computationally efficient and can identify fine-scale population structure in large population datasets. We find it particularly useful to reveal phenotypic variation among genetically related populations, and recommend it is a complement to principal component analysis in primary data visualization.},\n",
      "\n",
      "number = {11},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {PLoS Genetics},\n",
      "\n",
      "author = {Diaz-Papkovich, Alex and Anderson-Trocmé, Luke and Ben-Eghan, Chief and Gravel, Simon},\n",
      "\n",
      "month = nov,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pmid = {31675358},\n",
      "\n",
      "pmcid = {PMC6853336}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{li_past_2020,\n",
      "\n",
      "title = {Past demography and local adaptation in forest trees : {Insights} from natural populations and breeding programs of {Norway} spruce},\n",
      "\n",
      "shorttitle = {Past demography and local adaptation in forest trees},\n",
      "\n",
      "url = {http://urn.kb.se/resolve?urn=urn:nbn:se:uu:diva-403450},\n",
      "\n",
      "abstract = {DiVA portal is a finding tool for research publications and student theses written at the following 49 universities and research institutions.},\n",
      "\n",
      "language = {eng},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "author = {Li, Lili},\n",
      "\n",
      "year = {2020}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{becht_dimensionality_2019,\n",
      "\n",
      "title = {Dimensionality reduction for visualizing single-cell data using {UMAP}},\n",
      "\n",
      "volume = {37},\n",
      "\n",
      "copyright = {2018 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.},\n",
      "\n",
      "issn = {1546-1696},\n",
      "\n",
      "url = {https://www.nature.com/articles/nbt.4314},\n",
      "\n",
      "doi = {10.1038/nbt.4314},\n",
      "\n",
      "abstract = {A benchmarking analysis on single-cell RNA-seq and mass cytometry data reveals the best-performing technique for dimensionality reduction.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {1},\n",
      "\n",
      "urldate = {2020-05-21},\n",
      "\n",
      "journal = {Nature Biotechnology},\n",
      "\n",
      "author = {Becht, Etienne and McInnes, Leland and Healy, John and Dutertre, Charles-Antoine and Kwok, Immanuel W. H. and Ng, Lai Guan and Ginhoux, Florent and Newell, Evan W.},\n",
      "\n",
      "month = jan,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {38--44}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{kobak_art_2019,\n",
      "\n",
      "title = {The art of using t-{SNE} for single-cell transcriptomics},\n",
      "\n",
      "volume = {10},\n",
      "\n",
      "copyright = {2019 The Author(s)},\n",
      "\n",
      "issn = {2041-1723},\n",
      "\n",
      "url = {https://www.nature.com/articles/s41467-019-13056-x},\n",
      "\n",
      "doi = {10.1038/s41467-019-13056-x},\n",
      "\n",
      "abstract = {Single-cell transcriptomics yields ever growing data sets containing RNA expression levels for thousands of genes from up to millions of cells. Common data analysis pipelines include a dimensionality reduction step for visualising the data in two dimensions, most frequently performed using t-distributed stochastic neighbour embedding (t-SNE). It excels at revealing local structure in high-dimensional data, but naive applications often suffer from severe shortcomings, e.g. the global structure of the data is not represented accurately. Here we describe how to circumvent such pitfalls, and develop a protocol for creating more faithful t-SNE visualisations. It includes PCA initialisation, a high learning rate, and multi-scale similarity kernels; for very large data sets, we additionally use exaggeration and downsampling-based initialisation. We use published single-cell RNA-seq data sets to demonstrate that this protocol yields superior results compared to the naive application of t-SNE.},\n",
      "\n",
      "language = {en},\n",
      "\n",
      "number = {1},\n",
      "\n",
      "urldate = {2020-06-02},\n",
      "\n",
      "journal = {Nature Communications},\n",
      "\n",
      "author = {Kobak, Dmitry and Berens, Philipp},\n",
      "\n",
      "month = nov,\n",
      "\n",
      "year = {2019},\n",
      "\n",
      "pages = {5416}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{spear2020recent,\n",
      "\n",
      "title={Recent fluctuations in Mexican American genomes have altered the genetic architecture of biomedical traits},\n",
      "\n",
      "author={Spear, Melissa L and Diaz-Papkovich, Alex and Ziv, Elad and Gravel, Simon and Torgerson, Dara G and Hernandez, Ryan},\n",
      "\n",
      "journal={bioRxiv},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "publisher={Cold Spring Harbor Laboratory}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{patterson2006population,\n",
      "\n",
      "title={Population structure and eigenanalysis},\n",
      "\n",
      "author={Patterson, Nick and Price, Alkes L and Reich, David},\n",
      "\n",
      "journal={PLoS Genetics},\n",
      "\n",
      "volume={2},\n",
      "\n",
      "number={12},\n",
      "\n",
      "pages={e190},\n",
      "\n",
      "year={2006},\n",
      "\n",
      "publisher={Public Library of Science},\n",
      "\n",
      "doi={10.1371/journal.pgen.0020190}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{10002015global,\n",
      "\n",
      "title={A global reference for human genetic variation},\n",
      "\n",
      "author={1000 Genomes Project Consortium and others},\n",
      "\n",
      "journal={Nature},\n",
      "\n",
      "volume={526},\n",
      "\n",
      "number={7571},\n",
      "\n",
      "pages={68--74},\n",
      "\n",
      "year={2015},\n",
      "\n",
      "publisher={Nature Publishing Group}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{cann2002human,\n",
      "\n",
      "title={A human genome diversity cell line panel},\n",
      "\n",
      "author={Cann, Howard M and De Toma, Claudia and Cazes, Lucien and Legrand, Marie-Fernande and Morel, Valerie and Piouffre, Laurence and Bodmer, Julia and Bodmer, Walter F and Bonne-Tamir, Batsheva and Cambon-Thomsen, Anne and others},\n",
      "\n",
      "journal={Science},\n",
      "\n",
      "volume={296},\n",
      "\n",
      "number={5566},\n",
      "\n",
      "pages={261--262},\n",
      "\n",
      "year={2002},\n",
      "\n",
      "publisher={American Association for the Advancement of Science}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{matise2011next,\n",
      "\n",
      "title={The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study},\n",
      "\n",
      "author={Matise, Tara C and Ambite, Jose Luis and Buyske, Steven and Carlson, Christopher S and Cole, Shelley A and Crawford, Dana C and Haiman, Christopher A and Heiss, Gerardo and Kooperberg, Charles and Marchand, Loic Le and others},\n",
      "\n",
      "journal={American journal of epidemiology},\n",
      "\n",
      "volume={174},\n",
      "\n",
      "number={7},\n",
      "\n",
      "pages={849--859},\n",
      "\n",
      "year={2011},\n",
      "\n",
      "publisher={Oxford University Press}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{nelson2008population,\n",
      "\n",
      "title={The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research},\n",
      "\n",
      "author={Nelson, Matthew R and Bryc, Katarzyna and King, Karen S and Indap, Amit and Boyko, Adam R and Novembre, John and Briley, Linda P and Maruyama, Yuka and Waterworth, Dawn M and Waeber, G{\\'e}rard and others},\n",
      "\n",
      "journal={The American Journal of Human Genetics},\n",
      "\n",
      "volume={83},\n",
      "\n",
      "number={3},\n",
      "\n",
      "pages={347--358},\n",
      "\n",
      "year={2008},\n",
      "\n",
      "publisher={Elsevier}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{juster1995overview,\n",
      "\n",
      "title={An overview of the Health and Retirement Study},\n",
      "\n",
      "author={Juster, F Thomas and Suzman, Richard},\n",
      "\n",
      "journal={Journal of Human Resources},\n",
      "\n",
      "pages={S7--S56},\n",
      "\n",
      "year={1995},\n",
      "\n",
      "publisher={JSTOR}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sudlow2015uk,\n",
      "\n",
      "title={UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age},\n",
      "\n",
      "author={Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and others},\n",
      "\n",
      "journal={Plos med},\n",
      "\n",
      "volume={12},\n",
      "\n",
      "number={3},\n",
      "\n",
      "pages={e1001779},\n",
      "\n",
      "year={2015},\n",
      "\n",
      "publisher={Public Library of Science}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sorlie2010design,\n",
      "\n",
      "title={Design and implementation of the Hispanic community health study/study of Latinos},\n",
      "\n",
      "author={Sorlie, Paul D and Avil{\\'e}s-Santa, Larissa M and Wassertheil-Smoller, Sylvia and Kaplan, Robert C and Daviglus, Martha L and Giachello, Aida L and Schneiderman, Neil and Raij, Leopoldo and Talavera, Gregory and Allison, Matthew and others},\n",
      "\n",
      "journal={Annals of epidemiology},\n",
      "\n",
      "volume={20},\n",
      "\n",
      "number={8},\n",
      "\n",
      "pages={629--641},\n",
      "\n",
      "year={2010},\n",
      "\n",
      "publisher={Elsevier}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{nagai2017overview,\n",
      "\n",
      "title={Overview of the {BioBank Japan Project}: study design and profile},\n",
      "\n",
      "author={Nagai, Akiko and Hirata, Makoto and Kamatani, Yoichiro and Muto, Kaori and Matsuda, Koichi and Kiyohara, Yutaka and Ninomiya, Toshiharu and Tamakoshi, Akiko and Yamagata, Zentaro and Mushiroda, Taisei and others},\n",
      "\n",
      "journal={Journal of epidemiology},\n",
      "\n",
      "volume={27},\n",
      "\n",
      "number={Supplement\\_III},\n",
      "\n",
      "pages={S2--S8},\n",
      "\n",
      "year={2017},\n",
      "\n",
      "publisher={Japan Epidemiological Association}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{lawson2012inference,\n",
      "\n",
      "title={Inference of population structure using dense haplotype data},\n",
      "\n",
      "author={Lawson, Daniel John and Hellenthal, Garrett and Myers, Simon and Falush, Daniel},\n",
      "\n",
      "journal={PLoS Genetics},\n",
      "\n",
      "volume={8},\n",
      "\n",
      "number={1},\n",
      "\n",
      "pages={e1002453},\n",
      "\n",
      "year={2012},\n",
      "\n",
      "publisher={Public Library of Science}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{alexander2009fast,\n",
      "\n",
      "title={Fast model-based estimation of ancestry in unrelated individuals},\n",
      "\n",
      "author={Alexander, David H and Novembre, John and Lange, Kenneth},\n",
      "\n",
      "journal={Genome {R}esearch},\n",
      "\n",
      "volume={19},\n",
      "\n",
      "number={9},\n",
      "\n",
      "pages={1655--1664},\n",
      "\n",
      "year={2009},\n",
      "\n",
      "publisher={Cold Spring Harbor Lab}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@Article{Hunter2007,\n",
      "\n",
      "Author    = {Hunter, J. D.},\n",
      "\n",
      "Title     = {Matplotlib: A 2D graphics environment},\n",
      "\n",
      "Journal   = {Computing In Science \\& Engineering},\n",
      "\n",
      "Volume    = {9},\n",
      "\n",
      "Number    = {3},\n",
      "\n",
      "Pages     = {90--95},\n",
      "\n",
      "abstract  = {Matplotlib is a 2D graphics package used for Python\n",
      "\n",
      "for application development, interactive scripting, and\n",
      "\n",
      "publication-quality image generation across user\n",
      "\n",
      "interfaces and operating systems.},\n",
      "\n",
      "publisher = {IEEE COMPUTER SOC},\n",
      "\n",
      "doi       = {10.1109/MCSE.2007.55},\n",
      "\n",
      "year      = 2007\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{scikit-learn,\n",
      "\n",
      "title={Scikit-learn: Machine Learning in {P}ython},\n",
      "\n",
      "author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.\n",
      "\n",
      "and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.\n",
      "\n",
      "and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and\n",
      "\n",
      "Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},\n",
      "\n",
      "journal={Journal of Machine Learning Research},\n",
      "\n",
      "volume={12},\n",
      "\n",
      "pages={2825--2830},\n",
      "\n",
      "year={2011}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{kerminen2019geographic,\n",
      "\n",
      "title={Geographic variation and bias in the polygenic scores of complex diseases and traits in {F}inland},\n",
      "\n",
      "author={Kerminen, Sini and Martin, Alicia R and Koskela, Jukka and Ruotsalainen, Sanni E and Havulinna, Aki S and Surakka, Ida and Palotie, Aarno and Perola, Markus and Salomaa, Veikko and Daly, Mark J and others},\n",
      "\n",
      "journal={The American Journal of Human Genetics},\n",
      "\n",
      "volume={104},\n",
      "\n",
      "number={6},\n",
      "\n",
      "pages={1169--1181},\n",
      "\n",
      "year={2019},\n",
      "\n",
      "publisher={Elsevier}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{berg2019reduced,\n",
      "\n",
      "title={Reduced signal for polygenic adaptation of height in {UK} {B}iobank},\n",
      "\n",
      "author={Berg, Jeremy J and Harpak, Arbel and Sinnott-Armstrong, Nasa and Joergensen, Anja Moltke and Mostafavi, Hakhamanesh and Field, Yair and Boyle, Evan August and Zhang, Xinjun and Racimo, Fernando and Pritchard, Jonathan K and others},\n",
      "\n",
      "journal={e{L}ife},\n",
      "\n",
      "volume={8},\n",
      "\n",
      "pages={e39725},\n",
      "\n",
      "year={2019},\n",
      "\n",
      "publisher={eLife Sciences Publications Limited}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sohail2019polygenic,\n",
      "\n",
      "title={Polygenic adaptation on height is overestimated due to uncorrected stratification in genome-wide association studies},\n",
      "\n",
      "author={Sohail, Mashaal and Maier, Robert M and Ganna, Andrea and Bloemendal, Alex and Martin, Alicia R and Turchin, Michael C and Chiang, Charleston WK and Hirschhorn, Joel and Daly, Mark J and Patterson, Nick and others},\n",
      "\n",
      "journal={e{L}ife},\n",
      "\n",
      "volume={8},\n",
      "\n",
      "pages={e39702},\n",
      "\n",
      "year={2019},\n",
      "\n",
      "publisher={eLife Sciences Publications Limited}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcvean2009genealogical,\n",
      "\n",
      "title={A genealogical interpretation of principal components analysis},\n",
      "\n",
      "author={McVean, Gil},\n",
      "\n",
      "journal={PLoS Genetics},\n",
      "\n",
      "volume={5},\n",
      "\n",
      "number={10},\n",
      "\n",
      "pages={e1000686},\n",
      "\n",
      "year={2009},\n",
      "\n",
      "publisher={Public Library of Science},\n",
      "\n",
      "doi = {10.1371/journal.pgen.1000686}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{moon2019visualizing,\n",
      "\n",
      "title={Visualizing structure and transitions in high-dimensional biological data},\n",
      "\n",
      "author={Moon, Kevin R and van Dijk, David and Wang, Zheng and Gigante, Scott and Burkhardt, Daniel B and Chen, William S and Yim, Kristina and van den Elzen, Antonia and Hirn, Matthew J and Coifman, Ronald R and others},\n",
      "\n",
      "journal={Nature {B}iotechnology},\n",
      "\n",
      "volume={37},\n",
      "\n",
      "number={12},\n",
      "\n",
      "pages={1482--1492},\n",
      "\n",
      "year={2019},\n",
      "\n",
      "publisher={Nature Publishing Group}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{micheletti_genetic_2020,\n",
      "\n",
      "title = {Genetic {Consequences} of the {Transatlantic} {Slave} {Trade} in the {Americas}},\n",
      "\n",
      "volume = {0},\n",
      "\n",
      "issn = {0002-9297, 1537-6605},\n",
      "\n",
      "url = {https://www.cell.com/ajhg/abstract/S0002-9297(20)30200-7},\n",
      "\n",
      "doi = {10.1016/j.ajhg.2020.06.012},\n",
      "\n",
      "language = {English},\n",
      "\n",
      "number = {0},\n",
      "\n",
      "journal = {The American Journal of Human Genetics},\n",
      "\n",
      "author = {Micheletti, Steven J. and Bryc, Kasia and Esselmann, Samantha G. Ancona and Freyman, William A. and Moreno, Meghan E. and Poznik, G. David and Shastri, Anjali J. and Agee, M. and Aslibekyan, S. and Auton, A. and Bell, R. and Clark, S. and Das, S. and Elson, S. and Fletez-Brant, K. and Fontanillas, P. and Gandhi, P. and Heilbron, K. and Hicks, B. and Hinds, D. and Huber, K. and Jewett, E. and Jiang, Y. and Kleinman, A. and Lin, K. and Litterman, N. and McCreight, J. and McIntyre, M. and McManus, K. and Mozaffari, S. and Nandakumar, P. and Noblin, L. and Northover, C. and O’Connell, J. and Petrakovitz, A. and Pitts, S. and Shelton, J. and Shringarpure, S. and Tian, C. and Tung, J. and Tunney, R. and Vacic, V. and Wang, X. and Zare, A. and Beleza, Sandra and Mountain, Joanna L.},\n",
      "\n",
      "month = jul,\n",
      "\n",
      "year = {2020}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@book{holmes2018modern,\n",
      "\n",
      "title={Modern statistics for modern biology},\n",
      "\n",
      "author={Holmes, Susan and Huber, Wolfgang},\n",
      "\n",
      "year={2018},\n",
      "\n",
      "chapter={Introduction},\n",
      "\n",
      "publisher={Cambridge University Press}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@misc{genizon,\n",
      "\n",
      "title = {Genizon Biobank},\n",
      "\n",
      "publisher = {Genome Qu\\'{e}bec},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "url={http://www.genomequebec.com/genizon-biobank/}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcinnes2018software,\n",
      "\n",
      "title={{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection}},\n",
      "\n",
      "author={McInnes, Leland and Healy, John and Saul, Nathaniel and Grossberger, Lukas},\n",
      "\n",
      "journal={The Journal of Open Source Software},\n",
      "\n",
      "volume={3},\n",
      "\n",
      "number={29},\n",
      "\n",
      "pages={861},\n",
      "\n",
      "year={2018}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mathieson2020ancestry,\n",
      "\n",
      "title={What is ancestry?},\n",
      "\n",
      "author={Mathieson, Iain and Scally, Aylwyn},\n",
      "\n",
      "journal={PLoS Genetics},\n",
      "\n",
      "volume={16},\n",
      "\n",
      "number={3},\n",
      "\n",
      "pages={e1008624},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "publisher={Public Library of Science San Francisco, CA USA}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{maaten2008visualizing,\n",
      "\n",
      "title={Visualizing data using t-SNE},\n",
      "\n",
      "author={Maaten, Laurens van der and Hinton, Geoffrey},\n",
      "\n",
      "journal={Journal of machine learning research},\n",
      "\n",
      "volume={9},\n",
      "\n",
      "number={Nov},\n",
      "\n",
      "pages={2579--2605},\n",
      "\n",
      "year={2008}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{belkin2003laplacian,\n",
      "\n",
      "title={Laplacian eigenmaps for dimensionality reduction and data representation},\n",
      "\n",
      "author={Belkin, Mikhail and Niyogi, Partha},\n",
      "\n",
      "journal={Neural computation},\n",
      "\n",
      "volume={15},\n",
      "\n",
      "number={6},\n",
      "\n",
      "pages={1373--1396},\n",
      "\n",
      "year={2003},\n",
      "\n",
      "publisher={MIT Press}\n",
      "\n",
      "}\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# Fields to drop: copyright, abstract, publisher, note, language, month, \n",
    "new_lines = list()\n",
    "\n",
    "for l in lines:\n",
    "    print(l.strip())\n",
    "    print()\n",
    "    \n",
    "    if not l.strip().startswith((\"copyright\",\"abstract\",\"publisher\",\"note\",\"language\",\"month\",\"urldate\")):\n",
    "        new_lines.append(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "e85c3dd8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "@article{wei_graph_2019,\n",
      "\n",
      "\ttitle = {Graph {Embedding} {Method} {Based} genetical trajectory {Reveals} {Migration} {History} {Among} {East} {Asians}},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/870253v1},\n",
      "\n",
      "\tdoi = {10.1101/870253},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Wei, Zhuang and Chang, Ching-Wen and Luo, Van and Bian, Beilei and Ding, Xuewei},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {870253}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{simon_local_2019,\n",
      "\n",
      "\ttitle = {Local introgression at two spatial scales in mosaic hybrid zones of mussels},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/818559v1},\n",
      "\n",
      "\tdoi = {10.1101/818559},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Simon, Alexis and Fraïsse, Christelle and Ayari, Tahani El and Liautard-Haag, Cathy and Strelkov, Petr and Welch, John J. and Bierne, Nicolas},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {818559}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{dorrity_dimensionality_2020,\n",
      "\n",
      "\ttitle = {Dimensionality reduction by {UMAP} to visualize physical and genetic interactions},\n",
      "\n",
      "\tvolume = {11},\n",
      "\n",
      "\tissn = {2041-1723},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/s41467-020-15351-4},\n",
      "\n",
      "\tdoi = {10.1038/s41467-020-15351-4},\n",
      "\n",
      "\tnumber = {1},\n",
      "\n",
      "\tjournal = {Nature Communications},\n",
      "\n",
      "\tauthor = {Dorrity, Michael W. and Saunders, Lauren M. and Queitsch, Christine and Fields, Stanley and Trapnell, Cole},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {1537}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{yelmen_creating_2019,\n",
      "\n",
      "\ttitle = {Creating {Artificial} {Human} {Genomes} {Using} {Generative} {Models}},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/769091v2},\n",
      "\n",
      "\tdoi = {10.1101/769091},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Yelmen, Burak and Decelle, Aurélien and Ongaro, Linda and Marnetto, Davide and Tallec, Corentin and Montinaro, Francesco and Furtlehner, Cyril and Pagani, Luca and Jay, Flora},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {769091}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{schmidt_population_2020,\n",
      "\n",
      "\ttitle = {Population genomics of two invasive mosquitoes ({Aedes} aegypti and {Aedes} albopictus) from the {Indo}-{Pacific}},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.03.15.993055v3},\n",
      "\n",
      "\tdoi = {10.1101/2020.03.15.993055},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Schmidt, Thomas L. and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R. and Hoffmann, Ary A.},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {2020.03.15.993055}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{schmidt2020population,\n",
      "\n",
      "  title={Population genomics of two invasive mosquitoes (Aedes aegypti and Aedes albopictus) from the Indo-Pacific},\n",
      "\n",
      "  author={Schmidt, Thomas L and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R and Hoffmann, Ary A},\n",
      "\n",
      "  journal={bioRxiv},\n",
      "\n",
      "  year={2020},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{zhang_generalized_2020,\n",
      "\n",
      "\ttitle = {A {Generalized} {Robust} {Allele}-based {Genetic} {Association} {Test}},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.03.12.989004v1},\n",
      "\n",
      "\tdoi = {10.1101/2020.03.12.989004},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Zhang, Lin and Sun, Lei},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {2020.03.12.989004}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcinnes_umap_2018,\n",
      "\n",
      "\ttitle = {{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection} for {Dimension} {Reduction}},\n",
      "\n",
      "\tshorttitle = {{UMAP}},\n",
      "\n",
      "\turl = {http://arxiv.org/abs/1802.03426},\n",
      "\n",
      "\tjournal = {arXiv},\n",
      "\n",
      "\tauthor = {McInnes, Leland and Healy, John and Melville, James},\n",
      "\n",
      "\tyear = {2018}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sakaue_dimensionality_2020,\n",
      "\n",
      "\ttitle = {Dimensionality reduction reveals fine-scale structure in the {Japanese} population with consequences for polygenic risk prediction},\n",
      "\n",
      "\tvolume = {11},\n",
      "\n",
      "\tissn = {2041-1723},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/s41467-020-15194-z},\n",
      "\n",
      "\tdoi = {10.1038/s41467-020-15194-z},\n",
      "\n",
      "\tnumber = {1},\n",
      "\n",
      "\tjournal = {Nature Communications},\n",
      "\n",
      "\tauthor = {Sakaue, Saori and Hirata, Jun and Kanai, Masahiro and Suzuki, Ken and Akiyama, Masato and Lai Too, Chun and Arayssi, Thurayya and Hammoudeh, Mohammed and Al Emadi, Samar and Masri, Basel K. and Halabi, Hussein and Badsha, Humeira and Uthman, Imad W. and Saxena, Richa and Padyukov, Leonid and Hirata, Makoto and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Okada, Yukinori},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {1569}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{karczewski_mutational_2020,\n",
      "\n",
      "\ttitle = {The mutational constraint spectrum quantified from variation in 141,456 humans},\n",
      "\n",
      "\tvolume = {581},\n",
      "\n",
      "\tissn = {1476-4687},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/s41586-020-2308-7},\n",
      "\n",
      "\tdoi = {10.1038/s41586-020-2308-7},\n",
      "\n",
      "\tnumber = {7809},\n",
      "\n",
      "\tjournal = {Nature},\n",
      "\n",
      "\tauthor = {Karczewski, Konrad J. and Francioli, Laurent C. and Tiao, Grace and Cummings, Beryl B. and Alföldi, Jessica and Wang, Qingbo and Collins, Ryan L. and Laricchia, Kristen M. and Ganna, Andrea and Birnbaum, Daniel P. and Gauthier, Laura D. and Brand, Harrison and Solomonson, Matthew and Watts, Nicholas A. and Rhodes, Daniel and Singer-Berk, Moriel and England, Eleina M. and Seaby, Eleanor G. and Kosmicki, Jack A. and Walters, Raymond K. and Tashman, Katherine and Farjoun, Yossi and Banks, Eric and Poterba, Timothy and Wang, Arcturus and Seed, Cotton and Whiffin, Nicola and Chong, Jessica X. and Samocha, Kaitlin E. and Pierce-Hoffman, Emma and Zappala, Zachary and O’Donnell-Luria, Anne H. and Minikel, Eric Vallabh and Weisburd, Ben and Lek, Monkol and Ware, James S. and Vittal, Christopher and Armean, Irina M. and Bergelson, Louis and Cibulskis, Kristian and Connolly, Kristen M. and Covarrubias, Miguel and Donnelly, Stacey and Ferriera, Steven and Gabriel, Stacey and Gentry, Jeff and Gupta, Namrata and Jeandet, Thibault and Kaplan, Diane and Llanwarne, Christopher and Munshi, Ruchi and Novod, Sam and Petrillo, Nikelle and Roazen, David and Ruano-Rubio, Valentin and Saltzman, Andrea and Schleicher, Molly and Soto, Jose and Tibbetts, Kathleen and Tolonen, Charlotte and Wade, Gordon and Talkowski, Michael E. and Neale, Benjamin M. and Daly, Mark J. and MacArthur, Daniel G.},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {434--443}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{tonkin-hill_fast_2019,\n",
      "\n",
      "\ttitle = {Fast hierarchical {Bayesian} analysis of population structure},\n",
      "\n",
      "\tvolume = {47},\n",
      "\n",
      "\tissn = {0305-1048},\n",
      "\n",
      "\turl = {https://academic.oup.com/nar/article/47/11/5539/5488018},\n",
      "\n",
      "\tdoi = {10.1093/nar/gkz361},\n",
      "\n",
      "\tnumber = {11},\n",
      "\n",
      "\tjournal = {Nucleic Acids Research},\n",
      "\n",
      "\tauthor = {Tonkin-Hill, Gerry and Lees, John A. and Bentley, Stephen D. and Frost, Simon D. W. and Corander, Jukka},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {5539--5549}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{hunter-zinck_genotyping_2020,\n",
      "\n",
      "\ttitle = {Genotyping {Array} {Design} and {Data} {Quality} {Control} in the {Million} {Veteran} {Program}},\n",
      "\n",
      "\tvolume = {106},\n",
      "\n",
      "\tissn = {0002-9297},\n",
      "\n",
      "\turl = {http://www.sciencedirect.com/science/article/pii/S000292972030080X},\n",
      "\n",
      "\tdoi = {10.1016/j.ajhg.2020.03.004},\n",
      "\n",
      "\tnumber = {4},\n",
      "\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\n",
      "\tauthor = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Gong, Xin and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tkeywords = {GWAS, biobank, genetic ancestry, clinical variants, genetic relatedness, genotype data, Million Veteran Program, quality control, SNP array design, VA},\n",
      "\n",
      "\tpages = {535--548}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{hunter-zinck_measuring_2020,\n",
      "\n",
      "\ttitle = {Measuring genetic variation in the multi-ethnic {Million} {Veteran} {Program} ({MVP})},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.01.06.896613v1},\n",
      "\n",
      "\tdoi = {10.1101/2020.01.06.896613},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju and Program, on behalf of the VA Million Veteran},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {2020.01.06.896613}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{dai_population_2020,\n",
      "\n",
      "\ttitle = {Population {Histories} of the {United} {States} {Revealed} through {Fine}-{Scale} {Migration} and {Haplotype} {Analysis}},\n",
      "\n",
      "\tvolume = {106},\n",
      "\n",
      "\tissn = {00029297},\n",
      "\n",
      "\turl = {https://linkinghub.elsevier.com/retrieve/pii/S0002929720300446},\n",
      "\n",
      "\tdoi = {10.1016/j.ajhg.2020.02.002},\n",
      "\n",
      "\tnumber = {3},\n",
      "\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\n",
      "\tauthor = {Dai, Chengzhen L. and Vazifeh, Mohammad M. and Yeang, Chen-Hsiang and Tachet, Remi and Wells, R. Spencer and Vilar, Miguel G. and Daly, Mark J. and Ratti, Carlo and Martin, Alicia R.},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {371--388}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sanchez-barreiro_historical_2020,\n",
      "\n",
      "\ttitle = {Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros ({Ceratotherium} simum)},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.05.10.086686v1},\n",
      "\n",
      "\tdoi = {10.1101/2020.05.10.086686},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Sánchez-Barreiro, Fátima and Gopalakrishnan, Shyam and Ramos-Madrigal, Jazmín and Westbury, Michael V. and Manuel, Marc de and Margaryan, Ashot and Ciucani, Marta M. and Vieira, Filipe G. and Patramanis, Yannis and Kalthoff, Daniela C. and Timmons, Zena and Sicheritz-Pontén, Thomas and Dalén, Love and Ryder, Oliver A. and Zhang, Guojie and Marquès-Bonet, Tomás and Moodley, Yoshan and Gilbert, M. Thomas P.},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {2020.05.10.086686}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{almarri_population_2020,\n",
      "\n",
      "\ttitle = {Population {Structure}, {Stratification} and {Introgression} of {Human} {Structural} {Variation}},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/746172v2},\n",
      "\n",
      "\tdoi = {10.1101/746172},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Almarri, Mohamed A. and Bergström, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S. and Chen, Yuan and Hurles, Matthew E. and Tyler-Smith, Chris and Xue, Yali},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {746172}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{almarri2020population,\n",
      "\n",
      "  title={Population Structure, Stratification, and Introgression of Human Structural Variation},\n",
      "\n",
      "  author={Almarri, Mohamed A and Bergstr{\\\"o}m, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S and Chen, Yuan and Hurles, Matthew E and Tyler-Smith, Chris and Xue, Yali},\n",
      "\n",
      "  journal={Cell},\n",
      "\n",
      "  year={2020},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{belbin_towards_2019,\n",
      "\n",
      "\ttitle = {Towards a fine-scale population health monitoring system},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/780668v1},\n",
      "\n",
      "\tdoi = {10.1101/780668},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {780668}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{margaryan_population_2019,\n",
      "\n",
      "\ttitle = {Population genomics of the {Viking} world},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/703405v1},\n",
      "\n",
      "\tdoi = {10.1101/703405},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Margaryan, Ashot and Lawson, Daniel and Sikora, Martin and Racimo, Fernando and Rasmussen, Simon and Moltke, Ida and Cassidy, Lara and Jørsboe, Emil and Ingason, Andrés and Pedersen, Mikkel and Korneliussen, Thorfinn and Wilhelmson, Helene and Buś, Magdalena and Damgaard, Peter de Barros and Martiniano, Rui and Renaud, Gabriel and Bhérer, Claude and Moreno-Mayar, J. Víctor and Fotakis, Anna and Allen, Marie and Molak, Martyna and Cappellini, Enrico and Scorrano, Gabriele and Buzhilova, Alexandra and Fox, Allison and Albrechtsen, Anders and Schütz, Berit and Skar, Birgitte and Arcini, Caroline and Falys, Ceri and Jonson, Charlotte Hedenstierna and Błaszczyk, Dariusz and Pezhemsky, Denis and Turner-Walker, Gordon and Gestsdóttir, Hildur and Lundstrøm, Inge and Gustin, Ingrid and Mainland, Ingrid and Potekhina, Inna and Muntoni, Italo and Cheng, Jade and Stenderup, Jesper and Ma, Jilong and Gibson, Julie and Peets, Jüri and Gustafsson, Jörgen and Iversen, Katrine and Simpson, Linzi and Strand, Lisa and Loe, Louise and Sikora, Maeve and Florek, Marek and Vretemark, Maria and Redknap, Mark and Bajka, Monika and Pushkina, Tamara and Søvsø, Morten and Grigoreva, Natalia and Christensen, Tom and Kastholm, Ole and Uldum, Otto and Favia, Pasquale and Holck, Per and Allmäe, Raili and Sten, Sabine and Arge, Símun and Ellingvåg, Sturla and Moiseyev, Vayacheslav and Bogdanowicz, Wiesław and Magnusson, Yvonne and Orlando, Ludovic and Bradley, Daniel and Jørkov, Marie Louise and Arneborg, Jette and Lynnerup, Niels and Price, Neil and Gilbert, M. Thomas and Allentoft, Morten and Bill, Jan and Sindbæk, Søren and Hedeager, Lotte and Kristiansen, Kristian and Nielsen, Rasmus and Werge, Thomas and Willerslev, Eske},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {703405}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{consortium_genome_2020,\n",
      "\n",
      "\ttitle = {Genome variation and population structure among 1,142 mosquitoes of the {African} malaria vector species {Anopheles} gambiae and {Anopheles} coluzzii},\n",
      "\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/864314v2},\n",
      "\n",
      "\tdoi = {10.1101/864314},\n",
      "\n",
      "\tjournal = {bioRxiv},\n",
      "\n",
      "\tauthor = {Consortium, The Anopheles gambiae 1000 Genomes and Clarkson, Chris S. and Miles, Alistair and Harding, Nicholas J. and Lucas, Eric R. and Battey, C. J. and Amaya-Romero, Jorge Edouardo and Cano, Jorge and Diabate, Abdoulaye and Constant, Edi and Nwakanma, Davis C. and Jawara, Musa and Essandoh, John and Dinis, Joao and Goff, Gilbert Le and Robert, Vincent and Troco, Arlete D. and Costantini, Carlo and Rohatgi, Kyanne R. and Elissa, Nohal and Coulibaly, Boubacar and Midega, Janet and Mbogo, Charles and Mawejje, Henry D. and Stalker, Jim and Rockett, Kirk A. and Drury, Eleanor and Mead, Daniel and Jeffreys, Anna E. and Hubbart, Christina and Rowlands, Kate and Isaacs, Alison T. and Jyothi, Dushyanth and Malangone, Cinzia and Kamali, Maryam and Henrichs, Christa and Simpson, Victoria and Ayala, Diego and Besansky, Nora J. and Burt, Austin and Caputo, Beniamino and Torre, Alessandra della and Fontaine, Michael and Godfray, H. Charles J. and Hahn, Matthew W. and Kern, Andrew D. and Lawniczak, Mara K. N. and O’Loughlin, Samantha and Pinto, Joao and Riehle, Michelle M. and Sharakhov, Igor and Schrider, Daniel R. and Vernick, Kenneth D. and White, Bradley J. and Donnelly, Martin J. and Kwiatkowski, Dominic P.},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {864314}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{greenbaum_network-based_2019,\n",
      "\n",
      "\ttitle = {Network-based hierarchical population structure analysis for large genomic data sets},\n",
      "\n",
      "\tvolume = {29},\n",
      "\n",
      "\tissn = {1088-9051, 1549-5469},\n",
      "\n",
      "\turl = {http://genome.cshlp.org/content/29/12/2020},\n",
      "\n",
      "\tdoi = {10.1101/gr.250092.119},\n",
      "\n",
      "\tnumber = {12},\n",
      "\n",
      "\tjournal = {Genome {R}esearch},\n",
      "\n",
      "\tauthor = {Greenbaum, Gili and Rubin, Amir and Templeton, Alan R. and Rosenberg, Noah A.},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpmid = {31694865},\n",
      "\n",
      "\tpages = {2020--2033}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{yamamoto_genetic_2020,\n",
      "\n",
      "\ttitle = {Genetic and phenotypic landscape of the mitochondrial genome in the {Japanese} population},\n",
      "\n",
      "\tvolume = {3},\n",
      "\n",
      "\tissn = {2399-3642},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/s42003-020-0812-9},\n",
      "\n",
      "\tdoi = {10.1038/s42003-020-0812-9},\n",
      "\n",
      "\tnumber = {1},\n",
      "\n",
      "\tjournal = {Communications Biology},\n",
      "\n",
      "\tauthor = {Yamamoto, Kenichi and Sakaue, Saori and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Ozono, Keiichi and Momozawa, Yukihide and Okada, Yukinori},\n",
      "\n",
      "\tyear = {2020},\n",
      "\n",
      "\tpages = {1--11}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{derkarabetian_demonstration_2019,\n",
      "\n",
      "\ttitle = {A demonstration of unsupervised machine learning in species delimitation},\n",
      "\n",
      "\tvolume = {139},\n",
      "\n",
      "\tissn = {1055-7903},\n",
      "\n",
      "\turl = {http://www.sciencedirect.com/science/article/pii/S1055790319301721},\n",
      "\n",
      "\tdoi = {10.1016/j.ympev.2019.106562},\n",
      "\n",
      "\tjournal = {Molecular Phylogenetics and Evolution},\n",
      "\n",
      "\tauthor = {Derkarabetian, Shahan and Castillo, Stephanie and Koo, Peter K. and Ovchinnikov, Sergey and Hedin, Marshal},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tkeywords = {t-SNE, Integrative taxonomy, Opiliones, Random forest, Ultraconserved elements, Variational autoencoders},\n",
      "\n",
      "\tpages = {106562}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{diaz-papkovich_umap_2019,\n",
      "\n",
      "\ttitle = {{UMAP} reveals cryptic population structure and phenotype heterogeneity in large genomic cohorts},\n",
      "\n",
      "\tvolume = {15},\n",
      "\n",
      "\tissn = {1553-7390},\n",
      "\n",
      "\turl = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853336/},\n",
      "\n",
      "\tdoi = {10.1371/journal.pgen.1008432},\n",
      "\n",
      "\tnumber = {11},\n",
      "\n",
      "\tjournal = {PLoS Genetics},\n",
      "\n",
      "\tauthor = {Diaz-Papkovich, Alex and Anderson-Trocmé, Luke and Ben-Eghan, Chief and Gravel, Simon},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpmid = {31675358},\n",
      "\n",
      "\tpmcid = {PMC6853336}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{li_past_2020,\n",
      "\n",
      "\ttitle = {Past demography and local adaptation in forest trees : {Insights} from natural populations and breeding programs of {Norway} spruce},\n",
      "\n",
      "\tshorttitle = {Past demography and local adaptation in forest trees},\n",
      "\n",
      "\turl = {http://urn.kb.se/resolve?urn=urn:nbn:se:uu:diva-403450},\n",
      "\n",
      "\tauthor = {Li, Lili},\n",
      "\n",
      "\tyear = {2020}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{becht_dimensionality_2019,\n",
      "\n",
      "\ttitle = {Dimensionality reduction for visualizing single-cell data using {UMAP}},\n",
      "\n",
      "\tvolume = {37},\n",
      "\n",
      "\tissn = {1546-1696},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/nbt.4314},\n",
      "\n",
      "\tdoi = {10.1038/nbt.4314},\n",
      "\n",
      "\tnumber = {1},\n",
      "\n",
      "\tjournal = {Nature Biotechnology},\n",
      "\n",
      "\tauthor = {Becht, Etienne and McInnes, Leland and Healy, John and Dutertre, Charles-Antoine and Kwok, Immanuel W. H. and Ng, Lai Guan and Ginhoux, Florent and Newell, Evan W.},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {38--44}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{kobak_art_2019,\n",
      "\n",
      "\ttitle = {The art of using t-{SNE} for single-cell transcriptomics},\n",
      "\n",
      "\tvolume = {10},\n",
      "\n",
      "\tissn = {2041-1723},\n",
      "\n",
      "\turl = {https://www.nature.com/articles/s41467-019-13056-x},\n",
      "\n",
      "\tdoi = {10.1038/s41467-019-13056-x},\n",
      "\n",
      "\tnumber = {1},\n",
      "\n",
      "\tjournal = {Nature Communications},\n",
      "\n",
      "\tauthor = {Kobak, Dmitry and Berens, Philipp},\n",
      "\n",
      "\tyear = {2019},\n",
      "\n",
      "\tpages = {5416}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{spear2020recent,\n",
      "\n",
      "  title={Recent fluctuations in Mexican American genomes have altered the genetic architecture of biomedical traits},\n",
      "\n",
      "  author={Spear, Melissa L and Diaz-Papkovich, Alex and Ziv, Elad and Gravel, Simon and Torgerson, Dara G and Hernandez, Ryan},\n",
      "\n",
      "  journal={bioRxiv},\n",
      "\n",
      "  year={2020},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{patterson2006population,\n",
      "\n",
      "  title={Population structure and eigenanalysis},\n",
      "\n",
      "  author={Patterson, Nick and Price, Alkes L and Reich, David},\n",
      "\n",
      "  journal={PLoS Genetics},\n",
      "\n",
      "  volume={2},\n",
      "\n",
      "  number={12},\n",
      "\n",
      "  pages={e190},\n",
      "\n",
      "  year={2006},\n",
      "\n",
      "  doi={10.1371/journal.pgen.0020190}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{10002015global,\n",
      "\n",
      "  title={A global reference for human genetic variation},\n",
      "\n",
      "  author={1000 Genomes Project Consortium and others},\n",
      "\n",
      "  journal={Nature},\n",
      "\n",
      "  volume={526},\n",
      "\n",
      "  number={7571},\n",
      "\n",
      "  pages={68--74},\n",
      "\n",
      "  year={2015},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{cann2002human,\n",
      "\n",
      "  title={A human genome diversity cell line panel},\n",
      "\n",
      "  author={Cann, Howard M and De Toma, Claudia and Cazes, Lucien and Legrand, Marie-Fernande and Morel, Valerie and Piouffre, Laurence and Bodmer, Julia and Bodmer, Walter F and Bonne-Tamir, Batsheva and Cambon-Thomsen, Anne and others},\n",
      "\n",
      "  journal={Science},\n",
      "\n",
      "  volume={296},\n",
      "\n",
      "  number={5566},\n",
      "\n",
      "  pages={261--262},\n",
      "\n",
      "  year={2002},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{matise2011next,\n",
      "\n",
      "  title={The Next PAGE in understanding complex traits: design for the analysis of Population Architecture Using Genetics and Epidemiology (PAGE) Study},\n",
      "\n",
      "  author={Matise, Tara C and Ambite, Jose Luis and Buyske, Steven and Carlson, Christopher S and Cole, Shelley A and Crawford, Dana C and Haiman, Christopher A and Heiss, Gerardo and Kooperberg, Charles and Marchand, Loic Le and others},\n",
      "\n",
      "  journal={American journal of epidemiology},\n",
      "\n",
      "  volume={174},\n",
      "\n",
      "  number={7},\n",
      "\n",
      "  pages={849--859},\n",
      "\n",
      "  year={2011},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{nelson2008population,\n",
      "\n",
      "  title={The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research},\n",
      "\n",
      "  author={Nelson, Matthew R and Bryc, Katarzyna and King, Karen S and Indap, Amit and Boyko, Adam R and Novembre, John and Briley, Linda P and Maruyama, Yuka and Waterworth, Dawn M and Waeber, G{\\'e}rard and others},\n",
      "\n",
      "  journal={The American Journal of Human Genetics},\n",
      "\n",
      "  volume={83},\n",
      "\n",
      "  number={3},\n",
      "\n",
      "  pages={347--358},\n",
      "\n",
      "  year={2008},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{juster1995overview,\n",
      "\n",
      "  title={An overview of the Health and Retirement Study},\n",
      "\n",
      "  author={Juster, F Thomas and Suzman, Richard},\n",
      "\n",
      "  journal={Journal of Human Resources},\n",
      "\n",
      "  pages={S7--S56},\n",
      "\n",
      "  year={1995},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sudlow2015uk,\n",
      "\n",
      "  title={UK biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age},\n",
      "\n",
      "  author={Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and others},\n",
      "\n",
      "  journal={Plos med},\n",
      "\n",
      "  volume={12},\n",
      "\n",
      "  number={3},\n",
      "\n",
      "  pages={e1001779},\n",
      "\n",
      "  year={2015},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sorlie2010design,\n",
      "\n",
      "  title={Design and implementation of the Hispanic community health study/study of Latinos},\n",
      "\n",
      "  author={Sorlie, Paul D and Avil{\\'e}s-Santa, Larissa M and Wassertheil-Smoller, Sylvia and Kaplan, Robert C and Daviglus, Martha L and Giachello, Aida L and Schneiderman, Neil and Raij, Leopoldo and Talavera, Gregory and Allison, Matthew and others},\n",
      "\n",
      "  journal={Annals of epidemiology},\n",
      "\n",
      "  volume={20},\n",
      "\n",
      "  number={8},\n",
      "\n",
      "  pages={629--641},\n",
      "\n",
      "  year={2010},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{nagai2017overview,\n",
      "\n",
      "  title={Overview of the {BioBank Japan Project}: study design and profile},\n",
      "\n",
      "  author={Nagai, Akiko and Hirata, Makoto and Kamatani, Yoichiro and Muto, Kaori and Matsuda, Koichi and Kiyohara, Yutaka and Ninomiya, Toshiharu and Tamakoshi, Akiko and Yamagata, Zentaro and Mushiroda, Taisei and others},\n",
      "\n",
      "  journal={Journal of epidemiology},\n",
      "\n",
      "  volume={27},\n",
      "\n",
      "  number={Supplement\\_III},\n",
      "\n",
      "  pages={S2--S8},\n",
      "\n",
      "  year={2017},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{lawson2012inference,\n",
      "\n",
      "  title={Inference of population structure using dense haplotype data},\n",
      "\n",
      "  author={Lawson, Daniel John and Hellenthal, Garrett and Myers, Simon and Falush, Daniel},\n",
      "\n",
      "  journal={PLoS Genetics},\n",
      "\n",
      "  volume={8},\n",
      "\n",
      "  number={1},\n",
      "\n",
      "  pages={e1002453},\n",
      "\n",
      "  year={2012},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{alexander2009fast,\n",
      "\n",
      "  title={Fast model-based estimation of ancestry in unrelated individuals},\n",
      "\n",
      "  author={Alexander, David H and Novembre, John and Lange, Kenneth},\n",
      "\n",
      "  journal={Genome {R}esearch},\n",
      "\n",
      "  volume={19},\n",
      "\n",
      "  number={9},\n",
      "\n",
      "  pages={1655--1664},\n",
      "\n",
      "  year={2009},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@Article{Hunter2007,\n",
      "\n",
      "  Author    = {Hunter, J. D.},\n",
      "\n",
      "  Title     = {Matplotlib: A 2D graphics environment},\n",
      "\n",
      "  Journal   = {Computing In Science \\& Engineering},\n",
      "\n",
      "  Volume    = {9},\n",
      "\n",
      "  Number    = {3},\n",
      "\n",
      "  Pages     = {90--95},\n",
      "\n",
      "  for application development, interactive scripting, and\n",
      "\n",
      "  publication-quality image generation across user\n",
      "\n",
      "  interfaces and operating systems.},\n",
      "\n",
      "  doi       = {10.1109/MCSE.2007.55},\n",
      "\n",
      "  year      = 2007\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{scikit-learn,\n",
      "\n",
      " title={Scikit-learn: Machine Learning in {P}ython},\n",
      "\n",
      " author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.\n",
      "\n",
      "         and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.\n",
      "\n",
      "         and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and\n",
      "\n",
      "         Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},\n",
      "\n",
      " journal={Journal of Machine Learning Research},\n",
      "\n",
      " volume={12},\n",
      "\n",
      " pages={2825--2830},\n",
      "\n",
      " year={2011}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{kerminen2019geographic,\n",
      "\n",
      "  title={Geographic variation and bias in the polygenic scores of complex diseases and traits in {F}inland},\n",
      "\n",
      "  author={Kerminen, Sini and Martin, Alicia R and Koskela, Jukka and Ruotsalainen, Sanni E and Havulinna, Aki S and Surakka, Ida and Palotie, Aarno and Perola, Markus and Salomaa, Veikko and Daly, Mark J and others},\n",
      "\n",
      "  journal={The American Journal of Human Genetics},\n",
      "\n",
      "  volume={104},\n",
      "\n",
      "  number={6},\n",
      "\n",
      "  pages={1169--1181},\n",
      "\n",
      "  year={2019},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{berg2019reduced,\n",
      "\n",
      "  title={Reduced signal for polygenic adaptation of height in {UK} {B}iobank},\n",
      "\n",
      "  author={Berg, Jeremy J and Harpak, Arbel and Sinnott-Armstrong, Nasa and Joergensen, Anja Moltke and Mostafavi, Hakhamanesh and Field, Yair and Boyle, Evan August and Zhang, Xinjun and Racimo, Fernando and Pritchard, Jonathan K and others},\n",
      "\n",
      "  journal={e{L}ife},\n",
      "\n",
      "  volume={8},\n",
      "\n",
      "  pages={e39725},\n",
      "\n",
      "  year={2019},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{sohail2019polygenic,\n",
      "\n",
      "  title={Polygenic adaptation on height is overestimated due to uncorrected stratification in genome-wide association studies},\n",
      "\n",
      "  author={Sohail, Mashaal and Maier, Robert M and Ganna, Andrea and Bloemendal, Alex and Martin, Alicia R and Turchin, Michael C and Chiang, Charleston WK and Hirschhorn, Joel and Daly, Mark J and Patterson, Nick and others},\n",
      "\n",
      "  journal={e{L}ife},\n",
      "\n",
      "  volume={8},\n",
      "\n",
      "  pages={e39702},\n",
      "\n",
      "  year={2019},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcvean2009genealogical,\n",
      "\n",
      "  title={A genealogical interpretation of principal components analysis},\n",
      "\n",
      "  author={McVean, Gil},\n",
      "\n",
      "  journal={PLoS Genetics},\n",
      "\n",
      "  volume={5},\n",
      "\n",
      "  number={10},\n",
      "\n",
      "  pages={e1000686},\n",
      "\n",
      "  year={2009},\n",
      "\n",
      "  doi = {10.1371/journal.pgen.1000686}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{moon2019visualizing,\n",
      "\n",
      "  title={Visualizing structure and transitions in high-dimensional biological data},\n",
      "\n",
      "  author={Moon, Kevin R and van Dijk, David and Wang, Zheng and Gigante, Scott and Burkhardt, Daniel B and Chen, William S and Yim, Kristina and van den Elzen, Antonia and Hirn, Matthew J and Coifman, Ronald R and others},\n",
      "\n",
      "  journal={Nature {B}iotechnology},\n",
      "\n",
      "  volume={37},\n",
      "\n",
      "  number={12},\n",
      "\n",
      "  pages={1482--1492},\n",
      "\n",
      "  year={2019},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{micheletti_genetic_2020,\n",
      "\n",
      "\ttitle = {Genetic {Consequences} of the {Transatlantic} {Slave} {Trade} in the {Americas}},\n",
      "\n",
      "\tvolume = {0},\n",
      "\n",
      "\tissn = {0002-9297, 1537-6605},\n",
      "\n",
      "\turl = {https://www.cell.com/ajhg/abstract/S0002-9297(20)30200-7},\n",
      "\n",
      "\tdoi = {10.1016/j.ajhg.2020.06.012},\n",
      "\n",
      "\tnumber = {0},\n",
      "\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\n",
      "\tauthor = {Micheletti, Steven J. and Bryc, Kasia and Esselmann, Samantha G. Ancona and Freyman, William A. and Moreno, Meghan E. and Poznik, G. David and Shastri, Anjali J. and Agee, M. and Aslibekyan, S. and Auton, A. and Bell, R. and Clark, S. and Das, S. and Elson, S. and Fletez-Brant, K. and Fontanillas, P. and Gandhi, P. and Heilbron, K. and Hicks, B. and Hinds, D. and Huber, K. and Jewett, E. and Jiang, Y. and Kleinman, A. and Lin, K. and Litterman, N. and McCreight, J. and McIntyre, M. and McManus, K. and Mozaffari, S. and Nandakumar, P. and Noblin, L. and Northover, C. and O’Connell, J. and Petrakovitz, A. and Pitts, S. and Shelton, J. and Shringarpure, S. and Tian, C. and Tung, J. and Tunney, R. and Vacic, V. and Wang, X. and Zare, A. and Beleza, Sandra and Mountain, Joanna L.},\n",
      "\n",
      "\tyear = {2020}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@book{holmes2018modern,\n",
      "\n",
      "  title={Modern statistics for modern biology},\n",
      "\n",
      "  author={Holmes, Susan and Huber, Wolfgang},\n",
      "\n",
      "  year={2018},\n",
      "\n",
      "  chapter={Introduction},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@misc{genizon,\n",
      "\n",
      "title = {Genizon Biobank},\n",
      "\n",
      "year={2020},\n",
      "\n",
      "url={http://www.genomequebec.com/genizon-biobank/}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mcinnes2018software,\n",
      "\n",
      "  title={{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection}},\n",
      "\n",
      "  author={McInnes, Leland and Healy, John and Saul, Nathaniel and Grossberger, Lukas},\n",
      "\n",
      "  journal={The Journal of Open Source Software},\n",
      "\n",
      "  volume={3},\n",
      "\n",
      "  number={29},\n",
      "\n",
      "  pages={861},\n",
      "\n",
      "  year={2018}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{mathieson2020ancestry,\n",
      "\n",
      "  title={What is ancestry?},\n",
      "\n",
      "  author={Mathieson, Iain and Scally, Aylwyn},\n",
      "\n",
      "  journal={PLoS Genetics},\n",
      "\n",
      "  volume={16},\n",
      "\n",
      "  number={3},\n",
      "\n",
      "  pages={e1008624},\n",
      "\n",
      "  year={2020},\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{maaten2008visualizing,\n",
      "\n",
      "  title={Visualizing data using t-SNE},\n",
      "\n",
      "  author={Maaten, Laurens van der and Hinton, Geoffrey},\n",
      "\n",
      "  journal={Journal of machine learning research},\n",
      "\n",
      "  volume={9},\n",
      "\n",
      "  number={Nov},\n",
      "\n",
      "  pages={2579--2605},\n",
      "\n",
      "  year={2008}\n",
      "\n",
      "}\n",
      "\n",
      "\n",
      "\n",
      "@article{belkin2003laplacian,\n",
      "\n",
      "  title={Laplacian eigenmaps for dimensionality reduction and data representation},\n",
      "\n",
      "  author={Belkin, Mikhail and Niyogi, Partha},\n",
      "\n",
      "  journal={Neural computation},\n",
      "\n",
      "  volume={15},\n",
      "\n",
      "  number={6},\n",
      "\n",
      "  pages={1373--1396},\n",
      "\n",
      "  year={2003},\n",
      "\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "for l in new_lines:\n",
    "    print(l)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "7308677f",
   "metadata": {},
   "outputs": [],
   "source": [
    "bibfilepath_out = \"../ms_files/umap_popgen_review_trimmed.bib\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "e282319d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "28039"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f = open(bibfilepath_out, \"w\")\n",
    "f.write(\"\".join(str(i) for i in new_lines))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9d66adbe",
   "metadata": {},
   "source": [
    "# Add keywords to citation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "6a59bef4",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Find all citations used in intro and disco\n",
    "intro_path = \"../base/intro/intro.tex\"\n",
    "disco_path = \"../base/conclusion/discussion.tex\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "eae07ce7",
   "metadata": {},
   "outputs": [],
   "source": [
    "file = open(disco_path)\n",
    "lines = file.readlines()\n",
    "file.close()\n",
    "\n",
    "lines = \"\".join(lines)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "ca27b589",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Loop through and find each citation by making iterative substrings\n",
    "\n",
    "sublines = lines\n",
    "citations = list()\n",
    "\n",
    "while sublines.find(\"\\\\citep{\") > 0:\n",
    "    pos1 = sublines.find(\"\\\\citep{\")\n",
    "    subline = sublines[pos1:]\n",
    "    pos2 = subline.find(\"}\")\n",
    "    \n",
    "    cite = subline[7:pos2]\n",
    "    \n",
    "    citations = citations + cite.split(\",\")\n",
    "    \n",
    "    sublines = subline[pos2:]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "91033501",
   "metadata": {},
   "outputs": [],
   "source": [
    "#citations_intro = list(set(citations))\n",
    "#citations_disco = list(set(citations))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "3273f74e",
   "metadata": {},
   "outputs": [],
   "source": [
    "citations_all = citations_intro + citations_disco"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "id": "c3f059f8",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Loop through the bib file and identify the entries.\n",
    "# Add a \"keyword = {mainbib}\"\n",
    "\n",
    "bib_path = \"../phd_thesis.bib\"\n",
    "\n",
    "file = open(bib_path)\n",
    "lines = file.readlines()\n",
    "file.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "c5d5a68d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "@article{global_2015,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A global reference for human genetic variation},\n",
      "\tvolume = {526},\n",
      "\tissn = {1476-4687},\n",
      "\turl = {https://www.nature.com/articles/nature15393},\n",
      "\tdoi = {10.1038/nature15393},\n",
      "\tnumber = {7571},\n",
      "\tjournal = {Nature},\n",
      "\tauthor = {The 1000 Genomes Project Consortium},\n",
      "\tyear = {2015},\n",
      "\tpages = {68--74}}\n",
      "\n",
      "@article{awadalla_cohort_2013,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Cohort profile of the {CARTaGENE} study: {Quebec}’s population-based biobank for public health and personalized genomics},\n",
      "\tvolume = {42},\n",
      "\tissn = {0300-5771},\n",
      "\tshorttitle = {Cohort profile of the {CARTaGENE} study},\n",
      "\turl = {https://doi.org/10.1093/ije/dys160},\n",
      "\tdoi = {10.1093/ije/dys160},\n",
      "\tnumber = {5},\n",
      "\tjournal = {International Journal of Epidemiology},\n",
      "\tauthor = {Awadalla, Philip and Boileau, Catherine and Payette, Yves and Idaghdour, Youssef and Goulet, Jean-Philippe and Knoppers, Bartha and Hamet, Pavel and Laberge, Claude},\n",
      "\tyear = {2013},\n",
      "\tpages = {1285--1299}\n",
      "}\n",
      "\n",
      "@article{juster_overview_1995,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {An {Overview} of the {Health} and {Retirement} {Study}},\n",
      "\tvolume = {30},\n",
      "\tissn = {0022-166X},\n",
      "\turl = {https://www.jstor.org/stable/146277},\n",
      "\tdoi = {10.2307/146277},\n",
      "\tjournal = {The Journal of Human Resources},\n",
      "\tauthor = {Juster, F. Thomas and Suzman, Richard},\n",
      "\tyear = {1995},\n",
      "\tpages = {S7--S56}\n",
      "}\n",
      "\n",
      "@article{sudlow_uk_2015,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {{UK} {Biobank}: {An} {Open} {Access} {Resource} for {Identifying} the {Causes} of a {Wide} {Range} of {Complex} {Diseases} of {Middle} and {Old} {Age}},\n",
      "\tvolume = {12},\n",
      "\tissn = {1549-1676},\n",
      "\tshorttitle = {{UK} {Biobank}},\n",
      "\turl = {https://journals.plos.org/plosmedicine/article?id=10.1371/journal.pmed.1001779},\n",
      "\tdoi = {10.1371/journal.pmed.1001779},\n",
      "\tnumber = {3},\n",
      "\tjournal = {PLOS Medicine},\n",
      "\tauthor = {Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and Liu, Bette and Matthews, Paul and Ong, Giok and Pell, Jill and Silman, Alan and Young, Alan and Sprosen, Tim and Peakman, Tim and Collins, Rory},\n",
      "\tyear = {2015},\n",
      "\tpages = {e1001779}\n",
      "}\n",
      "\n",
      "@article{novembre2008europe,\n",
      "\tkeywords = {mainbib},\n",
      "\tAuthor = {Novembre, John and Johnson, Toby and Bryc, Katarzyna and Kutalik, Zolt{\\'a}n and Boyko, Adam R. and Auton, Adam and Indap, Amit and King, Karen S. and Bergmann, Sven and Nelson, Matthew R. and Stephens, Matthew and Bustamante, Carlos D.},\n",
      "\tDay = {31},\n",
      "\tJournal = {Nature},\n",
      "\tPages = {98-101},\n",
      "\tTitle = {Genes mirror geography within {E}urope},\n",
      "\tUrl = {http://dx.doi.org/10.1038/nature07331},\n",
      "\tVolume = {456},\n",
      "\tYear = {2008}\n",
      "}\n",
      "\t\n",
      "\n",
      "@article{wei_graph_2019,\n",
      "\ttitle = {Graph {Embedding} {Method} {Based} genetical trajectory {Reveals} {Migration} {History} {Among} {East} {Asians}},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/870253v1},\n",
      "\tdoi = {10.1101/870253},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Wei, Zhuang and Chang, Ching-Wen and Luo, Van and Bian, Beilei and Ding, Xuewei},\n",
      "\tyear = {2019},\n",
      "\tpages = {870253}\n",
      "}\n",
      "\n",
      "\n",
      "@article{gibbs_human_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {The {Human} {Genome} {Project} changed everything},\n",
      "\tvolume = {21},\n",
      "\tissn = {1471-0064},\n",
      "\turl = {https://www.nature.com/articles/s41576-020-0275-3},\n",
      "\tdoi = {10.1038/s41576-020-0275-3},\n",
      "\tnumber = {10},\n",
      "\tjournal = {Nature Reviews Genetics},\n",
      "\tauthor = {Gibbs, Richard A.},\n",
      "\tyear = {2020},\n",
      "\tpages = {575--576}\n",
      "}\n",
      "\n",
      "@book{jobling_human_2013,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Human {Evolutionary} {Genetics}: {Origins}, {Peoples} \\& {Disease}},\n",
      "\tisbn = {978-1-136-84414-0},\n",
      "\tshorttitle = {Human {Evolutionary} {Genetics}},\n",
      "\tauthor = {Jobling, Mark and Hurles, Matthew and Tyler-Smith, Chris},\n",
      "\tyear = {2013}\n",
      "}\n",
      "\n",
      "@article{altemose_map_2017,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A map of human {PRDM}9 binding provides evidence for novel behaviors of {PRDM}9 and other zinc-finger proteins in meiosis},\n",
      "\tvolume = {6},\n",
      "\tissn = {2050-084X},\n",
      "\turl = {https://elifesciences.org/articles/28383},\n",
      "\tdoi = {10.7554/eLife.28383},\n",
      "\tjournal = {eLife},\n",
      "\tauthor = {Altemose, Nicolas and Noor, Nudrat and Bitoun, Emmanuelle and Tumian, Afidalina and Imbeault, Michael and Chapman, J Ross and Aricescu, A Radu and Myers, Simon R},\n",
      "\tyear = {2017},\n",
      "}\n",
      "\n",
      "@article{segurel_determinants_2014,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Determinants of {Mutation} {Rate} {Variation} in the {Human} {Germline}},\n",
      "\tvolume = {15},\n",
      "\turl = {https://doi.org/10.1146/annurev-genom-031714-125740},\n",
      "\tdoi = {10.1146/annurev-genom-031714-125740},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Annual Review of Genomics and Human Genetics},\n",
      "\tauthor = {Ségurel, Laure and Wyman, Minyoung J. and Przeworski, Molly},\n",
      "\tyear = {2014},\n",
      "\tpmid = {25000986},\n",
      "\tpages = {47--70}\n",
      "}\n",
      "\n",
      "@book{holmes_modern_2019,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Modern {Statistics} for {Modern} {Biology}},\n",
      "\turl = {https://www.huber.embl.de/msmb/},\n",
      "\tauthor = {Holmes, Susan and Huber, Wolfgang},\n",
      "\tchapter = {Introduction},\n",
      "\tisbn = {9781108705295},\n",
      "\tyear = {2019},\n",
      "\tpublisher = {Cambridge University Press}\n",
      "}\n",
      "\n",
      "@article{hoaglin_john_2003,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {John {W}. {Tukey} and {Data} {Analysis}},\n",
      "\tvolume = {18},\n",
      "\tissn = {0883-4237},\n",
      "\turl = {https://www.jstor.org/stable/3182748},\n",
      "\tnumber = {3},\n",
      "\tjournal = {Statistical Science},\n",
      "\tauthor = {Hoaglin, David C.},\n",
      "\tyear = {2003},\n",
      "\tpages = {311--318}\n",
      "}\n",
      "\n",
      "@book{tukey_1977,\n",
      "\tkeywords = {mainbib},\n",
      "    title = {{Exploratory} {Data} {Analysis}},\n",
      "    publisher = {{Addison}-{Wesley} {Publishing} {Company}},\n",
      "    year = {1977},\n",
      "    author = {Tukey, John W.}\n",
      "}\n",
      "\n",
      "@article{tukey_we_1980,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {We {Need} {Both} {Exploratory} and {Confirmatory}},\n",
      "\tvolume = {34},\n",
      "\tissn = {0003-1305, 1537-2731},\n",
      "\turl = {http://www.tandfonline.com/doi/abs/10.1080/00031305.1980.10482706},\n",
      "\tdoi = {10.1080/00031305.1980.10482706},\n",
      "\tnumber = {1},\n",
      "\tjournal = {The American Statistician},\n",
      "\tauthor = {Tukey, John W.},\n",
      "\tyear = {1980},\n",
      "\tpages = {23--25}\n",
      "}\n",
      "\n",
      "@article{mcvean_genealogical_2009,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A {Genealogical} {Interpretation} of {Principal} {Components} {Analysis}},\n",
      "\tvolume = {5},\n",
      "\tissn = {1553-7404},\n",
      "\turl = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000686},\n",
      "\tdoi = {10.1371/journal.pgen.1000686},\n",
      "\tnumber = {10},\n",
      "\tjournal = {{PLOS} {Genetics}},\n",
      "\tauthor = {McVean, Gil},\n",
      "\tyear = {2009},\n",
      "\tpages = {e1000686}\n",
      "}\n",
      "\n",
      "@article{price_principal_2006,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Principal components analysis corrects for stratification in genome-wide association studies},\n",
      "\tvolume = {38},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/ng1847},\n",
      "\tdoi = {10.1038/ng1847},\n",
      "\tnumber = {8},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Price, Alkes L. and Patterson, Nick J. and Plenge, Robert M. and Weinblatt, Michael E. and Shadick, Nancy A. and Reich, David},\n",
      "\tyear = {2006},\n",
      "\tpages = {904}}\n",
      "\n",
      "\n",
      "@article{carlsson_topology_2009,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Topology and data},\n",
      "\tvolume = {46},\n",
      "\tissn = {0273-0979},\n",
      "\turl = {http://www.ams.org/journal-getitem?pii=S0273-0979-09-01249-X},\n",
      "\tdoi = {10.1090/S0273-0979-09-01249-X},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Bulletin of the American Mathematical Society},\n",
      "\tauthor = {Carlsson, Gunnar},\n",
      "\tyear = {2009},\n",
      "\tpages = {255--308}\n",
      "}\n",
      "\n",
      "@inproceedings{mcinnes_accelerated_2017,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Accelerated {Hierarchical} {Density} {Clustering}},\n",
      "\turl = {http://arxiv.org/abs/1705.07321},\n",
      "\tdoi = {10.1109/ICDMW.2017.12},\n",
      "\tbooktitle = {2017 {IEEE} {International} {Conference} on {Data} {Mining} {Workshops} ({ICDMW})},\n",
      "\tauthor = {McInnes, Leland and Healy, John},\n",
      "\tyear = {2017},\n",
      "\tpages = {33--42}\n",
      "}\n",
      "\n",
      "@article{maaten_visualizing_2008,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Visualizing {Data} using t-{SNE}},\n",
      "\tvolume = {9},\n",
      "\turl = {http://jmlr.org/papers/v9/vandermaaten08a.html},\n",
      "\tnumber = {86},\n",
      "\tjournal = {Journal of Machine Learning Research},\n",
      "\tauthor = {Maaten, Laurens van der and Hinton, Geoffrey},\n",
      "\tyear = {2008},\n",
      "\tpages = {2579--2605}\n",
      "}\n",
      "\n",
      "\n",
      "@article{wasserman_topological_2018,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Topological {Data} {Analysis}},\n",
      "\tvolume = {5},\n",
      "\turl = {https://doi.org/10.1146/annurev-statistics-031017-100045},\n",
      "\tdoi = {10.1146/annurev-statistics-031017-100045},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Annual Review of Statistics and Its Application},\n",
      "\tauthor = {Wasserman, Larry},\n",
      "\tyear = {2018},\n",
      "\tpages = {501--532},\n",
      "}\n",
      "\n",
      "@inproceedings{carreira-perpinan_elastic_2010,\n",
      "\tkeywords = {mainbib},\n",
      "\tseries = {{ICML} 10},\n",
      "\ttitle = {The elastic embedding algorithm for dimensionality reduction},\n",
      "\tisbn = {978-1-60558-907-7},\n",
      "\tbooktitle = {Proceedings of the 27th {International} {Conference} on {International} {Conference} on {Machine} {Learning}},\n",
      "\tauthor = {Carreira-Perpiñan, Miguel Á.},\n",
      "\tyear = {2010},\n",
      "\tpages = {167--174}\n",
      "}\n",
      "\n",
      "\n",
      "@article{izenman_introduction_2012,\n",
      "\ttitle = {Introduction to manifold learning},\n",
      "\tvolume = {4},\n",
      "\tissn = {1939-0068},\n",
      "\turl = {https://onlinelibrary.wiley.com/doi/abs/10.1002/wics.1222},\n",
      "\tdoi = {10.1002/wics.1222},\n",
      "\tnumber = {5},\n",
      "\tjournal = {WIREs Computational Statistics},\n",
      "\tauthor = {Izenman, Alan Julian},\n",
      "\tyear = {2012},\n",
      "\tpages = {439--446}\n",
      "}\n",
      "\n",
      "@article{kobak_art_2019,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {The art of using t-{SNE} for single-cell transcriptomics},\n",
      "\tvolume = {10},\n",
      "\tissn = {2041-1723},\n",
      "\turl = {https://www.nature.com/articles/s41467-019-13056-x},\n",
      "\tdoi = {10.1038/s41467-019-13056-x},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Communications},\n",
      "\tauthor = {Kobak, Dmitry and Berens, Philipp},\n",
      "\tmonth = nov,\n",
      "\tyear = {2019},\n",
      "\tkeywords = {RNA sequencing, Statistics},\n",
      "\tpages = {5416}\n",
      "}\n",
      "\n",
      "@article{li_application_2017,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Application of t-{SNE} to human genetic data},\n",
      "\tvolume = {15},\n",
      "\tissn = {0219-7200},\n",
      "\turl = {https://www.worldscientific.com/doi/abs/10.1142/S0219720017500172},\n",
      "\tdoi = {10.1142/S0219720017500172},\n",
      "\tnumber = {04},\n",
      "\tjournal = {Journal of Bioinformatics and Computational Biology},\n",
      "\tauthor = {Li, Wentian and Cerise, Jane E. and Yang, Yaning and Han, Henry},\n",
      "\tyear = {2017},\n",
      "\tpages = {1750017}\n",
      "}\n",
      "\n",
      "@misc{mcinnes_umap_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection} for {Dimension} {Reduction}},\n",
      "\tshorttitle = {{UMAP}},\n",
      "\turl = {http://arxiv.org/abs/1802.03426},\n",
      "\tdoi = {10.48550/arXiv.1802.03426},\n",
      "\tpublisher = {arXiv},\n",
      "\tauthor = {McInnes, Leland and Healy, John and Melville, James},\n",
      "\tyear = {2020}\n",
      "}\n",
      "\n",
      "@misc{mcinnes_umapdoc_2018,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {How {UMAP} {Works} — umap 0.5 documentation},\n",
      "\turl = {https://umap-learn.readthedocs.io/en/latest/how_umap_works.html},\n",
      "\turldate = {2023-07-25}\n",
      "}\n",
      "\n",
      "@article{ben-david_clustering_2018,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Clustering - {What} {Both} {Theoreticians} and {Practitioners} are {Doing} {Wrong}},\n",
      "\turl = {http://arxiv.org/abs/1805.08838},\n",
      "\tdoi = {10.48550/arXiv.1805.08838},\n",
      "\tjournal = {arXiv},\n",
      "\tauthor = {Ben-David, Shai},\n",
      "\tyear = {2018}\n",
      "}\n",
      "\n",
      "\n",
      "@article{hartigan_algorithm_1979,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Algorithm {AS} 136: {A} {K}-{Means} {Clustering} {Algorithm}},\n",
      "\tvolume = {28},\n",
      "\tissn = {0035-9254},\n",
      "\turl = {https://www.jstor.org/stable/2346830},\n",
      "\tdoi = {10.2307/2346830},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Journal of the Royal Statistical Society. Series C (Applied Statistics)},\n",
      "\tauthor = {Hartigan, J. A. and Wong, M. A.},\n",
      "\tyear = {1979},\n",
      "\tpages = {100--108}\n",
      "}\n",
      "\n",
      "@article{ding_polygenic_2023,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Polygenic scoring accuracy varies across the genetic ancestry continuum},\n",
      "\tissn = {1476-4687},\n",
      "\turl = {https://www.nature.com/articles/s41586-023-06079-4},\n",
      "\tdoi = {10.1038/s41586-023-06079-4},\n",
      "\tjournal = {Nature},\n",
      "\tauthor = {Ding, Yi and Hou, Kangcheng and Xu, Ziqi and Pimplaskar, Aditya and Petter, Ella and Boulier, Kristin and Privé, Florian and Vilhjálmsson, Bjarni J. and Olde Loohuis, Loes M. and Pasaniuc, Bogdan},\n",
      "\tyear = {2023},\n",
      "\tpages = {1--8}\n",
      "}\n",
      "\n",
      "@article{verity_estimating_2016,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Estimating the {Number} of {Subpopulations} ({K}) in {Structured} {Populations}},\n",
      "\tvolume = {203},\n",
      "\tissn = {0016-6731, 1943-2631},\n",
      "\turl = {https://www.genetics.org/content/203/4/1827},\n",
      "\tdoi = {10.1534/genetics.115.180992},\n",
      "\tnumber = {4},\n",
      "\tjournal = {Genetics},\n",
      "\tauthor = {Verity, Robert and Nichols, Richard A.},\n",
      "\tyear = {2016},\n",
      "\tpmid = {27317680},\n",
      "\tpages = {1827--1839}\n",
      "}\n",
      "\n",
      "@article{lawson_tutorial_2018,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A tutorial on how not to over-interpret {STRUCTURE} and {ADMIXTURE} bar plots},\n",
      "\tvolume = {9},\n",
      "\tissn = {2041-1723},\n",
      "\turl = {https://www.nature.com/articles/s41467-018-05257-7},\n",
      "\tdoi = {10.1038/s41467-018-05257-7},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Communications},\n",
      "\tauthor = {Lawson, Daniel J. and van Dorp, Lucy and Falush, Daniel},\n",
      "\tyear = {2018},\n",
      "\tpages = {3258}\n",
      "}\n",
      "\n",
      "@article{evanno_detecting_2005,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Detecting the number of clusters of individuals using the software structure: a simulation study},\n",
      "\tvolume = {14},\n",
      "\tissn = {1365-294X},\n",
      "\turl = {https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-294X.2005.02553.x},\n",
      "\tdoi = {10.1111/j.1365-294X.2005.02553.x},\n",
      "\tnumber = {8},\n",
      "\tjournal = {Molecular Ecology},\n",
      "\tauthor = {Evanno, G. and Regnaut, S. and Goudet, J.},\n",
      "\tyear = {2005},\n",
      "\tpages = {2611--2620}\n",
      "}\n",
      "\n",
      "@article{pritchard_inference_2000,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Inference of {Population} {Structure} {Using} {Multilocus} {Genotype} {Data}},\n",
      "\tvolume = {155},\n",
      "\tissn = {0016-6731, 1943-2631},\n",
      "\turl = {https://www.genetics.org/content/155/2/945},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Genetics},\n",
      "\tauthor = {Pritchard, Jonathan K. and Stephens, Matthew and Donnelly, Peter},\n",
      "\tyear = {2000},\n",
      "\tpmid = {10835412},\n",
      "\tpages = {945--959}\n",
      "}\n",
      "\n",
      "@article{alexander_fast_2009,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Fast model-based estimation of ancestry in unrelated individuals},\n",
      "\tvolume = {19},\n",
      "\tissn = {1088-9051, 1549-5469},\n",
      "\turl = {https://genome.cshlp.org/content/19/9/1655},\n",
      "\tdoi = {10.1101/gr.094052.109},\n",
      "\tnumber = {9},\n",
      "\tjournal = {Genome Research},\n",
      "\tauthor = {Alexander, David H. and Novembre, John and Lange, Kenneth},\n",
      "\tyear = {2009},\n",
      "\tpmid = {19648217},\n",
      "\tpages = {1655--1664}\n",
      "}\n",
      "\n",
      "\n",
      "@article{tang_estimation_2005,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Estimation of individual admixture: {Analytical} and study design considerations},\n",
      "\tvolume = {28},\n",
      "\tissn = {1098-2272},\n",
      "\tshorttitle = {Estimation of individual admixture},\n",
      "\turl = {https://onlinelibrary.wiley.com/doi/abs/10.1002/gepi.20064},\n",
      "\tdoi = {10.1002/gepi.20064},\n",
      "\tnumber = {4},\n",
      "\tjournal = {Genetic Epidemiology},\n",
      "\tauthor = {Tang, Hua and Peng, Jie and Wang, Pei and Risch, Neil J.},\n",
      "\tyear = {2005},\n",
      "\tpages = {289--301}\n",
      "}\n",
      "\n",
      "@article{frichot_fast_2014,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Fast and {Efficient} {Estimation} of {Individual} {Ancestry} {Coefficients}},\n",
      "\tvolume = {196},\n",
      "\tissn = {1943-2631},\n",
      "\turl = {https://doi.org/10.1534/genetics.113.160572},\n",
      "\tdoi = {10.1534/genetics.113.160572},\n",
      "\tnumber = {4},\n",
      "\tjournal = {Genetics},\n",
      "\tauthor = {Frichot, Eric and Mathieu, François and Trouillon, Théo and Bouchard, Guillaume and François, Olivier},\n",
      "\tyear = {2014},\n",
      "\tpages = {973--983}\n",
      "}\n",
      "\n",
      "\n",
      "@article{gimbernat-mayol_archetypal_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Archetypal {Analysis} for population genetics},\n",
      "\tvolume = {18},\n",
      "\tissn = {1553-7358},\n",
      "\turl = {https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010301},\n",
      "\tdoi = {10.1371/journal.pcbi.1010301},\n",
      "\tnumber = {8},\n",
      "\tjournal = {PLOS Computational Biology},\n",
      "\tauthor = {Gimbernat-Mayol, Julia and Mantes, Albert Dominguez and Bustamante, Carlos D. and Montserrat, Daniel Mas and Ioannidis, Alexander G.},\n",
      "\tyear = {2022},\n",
      "\tpages = {e1010301},\n",
      "}\n",
      "\n",
      "@book{lasker1985surnames,\n",
      "  title={Surnames and genetic structure},\n",
      "  author={Lasker, Gabriel Ward},\n",
      "  number={1},\n",
      "  year={1985},\n",
      "  publisher={Cambridge University Press}\n",
      "}\n",
      "\n",
      "@article{campbell_evolution_2010,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {The {Evolution} of {Human} {Genetic} and {Phenotypic} {Variation} in {Africa}},\n",
      "\tvolume = {20},\n",
      "\tissn = {0960-9822},\n",
      "\turl = {https://www.sciencedirect.com/science/article/pii/S096098220902065X},\n",
      "\tdoi = {10.1016/j.cub.2009.11.050},\n",
      "\tnumber = {4},\n",
      "\tjournal = {Current Biology},\n",
      "\tauthor = {Campbell, Michael C. and Tishkoff, Sarah A.},\n",
      "\tyear = {2010},\n",
      "\tpages = {R166--R173}\n",
      "}\n",
      "\n",
      "@inproceedings{malzer_hybrid_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A {Hybrid} {Approach} {To} {Hierarchical} {Density}-based {Cluster} {Selection}},\n",
      "\tdoi = {10.1109/MFI49285.2020.9235263},\n",
      "\tbooktitle = {2020 {IEEE} {International} {Conference} on {Multisensor} {Fusion} and {Integration} for {Intelligent} {Systems} ({MFI})},\n",
      "\tauthor = {Malzer, Claudia and Baum, Marcus},\n",
      "\tyear = {2020},\n",
      "\tpages = {223--228}\n",
      "}\n",
      "\n",
      "@book{hartl2007principles_2,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Principles of population genetics},\n",
      "  author={Hartl, Daniel L and Clark, Andrew G},\n",
      "  volume={116},\n",
      "  year={2007},\n",
      "  ed={4},\n",
      "  chapter={2},\n",
      "  publisher={Sinauer associates Sunderland, MA}\n",
      "}\n",
      "\n",
      "@book{hartl2007principles_3,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Principles of population genetics},\n",
      "  author={Hartl, Daniel L and Clark, Andrew G},\n",
      "  volume={116},\n",
      "  year={2007},\n",
      "  ed={4},\n",
      "  chapter={3},\n",
      "  publisher={Sinauer associates Sunderland, MA}\n",
      "}\n",
      "\n",
      "@book{hartl2007principles_6,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Principles of population genetics},\n",
      "  author={Hartl, Daniel L and Clark, Andrew G},\n",
      "  volume={116},\n",
      "  year={2007},\n",
      "  ed={4},\n",
      "  chapter={6},\n",
      "  publisher={Sinauer associates Sunderland, MA}\n",
      "}\n",
      "\n",
      "@article{robinson_genetic_2017,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Genetic evidence of assortative mating in humans},\n",
      "\tvolume = {1},\n",
      "\tissn = {2397-3374},\n",
      "\turl = {https://www.nature.com/articles/s41562-016-0016},\n",
      "\tdoi = {10.1038/s41562-016-0016},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Human Behaviour},\n",
      "\tauthor = {Robinson, Matthew R. and Kleinman, Aaron and Graff, Mariaelisa and Vinkhuyzen, Anna A. E. and Couper, David and Miller, Michael B. and Peyrot, Wouter J. and Abdellaoui, Abdel and Zietsch, Brendan P. and Nolte, Ilja M. and van Vliet-Ostaptchouk, Jana V. and Snieder, Harold and Medland, Sarah E. and Martin, Nicholas G. and Magnusson, Patrik K. E. and Iacono, William G. and McGue, Matt and North, Kari E. and Yang, Jian and Visscher, Peter M.},\n",
      "\tyear = {2017},\n",
      "\tpages = {1--13}\n",
      "}\n",
      "\n",
      "\n",
      "@article{peter_geometric_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A geometric relationship of {F2}, {F3} and {F4}-statistics with principal component analysis},\n",
      "\tvolume = {377},\n",
      "\turl = {https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0413},\n",
      "\tdoi = {10.1098/rstb.2020.0413},\n",
      "\tnumber = {1852},\n",
      "\tjournal = {Philosophical Transactions of the Royal Society B: Biological Sciences},\n",
      "\tauthor = {Peter, Benjamin M.},\n",
      "\tyear = {2022},\n",
      "\tpages = {20200413}\n",
      "}\n",
      "\n",
      "@book{crow_kimura_1970_3,\n",
      "\tkeywords = {mainbib},\n",
      "  title={An Introduction to Population Genetics Theory},\n",
      "  author={Crow, JF and Kimura, M},\n",
      "  year={1970},\n",
      "  chapter={3},\n",
      "  publisher={Harper \\& Row, NY}\n",
      "}\n",
      "\n",
      "@book{crow_kimura_1970_9,\n",
      "\tkeywords = {mainbib},\n",
      "  title={An Introduction to Population Genetics Theory},\n",
      "  author={Crow, JF and Kimura, M},\n",
      "  year={1970},\n",
      "  chapter={9},\n",
      "  publisher={Harper \\& Row, NY}\n",
      "}\n",
      "\n",
      "@book{crow_kimura_1970,\n",
      "  title={An Introduction to Population Genetics Theory},\n",
      "  author={Crow, JF and Kimura, M},\n",
      "  year={1970},\n",
      "  publisher={Harper \\& Row, NY}\n",
      "}\n",
      "\n",
      "\n",
      "@article{wright_isolation_1943,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Isolation by {Distance}},\n",
      "\tvolume = {28},\n",
      "\tissn = {0016-6731},\n",
      "\turl = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1209196/},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Genetics},\n",
      "\tauthor = {Wright, Sewall},\n",
      "\tyear = {1943},\n",
      "\tpmid = {17247074},\n",
      "\tpmcid = {PMC1209196},\n",
      "\tpages = {114--138}\n",
      "}\n",
      "\n",
      "@book{statcan2003,\n",
      "\tkeywords = {mainbib},\n",
      "  title = {Survey methods and practices},\n",
      "  year = 2003,\n",
      "  editor = {Sarah Franklin and Charlene Walker},\n",
      "  publisher = {Statistics Canada},\n",
      "  url={https://www150.statcan.gc.ca/n1/pub/12-587-x/12-587-x2003001-eng.pdf},\n",
      "  isbn={9781100164106}\n",
      "}\n",
      "\n",
      "@article{barbujani_zones_1990,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Zones of sharp genetic change in {Europe} are also linguistic boundaries.},\n",
      "\tvolume = {87},\n",
      "\turl = {https://www.pnas.org/doi/abs/10.1073/pnas.87.5.1816},\n",
      "\tdoi = {10.1073/pnas.87.5.1816},\n",
      "\tnumber = {5},\n",
      "\tjournal = {Proceedings of the National Academy of Sciences},\n",
      "\tauthor = {Barbujani, G and Sokal, R R},\n",
      "\tyear = {1990},\n",
      "\tpages = {1816--1819}\n",
      "}\n",
      "\n",
      "\n",
      "@article{ragsdale_weakly_2023,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A weakly structured stem for human origins in {Africa}},\n",
      "\tvolume = {617},\n",
      "\tissn = {1476-4687},\n",
      "\turl = {https://www.nature.com/articles/s41586-023-06055-y},\n",
      "\tdoi = {10.1038/s41586-023-06055-y},\n",
      "\tnumber = {7962},\n",
      "\tjournal = {Nature},\n",
      "\tauthor = {Ragsdale, Aaron P. and Weaver, Timothy D. and Atkinson, Elizabeth G. and Hoal, Eileen G. and Möller, Marlo and Henn, Brenna M. and Gravel, Simon},\n",
      "\tyear = {2023},\n",
      "\tpages = {755--763}\n",
      "}\n",
      "\n",
      "@article{zaidi_demographic_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Demographic history mediates the effect of stratification on polygenic scores},\n",
      "\tvolume = {9},\n",
      "\tissn = {2050-084X},\n",
      "\turl = {https://doi.org/10.7554/eLife.61548},\n",
      "\tdoi = {10.7554/eLife.61548},\n",
      "\tjournal = {eLife},\n",
      "\tauthor = {Zaidi, Arslan A and Mathieson, Iain},\n",
      "\tyear = {2020},\n",
      "\tpages = {e61548}\n",
      "}\n",
      "\n",
      "@article{gopalan_human_2022,\n",
      "\ttitle = {Human genetic admixture through the lens of population genomics},\n",
      "\tvolume = {377},\n",
      "\turl = {https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0410},\n",
      "\tdoi = {10.1098/rstb.2020.0410},\n",
      "\tnumber = {1852},\n",
      "\tjournal = {Philosophical Transactions of the Royal Society B: Biological Sciences},\n",
      "\tauthor = {Gopalan, Shyamalika and Smith, Samuel Pattillo and Korunes, Katharine and Hamid, Iman and Ramachandran, Sohini and Goldberg, Amy},\n",
      "\tyear = {2022},\n",
      "\tpages = {20200410}\n",
      "}\n",
      "\n",
      "@article{berg_reduced_2019,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Reduced signal for polygenic adaptation of height in {UK} {Biobank}},\n",
      "\tvolume = {8},\n",
      "\tissn = {2050-084X},\n",
      "\turl = {https://doi.org/10.7554/eLife.39725},\n",
      "\tdoi = {10.7554/eLife.39725},\n",
      "\tjournal = {eLife},\n",
      "\tauthor = {Berg, Jeremy J and Harpak, Arbel and Sinnott-Armstrong, Nasa and Joergensen, Anja Moltke and Mostafavi, Hakhamanesh and Field, Yair and Boyle, Evan August and Zhang, Xinjun and Racimo, Fernando and Pritchard, Jonathan K and Coop, Graham},\n",
      "\tyear = {2019},\n",
      "\tpages = {e39725}\n",
      "}\n",
      "\n",
      "@article{kaplan_polygenic_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Polygenic risk, population structure and ongoing difficulties with race in human genetics},\n",
      "\tvolume = {377},\n",
      "\turl = {https://royalsocietypublishing.org/doi/10.1098/rstb.2020.0427},\n",
      "\tdoi = {10.1098/rstb.2020.0427},\n",
      "\tnumber = {1852},\n",
      "\tjournal = {Philosophical Transactions of the Royal Society B: Biological Sciences},\n",
      "\tauthor = {Kaplan, Jonathan Michael and Fullerton, Stephanie M.},\n",
      "\tyear = {2022},\n",
      "\tpages = {20200427}\n",
      "}\n",
      "\n",
      "@article{kerminen_geographic_2019,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Geographic {Variation} and {Bias} in the {Polygenic} {Scores} of {Complex} {Diseases} and {Traits} in {Finland}},\n",
      "\tvolume = {104},\n",
      "\tissn = {0002-9297},\n",
      "\turl = {https://www.sciencedirect.com/science/article/pii/S0002929719301879},\n",
      "\tdoi = {10.1016/j.ajhg.2019.05.001},\n",
      "\tnumber = {6},\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\tauthor = {Kerminen, Sini and Martin, Alicia R. and Koskela, Jukka and Ruotsalainen, Sanni E. and Havulinna, Aki S. and Surakka, Ida and Palotie, Aarno and Perola, Markus and Salomaa, Veikko and Daly, Mark J. and Ripatti, Samuli and Pirinen, Matti},\n",
      "\tyear = {2019},\n",
      "\tpages = {1169--1181}\n",
      "}\n",
      "\n",
      "@article{rose_sick_2001,\n",
      "\ttitle = {Sick individuals and sick populations},\n",
      "\tvolume = {30},\n",
      "\tissn = {0300-5771},\n",
      "\turl = {https://doi.org/10.1093/ije/30.3.427},\n",
      "\tdoi = {10.1093/ije/30.3.427},\n",
      "\tnumber = {3},\n",
      "\tjournal = {International Journal of Epidemiology},\n",
      "\tauthor = {Rose, Geoffrey},\n",
      "\tyear = {2001},\n",
      "\tpages = {427--432},\n",
      "\tnote = {[Reprinted from 14.1 (1985)]}\n",
      "}\n",
      "@article{arbisser_fst_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\tseries = {Fifty years of {Theoretical} {Population} {Biology}},\n",
      "\ttitle = {{FST} and the triangle inequality for biallelic markers},\n",
      "\tvolume = {133},\n",
      "\tissn = {0040-5809},\n",
      "\turl = {https://www.sciencedirect.com/science/article/pii/S0040580918302041},\n",
      "\tdoi = {10.1016/j.tpb.2019.05.003},\n",
      "\tjournal = {Theoretical Population Biology},\n",
      "\tauthor = {Arbisser, Ilana M. and Rosenberg, Noah A.},\n",
      "\tyear = {2020},\n",
      "\tpages = {117--129}\n",
      "}\n",
      "\n",
      "@article{Maples:2013fia,\n",
      "author = {Maples, Brian K and Gravel, Simon and Kenny, Eimear E and Bustamante, Carlos D},\n",
      "title = {{RFM}ix: A Discriminative Modeling Approach for Rapid and Robust Local-Ancestry Inference.},\n",
      "journal = {Am J Hum Genet},\n",
      "year = {2013},\n",
      "volume = {93},\n",
      "number = {2},\n",
      "pages = {278--288}\n",
      "}\n",
      "\n",
      "@article{nelson2008population,\n",
      "  title={The {P}opulation {R}eference {S}ample, {POPRES}: a resource for population, disease, and pharmacological genetics research},\n",
      "  author={Nelson, Matthew R and Bryc, Katarzyna and King, Karen S and Indap, Amit and Boyko, Adam R and Novembre, John and Briley, Linda P and Maruyama, Yuka and Waterworth, Dawn M and Waeber, G{\\'e}rard and others},\n",
      "  journal={The American Journal of Human Genetics},\n",
      "  volume={83},\n",
      "  number={3},\n",
      "  pages={347--358},\n",
      "  year={2008}\n",
      "}\n",
      "\n",
      "\n",
      "@article{mcvean2009genealogical,\n",
      "\tkeywords = {mainbib},\n",
      "  title={A genealogical interpretation of principal components analysis},\n",
      "  author={McVean, Gil},\n",
      "  journal={PLoS genetics},\n",
      "  volume={5},\n",
      "  number={10},\n",
      "  pages={e1000686},\n",
      "  year={2009}\n",
      "}\n",
      "\n",
      "@article{li2017tsne,\n",
      "\tAuthor = {Li, Wentian and Cerise, Jane E. and Yang, Yaning and Han, Henry},\n",
      "\tDoi = {10.1142/S0219720017500172},\n",
      "\tEprint = {http://www.worldscientific.com/doi/pdf/10.1142/S0219720017500172},\n",
      "\tJournal = {Journal of Bioinformatics and Computational Biology},\n",
      "\tNote = {PMID: 28718343},\n",
      "\tNumber = {04},\n",
      "\tPages = {1750017},\n",
      "\tTitle = {Application of t-{SNE} to human genetic data},\n",
      "\tUrl = {http://www.worldscientific.com/doi/abs/10.1142/S0219720017500172},\n",
      "\tVolume = {15},\n",
      "\tYear = {2017}\n",
      "\t}\n",
      "\n",
      "@article{platzer2013visualization,\n",
      "  title={Visualization of {SNP}s with t-{SNE}},\n",
      "  author={Platzer, Alexander},\n",
      "  journal={PloS one},\n",
      "  volume={8},\n",
      "  number={2},\n",
      "  pages={e56883},\n",
      "  year={2013}\n",
      "}\n",
      "\n",
      "@article{10002015global,\n",
      "  title={A global reference for human genetic variation},\n",
      "  author={{1000 Genomes Project Consortium}},\n",
      "  journal={Nature},\n",
      "  volume={526},\n",
      "  number={7571},\n",
      "  pages={68},\n",
      "  year={2015}\n",
      "}\n",
      "\n",
      "@article{sudlow2015uk,\n",
      "  title={{UK} biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age},\n",
      "  author={Sudlow, Cathie and Gallacher, John and Allen, Naomi and Beral, Valerie and Burton, Paul and Danesh, John and Downey, Paul and Elliott, Paul and Green, Jane and Landray, Martin and others},\n",
      "  journal={PLoS medicine},\n",
      "  volume={12},\n",
      "  number={3},\n",
      "  pages={e1001779},\n",
      "  year={2015}\n",
      "}\n",
      "\n",
      "@article{scikit-learn,\n",
      " title={Scikit-learn: Machine Learning in {P}ython},\n",
      " author={Pedregosa, F. and Varoquaux, G. and Gramfort, A. and Michel, V.\n",
      "         and Thirion, B. and Grisel, O. and Blondel, M. and Prettenhofer, P.\n",
      "         and Weiss, R. and Dubourg, V. and Vanderplas, J. and Passos, A. and\n",
      "         Cournapeau, D. and Brucher, M. and Perrot, M. and Duchesnay, E.},\n",
      " journal={Journal of Machine Learning Research},\n",
      " volume={12},\n",
      " pages={2825--2830},\n",
      " year={2011}\n",
      "}\n",
      "\n",
      "@article{purcell2007plink,\n",
      "  title={{PLINK}: a tool set for whole-genome association and population-based linkage analyses},\n",
      "  author={Purcell, Shaun and Neale, Benjamin and Todd-Brown, Kathe and Thomas, Lori and Ferreira, Manuel AR and Bender, David and Maller, Julian and Sklar, Pamela and De Bakker, Paul IW and Daly, Mark J and others},\n",
      "  journal={The American Journal of Human Genetics},\n",
      "  volume={81},\n",
      "  number={3},\n",
      "  pages={559--575},\n",
      "  year={2007}\n",
      "}\n",
      "\n",
      "@software{plink,\n",
      "  author={Purcell, Shaun},\n",
      "  version={1.90b5.2 64-bit},\n",
      "  title={PLINK},\n",
      "  url={http://pngu.mgh.harvard.edu/purcell/plink/}\n",
      "}\n",
      "\n",
      "@article{eigen2006,\n",
      "    author = {Patterson, Nick AND Price, Alkes L AND Reich, David},\n",
      "    journal = {PLOS Genetics},\n",
      "    title = {Population Structure and Eigenanalysis},\n",
      "    year = {2006},\n",
      "    month = {12},\n",
      "    volume = {2},\n",
      "    url = {https://doi.org/10.1371/journal.pgen.0020190},\n",
      "    pages = {1-20},\n",
      "    doi = {10.1371/journal.pgen.0020190}\n",
      "}\n",
      "\n",
      "@article{2018arXivUMAP,\n",
      "     author = {{McInnes}, L. and {Healy}, J.},\n",
      "     title = {{UMAP}: Uniform Manifold Approximation and Projection for Dimension Reduction},\n",
      "     journal={arXiv preprint arXiv:1802.03426},\n",
      "     archivePrefix = \"arXiv\",\n",
      "     eprint = {1802.03426},\n",
      "     year = 2018\n",
      "}\n",
      "\n",
      "@article{alexander2009fast,\n",
      "  title={Fast model-based estimation of ancestry in unrelated individuals},\n",
      "  author={Alexander, David H and Novembre, John and Lange, Kenneth},\n",
      "  journal={Genome research},\n",
      "  volume={19},\n",
      "  number={9},\n",
      "  pages={1655--1664},\n",
      "  year={2009}\n",
      "}\n",
      "\n",
      "@article{baharian2016great,\n",
      "  title={The great migration and {A}frican-{A}merican genomic diversity},\n",
      "  author={Baharian, Soheil and Barakatt, Maxime and Gignoux, Christopher R and Shringarpure, Suyash and Errington, Jacob and Blot, William J and Bustamante, Carlos D and Kenny, Eimear E and Williams, Scott M and Aldrich, Melinda C and others},\n",
      "  journal={PLoS genetics},\n",
      "  volume={12},\n",
      "  number={5},\n",
      "  pages={e1006059},\n",
      "  year={2016}\n",
      "}\n",
      "\n",
      "@article{price2006principal,\n",
      "  title={Principal components analysis corrects for stratification in genome-wide association studies},\n",
      "  author={Price, Alkes L and Patterson, Nick J and Plenge, Robert M and Weinblatt, Michael E and Shadick, Nancy A and Reich, David},\n",
      "  journal={Nature genetics},\n",
      "  volume={38},\n",
      "  number={8},\n",
      "  pages={904},\n",
      "  year={2006}\n",
      "}\n",
      "\n",
      "@article{brisbin2012pcadmix,\n",
      "  title={{PCA}dmix: principal components-based assignment of ancestry along each chromosome in individuals with admixed ancestry from two or more populations},\n",
      "  author={Brisbin, Abra and Bryc, Katarzyna and Byrnes, Jake and Zakharia, Fouad and Omberg, Larsson and Degenhardt, Jeremiah and Reynolds, Andrew and Ostrer, Harry and Mezey, Jason G and Bustamante, Carlos D},\n",
      "  journal={Human biology},\n",
      "  volume={84},\n",
      "  number={4},\n",
      "  pages={343},\n",
      "  year={2012}\n",
      "}\n",
      "\n",
      "@article{juster1995overview,\n",
      "  title={An overview of the {H}ealth and {R}etirement {S}tudy},\n",
      "  author={Juster, F Thomas and Suzman, Richard},\n",
      "  journal={Journal of Human Resources},\n",
      "  pages={S7--S56},\n",
      "  year={1995}\n",
      "}\n",
      "\n",
      "@misc{plotly, \n",
      "\tkeywords = {mainbib},\n",
      "    author = {{Plotly Technologies Inc.}},\n",
      "    title = {Collaborative data science},\n",
      "    publisher = {Plotly Technologies Inc.},\n",
      "    address = {Montreal, QC},\n",
      "    year = {2015},\n",
      "    url = {https://plot.ly}\n",
      "} \n",
      "\n",
      "@misc{python2015,\n",
      "    title = {Python: A dynamic, open source programming language},\n",
      "    author = {Python Core Team},\n",
      "    publisher = {Python Software Foundation},\n",
      "    year = {2015},\n",
      "    url= {https://www.python.org/}\n",
      "}\n",
      "\n",
      "@misc{Ulyanov2016,\n",
      "  author = {Ulyanov, Dmitry},\n",
      "  title = {Multicore-TSNE},\n",
      "  year = {2016},\n",
      "  publisher = {GitHub},\n",
      "  journal = {GitHub repository},\n",
      "  howpublished = {\\url{https://github.com/DmitryUlyanov/Multicore-TSNE}},\n",
      "}\n",
      "\n",
      "@Manual{bokeh2014,\n",
      "title = {Bokeh: {P}ython library for interactive visualization},\n",
      "author = {{Bokeh Development Team}},\n",
      "year = {2014},\n",
      "url = {http://www.bokeh.pydata.org},\n",
      "}\n",
      "\n",
      "@Article{Hunter2007,\n",
      "  Author    = {Hunter, J. D.},\n",
      "  Title     = {Matplotlib: A 2D graphics environment},\n",
      "  Journal   = {Computing In Science \\& Engineering},\n",
      "  Volume    = {9},\n",
      "  Number    = {3},\n",
      "  Pages     = {90--95},\n",
      "  abstract  = {Matplotlib is a 2D graphics package used for Python\n",
      "  for application development, interactive scripting, and\n",
      "  publication-quality image generation across user\n",
      "  interfaces and operating systems.},\n",
      "  publisher = {IEEE COMPUTER SOC},\n",
      "  doi       = {10.1109/MCSE.2007.55},\n",
      "  year      = 2007\n",
      "}\n",
      "\n",
      "@article {Jordan333609,\n",
      "\tauthor = {Jordan, I.King and Rishishwar, Lavanya and Conley, Andrew B},\n",
      "\ttitle = {Cryptic {N}ative {A}merican ancestry recapitulates population-specific migration and settlement of the continental {U}nited {S}tates},\n",
      "\tyear = {2018},\n",
      "\tdoi = {10.1101/333609},\n",
      "\tpublisher = {Cold Spring Harbor Laboratory},\n",
      "\tURL = {https://www.biorxiv.org/content/early/2018/05/30/333609},\n",
      "\teprint = {https://www.biorxiv.org/content/early/2018/05/30/333609.full.pdf},\n",
      "\tjournal = {bioRxiv}\n",
      "}\n",
      "\n",
      "@article{novembre2008interpreting,\n",
      "  title={Interpreting principal component analyses of spatial population genetic variation},\n",
      "  author={Novembre, John and Stephens, Matthew},\n",
      "  journal={Nature genetics},\n",
      "  volume={40},\n",
      "  number={5},\n",
      "  pages={646},\n",
      "  year={2008}\n",
      "}\n",
      "\n",
      "@inproceedings{seabold2010statsmodels,\n",
      "  title={Statsmodels: Econometric and statistical modeling with python},\n",
      "  author={Seabold, Skipper and Perktold, Josef},\n",
      "  booktitle={9th Python in Science Conference},\n",
      "  year={2010},\n",
      "}\n",
      "\n",
      "@Misc{scipy,\n",
      "  author =    {Eric Jones and Travis Oliphant and Pearu Peterson and others},\n",
      "  title =     {{SciPy}: Open source scientific tools for {Python}},\n",
      "  year =      {2001},\n",
      "  url = {http://www.scipy.org/},\n",
      "  note = {[Online; accessed 2018-02-02]}\n",
      "}\n",
      "\n",
      "@article{robinson2015population,\n",
      "  title={Population genetic differentiation of height and body mass index across {E}urope},\n",
      "  author={Robinson, Matthew R and Hemani, Gibran and Medina-Gomez, Carolina and Mezzavilla, Massimo and Esko, Tonu and Shakhbazov, Konstantin and Powell, Joseph E and Vinkhuyzen, Anna and Berndt, Sonja I and Gustafsson, Stefan and others},\n",
      "  journal={Nature genetics},\n",
      "  volume={47},\n",
      "  number={11},\n",
      "  pages={1357},\n",
      "  year={2015}\n",
      "}\n",
      "\n",
      "@book{komlos1994stature,\n",
      "  title={Stature, living standards, and economic development: Essays in anthropometric history},\n",
      "  author={Komlos, AU},\n",
      "  year={1994},\n",
      "  page={52}\n",
      "}\n",
      "\n",
      "@article{Laayouni2668,\n",
      "\tAuthor = {Laayouni, Hafid and Oosting, Marije and Luisi, Pierre and Ioana, Mihai and Alonso, Santos and Rica{\\~n}o-Ponce, Isis and Trynka, Gosia and Zhernakova, Alexandra and Plantinga, Theo S. and Cheng, Shih-Chin and van der Meer, Jos W. M. and Popp, Radu and Sood, Ajit and Thelma, B. K. and Wijmenga, Cisca and Joosten, Leo A. B. and Bertranpetit, Jaume and Netea, Mihai G.},\n",
      "\tJournal = {Proceedings of the National Academy of Sciences},\n",
      "\tNumber = {7},\n",
      "\tPages = {2668--2673},\n",
      "\tTitle = {Convergent evolution in {E}uropean and {R}roma populations reveals pressure exerted by plague on Toll-like receptors},\n",
      "\tVolume = {111},\n",
      "\tYear = {2014}}\n",
      "\n",
      "@article{Hellenthal747,\n",
      "\tAuthor = {Hellenthal, Garrett and Busby, George B. J. and Band, Gavin and Wilson, James F. and Capelli, Cristian and Falush, Daniel and Myers, Simon},\n",
      "\tJournal = {Science},\n",
      "\tNumber = {6172},\n",
      "\tPages = {747--751},\n",
      "\tTitle = {A Genetic Atlas of Human Admixture History},\n",
      "\tVolume = {343},\n",
      "\tYear = {2014}}\n",
      "\n",
      "@article{lawson2012inference,\n",
      "  title={Inference of population structure using dense haplotype data},\n",
      "  author={Lawson, Daniel John and Hellenthal, Garrett and Myers, Simon and Falush, Daniel},\n",
      "  journal={PLoS genetics},\n",
      "  volume={8},\n",
      "  number={1},\n",
      "  pages={e1002453},\n",
      "  year={2012}\n",
      "}\n",
      "\n",
      "@article{leslie2015fine,\n",
      "  title={The fine-scale genetic structure of the {B}ritish population},\n",
      "  author={Leslie, Stephen and Winney, Bruce and Hellenthal, Garrett and Davison, Dan and Boumertit, Abdelhamid and Day, Tammy and Hutnik, Katarzyna and Royrvik, Ellen C and Cunliffe, Barry and Lawson, Daniel J and others},\n",
      "  journal={Nature},\n",
      "  volume={519},\n",
      "  number={7543},\n",
      "  pages={309},\n",
      "  year={2015}\n",
      "}\n",
      "\n",
      "@article{novembre2016recent,\n",
      "  title={Recent advances in the study of fine-scale population structure in humans},\n",
      "  author={Novembre, John and Peter, Benjamin M},\n",
      "  journal={Current opinion in genetics \\& development},\n",
      "  volume={41},\n",
      "  pages={98--105},\n",
      "  year={2016}\n",
      "}\n",
      "@article{spence2018inference,\n",
      "  title={Inference of population history using coalescent HMMs: review and outlook},\n",
      "  author={Spence, Jeffrey P and Steinr{\\\"u}cken, Matthias and Terhorst, Jonathan and Song, Yun S},\n",
      "  journal={Current opinion in genetics \\& development},\n",
      "  volume={53},\n",
      "  pages={70--76},\n",
      "  year={2018}\n",
      "}\n",
      "@article{umap2018singlecell,\n",
      "  title={Dimensionality reduction for visualizing single-cell data using UMAP},\n",
      "  author={Becht, Etienne and McInnes, Leland and Healy, John and Dutertre, Charles-Antoine and Kwok, Immanuel WH and Ng, Lai Guan and Ginhoux, Florent and Newell, Evan W},\n",
      "  journal={Nature biotechnology},\n",
      "  volume={37},\n",
      "  number={1},\n",
      "  pages={38--44},\n",
      "  year={2019}\n",
      "}\n",
      "\n",
      "@article{reich2009india,\n",
      "  title={Reconstructing Indian population history},\n",
      "  author={Reich, David and Thangaraj, Kumarasamy and Patterson, Nick and Price, Alkes L and Singh, Lalji},\n",
      "  journal={Nature},\n",
      "  volume={461},\n",
      "  number={7263},\n",
      "  pages={489--494},\n",
      "  year={2009},\n",
      "  publisher={Nature Publishing Group UK London}\n",
      "}\n",
      "\n",
      "\n",
      "@inproceedings{mcinnes_topological_2019,\n",
      "\taddress = {Cham},\n",
      "\tseries = {Lecture {Notes} in {Computer} {Science}},\n",
      "\ttitle = {Topological {Methods} for {Unsupervised} {Learning}},\n",
      "\tisbn = {978-3-030-26980-7},\n",
      "\tdoi = {10.1007/978-3-030-26980-7_35},\n",
      "\tbooktitle = {Geometric {Science} of {Information}},\n",
      "\tauthor = {McInnes, Leland},\n",
      "\teditor = {Nielsen, Frank and Barbaresco, Frédéric},\n",
      "\tyear = {2019},\n",
      "\tpages = {343--350}\n",
      "}\n",
      "\n",
      "\n",
      "@inproceedings{khan_dbscan_2014,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {{DBSCAN}: {Past}, present and future},\n",
      "\tdoi = {10.1109/ICADIWT.2014.6814687},\n",
      "\tbooktitle = {The {Fifth} {International} {Conference} on the {Applications} of {Digital} {Information} and {Web} {Technologies} ({ICADIWT} 2014)},\n",
      "\tauthor = {Khan, Kamran and Rehman, Saif Ur and Aziz, Kamran and Fong, Simon and Sarasvady, S.},\n",
      "\tyear = {2014},\n",
      "\tpages = {232--238}\n",
      "}\n",
      "\n",
      "@inproceedings{ester1996density,\n",
      "\tkeywords = {mainbib},\n",
      "  title={A density-based algorithm for discovering clusters in large spatial databases with noise},\n",
      "  author={Ester, Martin and Kriegel, Hans-Peter and Sander, J{\\\"o}rg and Xu, Xiaowei and others},\n",
      "  booktitle={Proceedings of the 2nd {I}nternational {C}onference on {K}nowledge {D}iscovery and {D}ata {M}ining},\n",
      "  volume={96},\n",
      "  number={34},\n",
      "  pages={226--231},\n",
      "  year={1996}\n",
      "}\n",
      "\n",
      "\n",
      "@article{benonisdottir_studying_2023,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Studying the genetics of participation using footprints left on the ascertained genotypes},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-023-01439-2},\n",
      "\tdoi = {10.1038/s41588-023-01439-2},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Benonisdottir, Stefania and Kong, Augustine},\n",
      "\tyear = {2023},\n",
      "\tpages = {1--8}\n",
      "}\n",
      "\n",
      "\n",
      "@inproceedings{campello_density-based_2013,\n",
      "\tkeywords = {mainbib},\n",
      "\taddress = {Berlin, Heidelberg},\n",
      "\tseries = {Lecture {Notes} in {Computer} {Science}},\n",
      "\ttitle = {Density-{Based} {Clustering} {Based} on {Hierarchical} {Density} {Estimates}},\n",
      "\tisbn = {978-3-642-37456-2},\n",
      "\tdoi = {10.1007/978-3-642-37456-2_14},\n",
      "\tbooktitle = {Advances in {Knowledge} {Discovery} and {Data} {Mining}},\n",
      "\tpublisher = {Springer},\n",
      "\tauthor = {Campello, Ricardo J. G. B. and Moulavi, Davoud and Sander, Joerg},\n",
      "\teditor = {Pei, Jian and Tseng, Vincent S. and Cao, Longbing and Motoda, Hiroshi and Xu, Guandong},\n",
      "\tyear = {2013},\n",
      "\tpages = {160--172}\n",
      "}\n",
      "\n",
      "@misc{herrmann_enhancing_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Enhancing cluster analysis via topological manifold learning},\n",
      "\turl = {http://arxiv.org/abs/2207.00510},\n",
      "\tpublisher = {arXiv},\n",
      "\tauthor = {Herrmann, Moritz and Kazempour, Daniyal and Scheipl, Fabian and Kröger, Peer},\n",
      "\tyear = {2022}\n",
      "}\n",
      "\n",
      "@article{bhatia_estimating_2013,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Estimating and interpreting {FST}: {The} impact of rare variants},\n",
      "\tvolume = {23},\n",
      "\tissn = {1088-9051, 1549-5469},\n",
      "\tshorttitle = {Estimating and interpreting {FST}},\n",
      "\turl = {https://genome.cshlp.org/content/23/9/1514},\n",
      "\tdoi = {10.1101/gr.154831.113},\n",
      "\tnumber = {9},\n",
      "\tjournal = {Genome Research},\n",
      "\tauthor = {Bhatia, Gaurav and Patterson, Nick and Sankararaman, Sriram and Price, Alkes L.},\n",
      "\tyear = {2013},\n",
      "\tpmid = {23861382},\n",
      "\tpages = {1514--1521}\n",
      "}\n",
      "\n",
      "\n",
      "@article{uffelmann_genome-wide_2021,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Genome-wide association studies},\n",
      "\tvolume = {1},\n",
      "\tissn = {2662-8449},\n",
      "\turl = {https://www.nature.com/articles/s43586-021-00056-9},\n",
      "\tdoi = {10.1038/s43586-021-00056-9},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Reviews Methods Primers},\n",
      "\tauthor = {Uffelmann, Emil and Huang, Qin Qin and Munung, Nchangwi Syntia and de Vries, Jantina and Okada, Yukinori and Martin, Alicia R. and Martin, Hilary C. and Lappalainen, Tuuli and Posthuma, Danielle},\n",
      "\tyear = {2021},\n",
      "\tpages = {1--21}\n",
      "}\n",
      "\n",
      "@article{wang_theoretical_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Theoretical and empirical quantification of the accuracy of polygenic scores in ancestry divergent populations},\n",
      "\tvolume = {11},\n",
      "\tissn = {2041-1723},\n",
      "\turl = {https://www.nature.com/articles/s41467-020-17719-y},\n",
      "\tdoi = {10.1038/s41467-020-17719-y},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Communications},\n",
      "\tauthor = {Wang, Ying and Guo, Jing and Ni, Guiyan and Yang, Jian and Visscher, Peter M. and Yengo, Loic},\n",
      "\tyear = {2020},\n",
      "\tpages = {3865}\n",
      "}\n",
      "\n",
      "@article{zhou_global_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Global {Biobank} {Meta}-analysis {Initiative}: {Powering} genetic discovery across human disease},\n",
      "\tvolume = {2},\n",
      "\tissn = {2666-979X},\n",
      "\tshorttitle = {Global {Biobank} {Meta}-analysis {Initiative}},\n",
      "\turl = {https://www.sciencedirect.com/science/article/pii/S2666979X22001410},\n",
      "\tdoi = {10.1016/j.xgen.2022.100192},\n",
      "\tnumber = {10},\n",
      "\tjournal = {Cell Genomics},\n",
      "\tauthor = {Zhou, Wei and Kanai, Masahiro and Wu, Kuan-Han H. and Rasheed, Humaira and Tsuo, Kristin and Hirbo, Jibril B. and Wang, Ying and Bhattacharya, Arjun and Zhao, Huiling and Namba, Shinichi and Surakka, Ida and Wolford, Brooke N. and Lo Faro, Valeria and Lopera-Maya, Esteban A. and Läll, Kristi and Favé, Marie-Julie and Partanen, Juulia J. and Chapman, Sinéad B. and Karjalainen, Juha and Kurki, Mitja and Maasha, Mutaamba and Brumpton, Ben M. and Chavan, Sameer and Chen, Tzu-Ting and Daya, Michelle and Ding, Yi and Feng, Yen-Chen A. and Guare, Lindsay A. and Gignoux, Christopher R. and Graham, Sarah E. and Hornsby, Whitney E. and Ingold, Nathan and Ismail, Said I. and Johnson, Ruth and Laisk, Triin and Lin, Kuang and Lv, Jun and Millwood, Iona Y. and Moreno-Grau, Sonia and Nam, Kisung and Palta, Priit and Pandit, Anita and Preuss, Michael H. and Saad, Chadi and Setia-Verma, Shefali and Thorsteinsdottir, Unnur and Uzunovic, Jasmina and Verma, Anurag and Zawistowski, Matthew and Zhong, Xue and Afifi, Nahla and Al-Dabhani, Kawthar M. and Al Thani, Asma and Bradford, Yuki and Campbell, Archie and Crooks, Kristy and de Bock, Geertruida H. and Damrauer, Scott M. and Douville, Nicholas J. and Finer, Sarah and Fritsche, Lars G. and Fthenou, Eleni and Gonzalez-Arroyo, Gilberto and Griffiths, Christopher J. and Guo, Yu and Hunt, Karen A. and Ioannidis, Alexander and Jansonius, Nomdo M. and Konuma, Takahiro and Lee, Ming Ta Michael and Lopez-Pineda, Arturo and Matsuda, Yuta and Marioni, Riccardo E. and Moatamed, Babak and Nava-Aguilar, Marco A. and Numakura, Kensuke and Patil, Snehal and Rafaels, Nicholas and Richmond, Anne and Rojas-Muñoz, Agustin and Shortt, Jonathan A. and Straub, Peter and Tao, Ran and Vanderwerff, Brett and Vernekar, Manvi and Veturi, Yogasudha and Barnes, Kathleen C. and Boezen, Marike and Chen, Zhengming and Chen, Chia-Yen and Cho, Judy and Smith, George Davey and Finucane, Hilary K. and Franke, Lude and Gamazon, Eric R. and Ganna, Andrea and Gaunt, Tom R. and Ge, Tian and Huang, Hailiang and Huffman, Jennifer and Katsanis, Nicholas and Koskela, Jukka T. and Lajonchere, Clara and Law, Matthew H. and Li, Liming and Lindgren, Cecilia M. and Loos, Ruth J. F. and MacGregor, Stuart and Matsuda, Koichi and Olsen, Catherine M. and Porteous, David J. and Shavit, Jordan A. and Snieder, Harold and Takano, Tomohiro and Trembath, Richard C. and Vonk, Judith M. and Whiteman, David C. and Wicks, Stephen J. and Wijmenga, Cisca and Wright, John and Zheng, Jie and Zhou, Xiang and Awadalla, Philip and Boehnke, Michael and Bustamante, Carlos D. and Cox, Nancy J. and Fatumo, Segun and Geschwind, Daniel H. and Hayward, Caroline and Hveem, Kristian and Kenny, Eimear E. and Lee, Seunggeun and Lin, Yen-Feng and Mbarek, Hamdi and Mägi, Reedik and Martin, Hilary C. and Medland, Sarah E. and Okada, Yukinori and Palotie, Aarno V. and Pasaniuc, Bogdan and Rader, Daniel J. and Ritchie, Marylyn D. and Sanna, Serena and Smoller, Jordan W. and Stefansson, Kari and van Heel, David A. and Walters, Robin G. and Zöllner, Sebastian and Martin, Alicia R. and Willer, Cristen J. and Daly, Mark J. and Neale, Benjamin M.},\n",
      "\tyear = {2022},\n",
      "\tpages = {100192},\n",
      "}\n",
      "\n",
      "@misc{23andme,\n",
      "    url = {https://blog.23andme.com/articles/central-and-south-asia},\n",
      "    title = {23and{M}e {T}ests {N}ew {A}ncestry {B}reakdown in {C}entral and {S}outh {A}sia},\n",
      "    url-date = {2019-04-04},\n",
      "    year = {2019},\n",
      "    author = {23andMe}\n",
      "}\n",
      "\n",
      "@article{han2017clustering,\n",
      "  title={Clustering of 770,000 genomes reveals post-colonial population structure of North America},\n",
      "  author={Han, Eunjung and Carbonetto, Peter and Curtis, Ross E and Wang, Yong and Granka, Julie M and Byrnes, Jake and Noto, Keith and Kermany, Amir R and Myres, Natalie M and Barber, Mathew J and others},\n",
      "  journal={Nature communications},\n",
      "  volume={8},\n",
      "  number={1},\n",
      "  pages={14238},\n",
      "  year={2017}\n",
      "}\n",
      "\n",
      "@misc{alten_reweighting_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Reweighting the {UK} {Biobank} to reflect its underlying sampling population substantially reduces pervasive selection bias due to volunteering},\n",
      "\turl = {https://www.medrxiv.org/content/10.1101/2022.05.16.22275048v1},\n",
      "\tdoi = {10.1101/2022.05.16.22275048},\n",
      "\tpublisher = {medRxiv},\n",
      "\tauthor = {Alten, Sjoerd van and Domingue, Benjamin W. and Galama, Titus and Marees, Andries T.},\n",
      "\tyear = {2022}\n",
      "}\n",
      "\n",
      "@article{martin_human_2017,\n",
      "\ttitle = {Human {Demographic} {History} {Impacts} {Genetic} {Risk} {Prediction} across {Diverse} {Populations}},\n",
      "\tvolume = {100},\n",
      "\tissn = {0002-9297},\n",
      "\turl = {https://www.sciencedirect.com/science/article/pii/S0002929717301076},\n",
      "\tdoi = {10.1016/j.ajhg.2017.03.004},\n",
      "\tnumber = {4},\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\tauthor = {Martin, Alicia R. and Gignoux, Christopher R. and Walters, Raymond K. and Wojcik, Genevieve L. and Neale, Benjamin M. and Gravel, Simon and Daly, Mark J. and Bustamante, Carlos D. and Kenny, Eimear E.},\n",
      "\tyear = {2017},\n",
      "\tpages = {635--649}\n",
      "}\n",
      "\n",
      "@article{pirastu_genetic_2021,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Genetic analyses identify widespread sex-differential participation bias},\n",
      "\tvolume = {53},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-021-00846-7},\n",
      "\tdoi = {10.1038/s41588-021-00846-7},\n",
      "\tnumber = {5},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Pirastu, Nicola and Cordioli, Mattia and Nandakumar, Priyanka and Mignogna, Gianmarco and Abdellaoui, Abdel and Hollis, Benjamin and Kanai, Masahiro and Rajagopal, Veera M. and Parolo, Pietro Della Briotta and Baya, Nikolas and Carey, Caitlin E. and Karjalainen, Juha and Als, Thomas D. and Van der Zee, Matthijs D. and Day, Felix R. and Ong, Ken K. and Morisaki, Takayuki and de Geus, Eco and Bellocco, Rino and Okada, Yukinori and Børglum, Anders D. and Joshi, Peter and Auton, Adam and Hinds, David and Neale, Benjamin M. and Walters, Raymond K. and Nivard, Michel G. and Perry, John R. B. and Ganna, Andrea},\n",
      "\tyear = {2021},\n",
      "\tpages = {663--671}\n",
      "}\n",
      "\n",
      "@article{mathieson_what_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {What is ancestry?},\n",
      "\tvolume = {16},\n",
      "\tissn = {1553-7404},\n",
      "\turl = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008624},\n",
      "\tdoi = {10.1371/journal.pgen.1008624},\n",
      "\tnumber = {3},\n",
      "\tjournal = {PLOS Genetics},\n",
      "\tauthor = {Mathieson, Iain and Scally, Aylwyn},\n",
      "\tyear = {2020},\n",
      "\tpages = {e1008624}\n",
      "}\n",
      "\n",
      "@article{oconnell_population-specific_2021,\n",
      "\ttitle = {A population-specific reference panel for improved genotype imputation in {African} {Americans}},\n",
      "\tvolume = {4},\n",
      "\tissn = {2399-3642},\n",
      "\turl = {https://www.nature.com/articles/s42003-021-02777-9},\n",
      "\tdoi = {10.1038/s42003-021-02777-9},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Communications Biology},\n",
      "\tauthor = {O’Connell, Jared and Yun, Taedong and Moreno, Meghan and Li, Helen and Litterman, Nadia and Kolesnikov, Alexey and Noblin, Elizabeth and Chang, Pi-Chuan and Shastri, Anjali and Dorfman, Elizabeth H. and Shringarpure, Suyash and Auton, Adam and Carroll, Andrew and McLean, Cory Y.},\n",
      "\tyear = {2021},\n",
      "\tpages = {1--9}\n",
      "}\n",
      "\n",
      "@misc{quanjer2012multi,\n",
      "  title={Multi-ethnic reference values for spirometry for the 3--95-yr age range: the global lung function 2012 equations},\n",
      "  author={Quanjer, Philip H and Stanojevic, Sanja and Cole, Tim J and Baur, Xaver and Hall, Graham L and Culver, Bruce H and Enright, Paul L and Hankinson, John L and Ip, Mary SM and Zheng, Jinping and others},\n",
      "  year={2012},\n",
      "  publisher={Eur Respiratory Soc}\n",
      "}\n",
      "\n",
      "@article{ortega2015effect,\n",
      "  title={The effect of ancestry and genetic variation on lung function predictions: what is “normal” lung function in diverse human populations?},\n",
      "  author={Ortega, Victor E and Kumar, Rajesh},\n",
      "  journal={Current allergy and asthma reports},\n",
      "  volume={15},\n",
      "  pages={1--11},\n",
      "  year={2015},\n",
      "  publisher={Springer}\n",
      "}\n",
      "\n",
      "@Manual{r2013,\n",
      "    title = {R: A Language and Environment for Statistical Computing},\n",
      "    author = {{R Core Team}},\n",
      "    organization = {R Foundation for Statistical Computing},\n",
      "    address = {Vienna, Austria},\n",
      "    year = {2013},\n",
      "    url = {https://www.R-project.org/}\n",
      "}\n",
      "\n",
      "@Book{ggplot2016,\n",
      "    author = {Hadley Wickham},\n",
      "    title = {ggplot2: Elegant Graphics for Data Analysis},\n",
      "    publisher = {Springer-Verlag New York},\n",
      "    year = {2016},\n",
      "    isbn = {978-3-319-24277-4},\n",
      "    url = {https://ggplot2.tidyverse.org}\n",
      "}\n",
      "\n",
      "@article{kelleher2019inferring,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Inferring whole-genome histories in large population datasets},\n",
      "  author={Kelleher, Jerome and Wong, Yan and Wohns, Anthony W and Fadil, Chaimaa and Albers, Patrick K and McVean, Gil},\n",
      "  journal={Nature genetics},\n",
      "  volume={51},\n",
      "  number={9},\n",
      "  pages={1330--1338},\n",
      "  year={2019}\n",
      "}\n",
      "\n",
      "@article{zhang2023biobank,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Biobank-scale inference of ancestral recombination graphs enables genealogical analysis of complex traits},\n",
      "  author={Zhang, Brian C and Biddanda, Arjun and Gunnarsson, {\\'A}rni Freyr and Cooper, Fergus and Palamara, Pier Francesco},\n",
      "  journal={Nature Genetics},\n",
      "  pages={1--9},\n",
      "  year={2023}\n",
      "}\n",
      "\n",
      "@article{yc2022evaluation,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Evaluation of methods for estimating coalescence times using ancestral recombination graphs},\n",
      "  author={YC Brandt, D{\\'e}bora and Wei, Xinzhu and Deng, Yun and Vaughn, Andrew H and Nielsen, Rasmus},\n",
      "  journal={Genetics},\n",
      "  volume={221},\n",
      "  number={1},\n",
      "  pages={iyac044},\n",
      "  year={2022}\n",
      "}\n",
      "\n",
      "@article{patterson2006population,\n",
      "  title={Population structure and eigenanalysis},\n",
      "  author={Patterson, Nick and Price, Alkes L and Reich, David},\n",
      "  journal={PLoS genetics},\n",
      "  volume={2},\n",
      "  number={12},\n",
      "  pages={e190},\n",
      "  year={2006}\n",
      "}\n",
      "\n",
      "@article{becht2019dimensionality,\n",
      "  title={Dimensionality reduction for visualizing single-cell data using {UMAP}},\n",
      "  author={Becht, Etienne and McInnes, Leland and Healy, John and Dutertre, Charles-Antoine and Kwok, Immanuel WH and Ng, Lai Guan and Ginhoux, Florent and Newell, Evan W},\n",
      "  journal={Nature biotechnology},\n",
      "  volume={37},\n",
      "  number={1},\n",
      "  pages={38--44},\n",
      "  year={2019},\n",
      "  publisher={Nature Publishing Group US New York}\n",
      "}\n",
      "\n",
      "@article{moon2019visualizing,\n",
      "\tkeywords = {mainbib},\n",
      "  title={Visualizing structure and transitions in high-dimensional biological data},\n",
      "  author={Moon, Kevin R and van Dijk, David and Wang, Zheng and Gigante, Scott and Burkhardt, Daniel B and Chen, William S and Yim, Kristina and Elzen, Antonia van den and Hirn, Matthew J and Coifman, Ronald R and others},\n",
      "  journal={Nature biotechnology},\n",
      "  volume={37},\n",
      "  number={12},\n",
      "  pages={1482--1492},\n",
      "  year={2019}\n",
      "}\n",
      "\n",
      "@article{diaz-papkovich_umap_2019,\n",
      "  title={{UMAP} reveals cryptic population structure and phenotype heterogeneity in large genomic cohorts},\n",
      "  author={Diaz-Papkovich, Alex and Anderson-Trocm{\\'e}, Luke and Ben-Eghan, Chief and Gravel, Simon},\n",
      "  journal={PLoS genetics},\n",
      "  volume={15},\n",
      "  number={11},\n",
      "  pages={e1008432},\n",
      "  year={2019},\n",
      "  publisher={Public Library of Science San Francisco, CA USA}\n",
      "}\n",
      "\n",
      "@article{cann2002human,\n",
      "  title={A human genome diversity cell line panel},\n",
      "  author={Cann, Howard M and De Toma, Claudia and Cazes, Lucien and Legrand, Marie-Fernande and Morel, Valerie and Piouffre, Laurence and Bodmer, Julia and Bodmer, Walter F and Bonne-Tamir, Batsheva and Cambon-Thomsen, Anne and others},\n",
      "  journal={Science},\n",
      "  volume={296},\n",
      "  number={5566},\n",
      "  pages={261--262},\n",
      "  year={2002}\n",
      "}\n",
      "\n",
      "@article{karczewski_mutational_2020,\n",
      "\ttitle = {The mutational constraint spectrum quantified from variation in 141,456 humans},\n",
      "\tvolume = {581},\n",
      "\tissn = {1476-4687},\n",
      "\turl = {https://www.nature.com/articles/s41586-020-2308-7},\n",
      "\tdoi = {10.1038/s41586-020-2308-7},\n",
      "\tnumber = {7809},\n",
      "\tjournal = {Nature},\n",
      "\tauthor = {Karczewski, Konrad J. and Francioli, Laurent C. and Tiao, Grace and Cummings, Beryl B. and Alföldi, Jessica and Wang, Qingbo and Collins, Ryan L. and Laricchia, Kristen M. and Ganna, Andrea and Birnbaum, Daniel P. and Gauthier, Laura D. and Brand, Harrison and Solomonson, Matthew and Watts, Nicholas A. and Rhodes, Daniel and Singer-Berk, Moriel and England, Eleina M. and Seaby, Eleanor G. and Kosmicki, Jack A. and Walters, Raymond K. and Tashman, Katherine and Farjoun, Yossi and Banks, Eric and Poterba, Timothy and Wang, Arcturus and Seed, Cotton and Whiffin, Nicola and Chong, Jessica X. and Samocha, Kaitlin E. and Pierce-Hoffman, Emma and Zappala, Zachary and O’Donnell-Luria, Anne H. and Minikel, Eric Vallabh and Weisburd, Ben and Lek, Monkol and Ware, James S. and Vittal, Christopher and Armean, Irina M. and Bergelson, Louis and Cibulskis, Kristian and Connolly, Kristen M. and Covarrubias, Miguel and Donnelly, Stacey and Ferriera, Steven and Gabriel, Stacey and Gentry, Jeff and Gupta, Namrata and Jeandet, Thibault and Kaplan, Diane and Llanwarne, Christopher and Munshi, Ruchi and Novod, Sam and Petrillo, Nikelle and Roazen, David and Ruano-Rubio, Valentin and Saltzman, Andrea and Schleicher, Molly and Soto, Jose and Tibbetts, Kathleen and Tolonen, Charlotte and Wade, Gordon and Talkowski, Michael E. and Neale, Benjamin M. and Daly, Mark J. and MacArthur, Daniel G.},\n",
      "\tyear = {2020},\n",
      "\tpages = {434--443}\n",
      "}\n",
      "\n",
      "@article{nagai2017overview,\n",
      "  title={Overview of the {BioBank Japan Project}: study design and profile},\n",
      "  author={Nagai, Akiko and Hirata, Makoto and Kamatani, Yoichiro and Muto, Kaori and Matsuda, Koichi and Kiyohara, Yutaka and Ninomiya, Toshiharu and Tamakoshi, Akiko and Yamagata, Zentaro and Mushiroda, Taisei and others},\n",
      "  journal={Journal of epidemiology},\n",
      "  volume={27},\n",
      "  number={Supplement\\_III},\n",
      "  pages={S2--S8},\n",
      "  year={2017}\n",
      "}\n",
      "\n",
      "@article{sakaue_dimensionality_2020,\n",
      "\ttitle = {Dimensionality reduction reveals fine-scale structure in the {Japanese} population with consequences for polygenic risk prediction},\n",
      "\tvolume = {11},\n",
      "\tissn = {2041-1723},\n",
      "\turl = {https://www.nature.com/articles/s41467-020-15194-z},\n",
      "\tdoi = {10.1038/s41467-020-15194-z},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Nature Communications},\n",
      "\tauthor = {Sakaue, Saori and Hirata, Jun and Kanai, Masahiro and Suzuki, Ken and Akiyama, Masato and Lai Too, Chun and Arayssi, Thurayya and Hammoudeh, Mohammed and Al Emadi, Samar and Masri, Basel K. and Halabi, Hussein and Badsha, Humeira and Uthman, Imad W. and Saxena, Richa and Padyukov, Leonid and Hirata, Makoto and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Okada, Yukinori},\n",
      "\tyear = {2020},\n",
      "\tpages = {1569}\n",
      "}\n",
      "\n",
      "@article{belbin_towards_2019,\n",
      "\ttitle = {Towards a fine-scale population health monitoring system},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/780668v1},\n",
      "\tdoi = {10.1101/780668},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Belbin, Gillian M. and Wenric, Stephane and Cullina, Sinead and Glicksberg, Benjamin S. and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Team, CBIPM Genomics and Team, CBIPM Genomics and Center, Regeneron Genetics and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth JF and Abul-husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},\n",
      "\tyear = {2019},\n",
      "\tpages = {780668}\n",
      "}\n",
      "\n",
      "@article{hunter-zinck_genotyping_2020,\n",
      "\ttitle = {Genotyping {Array} {Design} and {Data} {Quality} {Control} in the {Million} {Veteran} {Program}},\n",
      "\tvolume = {106},\n",
      "\tissn = {0002-9297},\n",
      "\turl = {http://www.sciencedirect.com/science/article/pii/S000292972030080X},\n",
      "\tdoi = {10.1016/j.ajhg.2020.03.004},\n",
      "\tnumber = {4},\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\tauthor = {Hunter-Zinck, Haley and Shi, Yunling and Li, Man and Gorman, Bryan R. and Ji, Sun-Gou and Sun, Ning and Webster, Teresa and Liem, Andrew and Hsieh, Paul and Devineni, Poornima and Karnam, Purushotham and Gong, Xin and Radhakrishnan, Lakshmi and Schmidt, Jeanette and Assimes, Themistocles L. and Huang, Jie and Pan, Cuiping and Humphries, Donald and Brophy, Mary and Moser, Jennifer and Muralidhar, Sumitra and Huang, Grant D. and Przygodzki, Ronald and Concato, John and Gaziano, John M. and Gelernter, Joel and O’Donnell, Christopher J. and Hauser, Elizabeth R. and Zhao, Hongyu and O’Leary, Timothy J. and Tsao, Philip S. and Pyarajan, Saiju},\n",
      "\tyear = {2020},\n",
      "\tpages = {535--548}\n",
      "}\n",
      "\n",
      "@article{margaryan_population_2019,\n",
      "\ttitle = {Population genomics of the {Viking} world},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/703405v1},\n",
      "\tdoi = {10.1101/703405},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Margaryan, Ashot and Lawson, Daniel and Sikora, Martin and Racimo, Fernando and Rasmussen, Simon and Moltke, Ida and Cassidy, Lara and Jørsboe, Emil and Ingason, Andrés and Pedersen, Mikkel and Korneliussen, Thorfinn and Wilhelmson, Helene and Buś, Magdalena and Damgaard, Peter de Barros and Martiniano, Rui and Renaud, Gabriel and Bhérer, Claude and Moreno-Mayar, J. Víctor and Fotakis, Anna and Allen, Marie and Molak, Martyna and Cappellini, Enrico and Scorrano, Gabriele and Buzhilova, Alexandra and Fox, Allison and Albrechtsen, Anders and Schütz, Berit and Skar, Birgitte and Arcini, Caroline and Falys, Ceri and Jonson, Charlotte Hedenstierna and Błaszczyk, Dariusz and Pezhemsky, Denis and Turner-Walker, Gordon and Gestsdóttir, Hildur and Lundstrøm, Inge and Gustin, Ingrid and Mainland, Ingrid and Potekhina, Inna and Muntoni, Italo and Cheng, Jade and Stenderup, Jesper and Ma, Jilong and Gibson, Julie and Peets, Jüri and Gustafsson, Jörgen and Iversen, Katrine and Simpson, Linzi and Strand, Lisa and Loe, Louise and Sikora, Maeve and Florek, Marek and Vretemark, Maria and Redknap, Mark and Bajka, Monika and Pushkina, Tamara and Søvsø, Morten and Grigoreva, Natalia and Christensen, Tom and Kastholm, Ole and Uldum, Otto and Favia, Pasquale and Holck, Per and Allmäe, Raili and Sten, Sabine and Arge, Símun and Ellingvåg, Sturla and Moiseyev, Vayacheslav and Bogdanowicz, Wiesław and Magnusson, Yvonne and Orlando, Ludovic and Bradley, Daniel and Jørkov, Marie Louise and Arneborg, Jette and Lynnerup, Niels and Price, Neil and Gilbert, M. Thomas and Allentoft, Morten and Bill, Jan and Sindbæk, Søren and Hedeager, Lotte and Kristiansen, Kristian and Nielsen, Rasmus and Werge, Thomas and Willerslev, Eske},\n",
      "\tyear = {2019},\n",
      "\tpages = {703405}\n",
      "}\n",
      "\n",
      "@article{simon_local_2019,\n",
      "\ttitle = {Local introgression at two spatial scales in mosaic hybrid zones of mussels},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/818559v1},\n",
      "\tdoi = {10.1101/818559},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Simon, Alexis and Fraïsse, Christelle and Ayari, Tahani El and Liautard-Haag, Cathy and Strelkov, Petr and Welch, John J. and Bierne, Nicolas},\n",
      "\tyear = {2019},\n",
      "\tpages = {818559}\n",
      "}\n",
      "\n",
      "@article{sanchez-barreiro_historical_2020,\n",
      "\ttitle = {Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros ({Ceratotherium} simum)},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.05.10.086686v1},\n",
      "\tdoi = {10.1101/2020.05.10.086686},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Sánchez-Barreiro, Fátima and Gopalakrishnan, Shyam and Ramos-Madrigal, Jazmín and Westbury, Michael V. and Manuel, Marc de and Margaryan, Ashot and Ciucani, Marta M. and Vieira, Filipe G. and Patramanis, Yannis and Kalthoff, Daniela C. and Timmons, Zena and Sicheritz-Pontén, Thomas and Dalén, Love and Ryder, Oliver A. and Zhang, Guojie and Marquès-Bonet, Tomás and Moodley, Yoshan and Gilbert, M. Thomas P.},\n",
      "\tyear = {2020},\n",
      "\tpages = {2020.05.10.086686}\n",
      "}\n",
      "\n",
      "@article{consortium_genome_2020,\n",
      "\ttitle = {Genome variation and population structure among 1,142 mosquitoes of the {African} malaria vector species {Anopheles} gambiae and {Anopheles} coluzzii},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/864314v2},\n",
      "\tdoi = {10.1101/864314},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Consortium, The Anopheles gambiae 1000 Genomes and Clarkson, Chris S. and Miles, Alistair and Harding, Nicholas J. and Lucas, Eric R. and Battey, C. J. and Amaya-Romero, Jorge Edouardo and Cano, Jorge and Diabate, Abdoulaye and Constant, Edi and Nwakanma, Davis C. and Jawara, Musa and Essandoh, John and Dinis, Joao and Goff, Gilbert Le and Robert, Vincent and Troco, Arlete D. and Costantini, Carlo and Rohatgi, Kyanne R. and Elissa, Nohal and Coulibaly, Boubacar and Midega, Janet and Mbogo, Charles and Mawejje, Henry D. and Stalker, Jim and Rockett, Kirk A. and Drury, Eleanor and Mead, Daniel and Jeffreys, Anna E. and Hubbart, Christina and Rowlands, Kate and Isaacs, Alison T. and Jyothi, Dushyanth and Malangone, Cinzia and Kamali, Maryam and Henrichs, Christa and Simpson, Victoria and Ayala, Diego and Besansky, Nora J. and Burt, Austin and Caputo, Beniamino and Torre, Alessandra della and Fontaine, Michael and Godfray, H. Charles J. and Hahn, Matthew W. and Kern, Andrew D. and Lawniczak, Mara K. N. and O’Loughlin, Samantha and Pinto, Joao and Riehle, Michelle M. and Sharakhov, Igor and Schrider, Daniel R. and Vernick, Kenneth D. and White, Bradley J. and Donnelly, Martin J. and Kwiatkowski, Dominic P.},\n",
      "\tyear = {2020},\n",
      "\tpages = {864314}\n",
      "}\n",
      "\n",
      "@article{schmidt2020population,\n",
      "\ttitle = {Population genomics of two invasive mosquitoes ({Aedes} aegypti and {Aedes} albopictus) from the {Indo}-{Pacific}},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2020.03.15.993055v3},\n",
      "\tdoi = {10.1101/2020.03.15.993055},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Schmidt, Thomas L. and Chung, Jessica and Honnen, Ann-Christin and Weeks, Andrew R. and Hoffmann, Ary A.},\n",
      "\tyear = {2020},\n",
      "\tpages = {2020.03.15.993055}\n",
      "}\n",
      "\n",
      "@article{dai_population_2020,\n",
      "\ttitle = {Population {Histories} of the {United} {States} {Revealed} through {Fine}-{Scale} {Migration} and {Haplotype} {Analysis}},\n",
      "\tvolume = {106},\n",
      "\tissn = {00029297},\n",
      "\turl = {https://linkinghub.elsevier.com/retrieve/pii/S0002929720300446},\n",
      "\tdoi = {10.1016/j.ajhg.2020.02.002},\n",
      "\tnumber = {3},\n",
      "\tjournal = {The American Journal of Human Genetics},\n",
      "\tauthor = {Dai, Chengzhen L. and Vazifeh, Mohammad M. and Yeang, Chen-Hsiang and Tachet, Remi and Wells, R. Spencer and Vilar, Miguel G. and Daly, Mark J. and Ratti, Carlo and Martin, Alicia R.},\n",
      "\tyear = {2020},\n",
      "\tpages = {371--388}\n",
      "}\n",
      "\n",
      "@article{spear2020recent,\n",
      "  title={Recent shifts in the genomic ancestry of {M}exican {A}mericans may alter the genetic architecture of biomedical traits},\n",
      "  author={Spear, Melissa L and Diaz-Papkovich, Alex and Ziv, Elad and Yracheta, Joseph M and Gravel, Simon and Torgerson, Dara G and Hernandez, Ryan D},\n",
      "  journal={Elife},\n",
      "  volume={9},\n",
      "  pages={e56029},\n",
      "  year={2020}\n",
      "}\n",
      "\n",
      "@article{tonkin-hill_fast_2019,\n",
      "\ttitle = {Fast hierarchical {Bayesian} analysis of population structure},\n",
      "\tvolume = {47},\n",
      "\tissn = {0305-1048},\n",
      "\turl = {https://academic.oup.com/nar/article/47/11/5539/5488018},\n",
      "\tdoi = {10.1093/nar/gkz361},\n",
      "\tnumber = {11},\n",
      "\tjournal = {Nucleic Acids Research},\n",
      "\tauthor = {Tonkin-Hill, Gerry and Lees, John A. and Bentley, Stephen D. and Frost, Simon D. W. and Corander, Jukka},\n",
      "\tyear = {2019},\n",
      "\tpages = {5539--5549}\n",
      "}\n",
      "\n",
      "@article{almarri2020population,\n",
      "\ttitle = {Population {Structure}, {Stratification} and {Introgression} of {Human} {Structural} {Variation}},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/746172v2},\n",
      "\tdoi = {10.1101/746172},\n",
      "\tjournal = {bioRxiv},\n",
      "\tauthor = {Almarri, Mohamed A. and Bergström, Anders and Prado-Martinez, Javier and Yang, Fengtang and Fu, Beiyuan and Dunham, Alistair S. and Chen, Yuan and Hurles, Matthew E. and Tyler-Smith, Chris and Xue, Yali},\n",
      "\tyear = {2020},\n",
      "\tpages = {746172}\n",
      "}\n",
      "\n",
      "@article{sohail2019polygenic,\n",
      "  title={Polygenic adaptation on height is overestimated due to uncorrected stratification in genome-wide association studies},\n",
      "  url = {https://elifesciences.org/articles/39702},\n",
      "  doi = {10.7554/eLife.39702},\n",
      "  author={Sohail, Mashaal and Maier, Robert M and Ganna, Andrea and Bloemendal, Alex and Martin, Alicia R and Turchin, Michael C and Chiang, Charleston WK and Hirschhorn, Joel and Daly, Mark J and Patterson, Nick and others},\n",
      "  journal={e{L}ife},\n",
      "  volume={8},\n",
      "  pages={e39702},\n",
      "  year={2019},\n",
      "  publisher={eLife Sciences Publications Limited}\n",
      "}\n",
      "\n",
      "@article{yamamoto_genetic_2020,\n",
      "\ttitle = {Genetic and phenotypic landscape of the mitochondrial genome in the {Japanese} population},\n",
      "\tvolume = {3},\n",
      "\tissn = {2399-3642},\n",
      "\turl = {https://www.nature.com/articles/s42003-020-0812-9},\n",
      "\tdoi = {10.1038/s42003-020-0812-9},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Communications Biology},\n",
      "\tauthor = {Yamamoto, Kenichi and Sakaue, Saori and Matsuda, Koichi and Murakami, Yoshinori and Kamatani, Yoichiro and Ozono, Keiichi and Momozawa, Yukihide and Okada, Yukinori},\n",
      "\tyear = {2020},\n",
      "\tpages = {1--11}\n",
      "}\n",
      "\n",
      "@article{mcinnes2018software,\n",
      "  title={{UMAP}: {Uniform} {Manifold} {Approximation} and {Projection}},\n",
      "  author={McInnes, Leland and Healy, John and Saul, Nathaniel and Grossberger, Lukas},\n",
      "  url={https://joss.theoj.org/papers/10.21105/joss.00861},\n",
      "  doi={10.21105/joss.00861},\n",
      "  journal={The Journal of Open Source Software},\n",
      "  volume={3},\n",
      "  number={29},\n",
      "  pages={861},\n",
      "  year={2018}\n",
      "}\n",
      "\n",
      "@incollection{gebru_race_2020,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Race and {Gender}},\n",
      "\tisbn = {978-0-19-006739-7},\n",
      "\turl = {https://doi.org/10.1093/oxfordhb/9780190067397.013.16},\n",
      "\tbooktitle = {The {Oxford} {Handbook} of {Ethics} of {AI}},\n",
      "\tpublisher = {Oxford University Press},\n",
      "\tauthor = {Gebru, Timnit},\n",
      "\teditor = {Dubber, Markus D. and Pasquale, Frank and Das, Sunit},\n",
      "\tyear = {2020},\n",
      "\tdoi = {10.1093/oxfordhb/9780190067397.013.16},\n",
      "\tchapter = {13},\n",
      "\tpages = {253--269}\n",
      "}\n",
      "\n",
      "@article{cohen_mathematics_2004,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Mathematics {Is} {Biology}'s {Next} {Microscope}, {Only} {Better}; {Biology} {Is} {Mathematics}' {Next} {Physics}, {Only} {Better}},\n",
      "\tvolume = {2},\n",
      "\tissn = {1545-7885},\n",
      "\turl = {https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0020439},\n",
      "\tdoi = {10.1371/journal.pbio.0020439},\n",
      "\tnumber = {12},\n",
      "\tjournal = {{PLOS} Biology},\n",
      "\tauthor = {Cohen, Joel E.},\n",
      "\tyear = {2004},\n",
      "\tpages = {e439}\n",
      "}\n",
      "\n",
      "@article{hunter_biology_2010,\n",
      "\ttitle = {Biology is the new physics},\n",
      "\tvolume = {11},\n",
      "\tissn = {1469-221X},\n",
      "\turl = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2868533/},\n",
      "\tdoi = {10.1038/embor.2010.55},\n",
      "\tnumber = {5},\n",
      "\tjournal = {EMBO Reports},\n",
      "\tauthor = {Hunter, Philip},\n",
      "\tyear = {2010},\n",
      "\tpmid = {20428108},\n",
      "\tpmcid = {PMC2868533},\n",
      "\tpages = {350--352}\n",
      "}\n",
      "\n",
      "@misc{correard_allele_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Allele {Dispersion} {Score}: {Quantifying} the range of allele frequencies across populations, based on {UMAP}},\n",
      "\tshorttitle = {Allele {Dispersion} {Score}},\n",
      "\turl = {https://www.biorxiv.org/content/10.1101/2022.02.11.479970v1},\n",
      "\tdoi = {10.1101/2022.02.11.479970},\n",
      "\tpublisher = {bioRxiv},\n",
      "\tauthor = {Correard, Solenne and Arbour, Laura and Wasserman, Wyeth W.},\n",
      "\tyear = {2022}\n",
      "}\n",
      "\n",
      "@article{battey_visualizing_2021,\n",
      "\tkeywords = {mainbib},\n",
      "title = {Visualizing population structure with variational autoencoders},\n",
      "volume = {11},\n",
      "issn = {2160-1836},\n",
      "url = {https://doi.org/10.1093/g3journal/jkaa036},\n",
      "doi = {10.1093/g3journal/jkaa036},\n",
      "number = {1},\n",
      "journal = {G3 Genes{\\textbar}Genomes{\\textbar}Genetics},\n",
      "author = {Battey, C J and Coffing, Gabrielle C and Kern, Andrew D},\n",
      "year = {2021},\n",
      "pages = {jkaa036}\n",
      "}\n",
      "\n",
      "@article{kaseniit_genetic_2020,\n",
      "title = {Genetic ancestry analysis on {\\textgreater}93,000 individuals undergoing expanded carrier screening reveals limitations of ethnicity-based medical guidelines},\n",
      "volume = {22},\n",
      "issn = {1530-0366},\n",
      "url = {https://www.nature.com/articles/s41436-020-0869-3},\n",
      "doi = {10.1038/s41436-020-0869-3},\n",
      "number = {10},\n",
      "journal = {Genetics in Medicine},\n",
      "author = {Kaseniit, Kristjan E. and Haque, Imran S. and Goldberg, James D. and Shulman, Lee P. and Muzzey, Dale},\n",
      "year = {2020},\n",
      "pages = {1694--1702}\n",
      "}\n",
      "\n",
      "@article{freyman_fast_2021,\n",
      "title = {Fast and {Robust} {Identity}-by-{Descent} {Inference} with the {Templated} {Positional} {Burrows}–{Wheeler} {Transform}},\n",
      "volume = {38},\n",
      "issn = {1537-1719},\n",
      "url = {https://doi.org/10.1093/molbev/msaa328},\n",
      "doi = {10.1093/molbev/msaa328},\n",
      "number = {5},\n",
      "journal = {Molecular Biology and Evolution},\n",
      "author = {Freyman, William A and McManus, Kimberly F and Shringarpure, Suyash S and Jewett, Ethan M and Bryc, Katarzyna and {The 23 and Me Research Team} and Auton, Adam},\n",
      "year = {2021},\n",
      "pages = {2131--2151}\n",
      "}\n",
      "\n",
      "@article{shemirani_rapid_2021,\n",
      "title = {Rapid detection of identity-by-descent tracts for mega-scale datasets},\n",
      "volume = {12},\n",
      "issn = {2041-1723},\n",
      "url = {https://www.nature.com/articles/s41467-021-22910-w},\n",
      "doi = {10.1038/s41467-021-22910-w},\n",
      "number = {1},\n",
      "journal = {Nature Communications},\n",
      "author = {Shemirani, Ruhollah and Belbin, Gillian M. and Avery, Christy L. and Kenny, Eimear E. and Gignoux, Christopher R. and Ambite, José Luis},\n",
      "year = {2021},\n",
      "pages = {3546}\n",
      "}\n",
      "\n",
      "@book{committee_2023,\n",
      "address = {Washington, D.C.},\n",
      "author = {{Committee on the Use of Race, Ethnicity, and Ancestry as Population Descriptors in Genomics Research}},\n",
      "title = {Using {Population} {Descriptors} in {Genetics} and {Genomics} {Research}: {A} {New} {Framework} for an {Evolving} {Field}},\n",
      "isbn = {978-0-309-70065-8},\n",
      "shorttitle = {Using {Population} {Descriptors} in {Genetics} and {Genomics} {Research}},\n",
      "url = {https://www.nap.edu/catalog/26902},\n",
      "publisher = {National Academies Press},\n",
      "collaborator = {{Committee on the Use of Race, Ethnicity, and Ancestry as Population Descriptors in Genomics Research} and {Board on Health Sciences Policy} and {Committee on Population} and {Health and Medicine Division} and {Division of Behavioral and Social Sciences and Education} and {National Academies of Sciences, Engineering, and Medicine}},\n",
      "year = {2023},\n",
      "doi = {10.17226/26902}\n",
      "}\n",
      "\n",
      "@article{halldorsson_sequences_2022,\n",
      "title = {The sequences of 150,119 genomes in the {UK} {Biobank}},\n",
      "volume = {607},\n",
      "issn = {1476-4687},\n",
      "url = {https://www.nature.com/articles/s41586-022-04965-x},\n",
      "doi = {10.1038/s41586-022-04965-x},\n",
      "number = {7920},\n",
      "journal = {Nature},\n",
      "author = {Halldorsson, Bjarni V. and Eggertsson, Hannes P. and Moore, Kristjan H. S. and Hauswedell, Hannes and Eiriksson, Ogmundur and Ulfarsson, Magnus O. and Palsson, Gunnar and Hardarson, Marteinn T. and Oddsson, Asmundur and Jensson, Brynjar O. and Kristmundsdottir, Snaedis and Sigurpalsdottir, Brynja D. and Stefansson, Olafur A. and Beyter, Doruk and Holley, Guillaume and Tragante, Vinicius and Gylfason, Arnaldur and Olason, Pall I. and Zink, Florian and Asgeirsdottir, Margret and Sverrisson, Sverrir T. and Sigurdsson, Brynjar and Gudjonsson, Sigurjon A. and Sigurdsson, Gunnar T. and Halldorsson, Gisli H. and Sveinbjornsson, Gardar and Norland, Kristjan and Styrkarsdottir, Unnur and Magnusdottir, Droplaug N. and Snorradottir, Steinunn and Kristinsson, Kari and Sobech, Emilia and Jonsson, Helgi and Geirsson, Arni J. and Olafsson, Isleifur and Jonsson, Palmi and Pedersen, Ole Birger and Erikstrup, Christian and Brunak, Søren and Ostrowski, Sisse Rye and Thorleifsson, Gudmar and Jonsson, Frosti and Melsted, Pall and Jonsdottir, Ingileif and Rafnar, Thorunn and Holm, Hilma and Stefansson, Hreinn and Saemundsdottir, Jona and Gudbjartsson, Daniel F. and Magnusson, Olafur T. and Masson, Gisli and Thorsteinsdottir, Unnur and Helgason, Agnar and Jonsson, Hakon and Sulem, Patrick and Stefansson, Kari},\n",
      "year = {2022},\n",
      "pages = {732--740}\n",
      "}\n",
      "\n",
      "@article{martin_clinical_2019,\n",
      "title = {Clinical use of current polygenic risk scores may exacerbate health disparities},\n",
      "volume = {51},\n",
      "issn = {1546-1718},\n",
      "url = {https://www.nature.com/articles/s41588-019-0379-x},\n",
      "doi = {10.1038/s41588-019-0379-x},\n",
      "number = {4},\n",
      "journal = {Nature Genetics},\n",
      "author = {Martin, Alicia R. and Kanai, Masahiro and Kamatani, Yoichiro and Okada, Yukinori and Neale, Benjamin M. and Daly, Mark J.},\n",
      "year = {2019},\n",
      "pages = {584--591}\n",
      "}\n",
      "\n",
      "@article{ben-eghan_dont_2020,\n",
      "\tkeywords = {mainbib},\n",
      "title = {Don't ignore genetic data from minority populations},\n",
      "volume = {585},\n",
      "url = {https://www.nature.com/articles/d41586-020-02547-3},\n",
      "doi = {10.1038/d41586-020-02547-3},\n",
      "number = {7824},\n",
      "journal = {Nature},\n",
      "author = {Ben-Eghan, Chief and Sun, Rosie and Hleap, Jose Sergio and Diaz-Papkovich, Alex and Munter, Hans Markus and Grant, Audrey V. and Dupras, Charles and Gravel, Simon},\n",
      "year = {2020},\n",
      "pages = {184--186}\n",
      "}\n",
      "\n",
      "@article{martschenko_including_2023,\n",
      "title = {Including multiracial individuals is crucial for race, ethnicity and ancestry frameworks in genetics and genomics},\n",
      "volume = {55},\n",
      "issn = {1546-1718},\n",
      "url = {https://www.nature.com/articles/s41588-023-01394-y},\n",
      "doi = {10.1038/s41588-023-01394-y},\n",
      "number = {6},\n",
      "journal = {Nature Genetics},\n",
      "author = {Martschenko, Daphne O. and Wand, Hannah and Young, Jennifer L. and Wojcik, Genevieve L.},\n",
      "year = {2023},\n",
      "pages = {895--900}\n",
      "}\n",
      "\n",
      "@article{romero_diet_2017,\n",
      "title = {Diet {Networks}: {Thin} {Parameters} for {Fat} {Genomics}},\n",
      "url = {http://arxiv.org/abs/1611.09340},\n",
      "journal = {arXiv:1611.09340 [cs, stat]},\n",
      "author = {Romero, Adriana and Carrier, Pierre Luc and Erraqabi, Akram and Sylvain, Tristan and Auvolat, Alex and Dejoie, Etienne and Legault, Marc-André and Dubé, Marie-Pierre and Hussin, Julie G. and Bengio, Yoshua},\n",
      "year = {2017}\n",
      "}\n",
      "\n",
      "@article{reich_reconstructing_2009,\n",
      "title = {Reconstructing {Indian} population history},\n",
      "volume = {461},\n",
      "issn = {1476-4687},\n",
      "url = {https://www.nature.com/articles/nature08365},\n",
      "doi = {10.1038/nature08365},\n",
      "number = {7263},\n",
      "journal = {Nature},\n",
      "author = {Reich, David and Thangaraj, Kumarasamy and Patterson, Nick and Price, Alkes L. and Singh, Lalji},\n",
      "year = {2009},\n",
      "pages = {489--494}\n",
      "}\n",
      "\n",
      "@article{abdellaoui_genetic_2019,\n",
      "title = {Genetic correlates of social stratification in {Great} {Britain}},\n",
      "volume = {3},\n",
      "issn = {2397-3374},\n",
      "url = {https://www.nature.com/articles/s41562-019-0757-5},\n",
      "doi = {10.1038/s41562-019-0757-5},\n",
      "number = {12},\n",
      "journal = {Nature Human Behaviour},\n",
      "author = {Abdellaoui, Abdel and Hugh-Jones, David and Yengo, Loic and Kemper, Kathryn E. and Nivard, Michel G. and Veul, Laura and Holtz, Yan and Zietsch, Brendan P. and Frayling, Timothy M. and Wray, Naomi R. and Yang, Jian and Verweij, Karin J. H. and Visscher, Peter M.},\n",
      "year = {2019},\n",
      "pages = {1332--1342}\n",
      "}\n",
      "\n",
      "@article{gilbert_revealing_2022,\n",
      "title = {Revealing the recent demographic history of {Europe} via haplotype sharing in the {UK} {Biobank}},\n",
      "volume = {119},\n",
      "url = {https://www.pnas.org/doi/10.1073/pnas.2119281119},\n",
      "doi = {10.1073/pnas.2119281119},\n",
      "number = {25},\n",
      "journal = {{P}roceedings of the {N}ational {A}cademy of {S}ciences},\n",
      "author = {Gilbert, Edmund and Shanmugam, Ashwini and Cavalleri, Gianpiero L.},\n",
      "year = {2022},\n",
      "pages = {e2119281119}\n",
      "}\n",
      "\n",
      "@article{chiu_inferring_2022,\n",
      "title = {Inferring population structure in biobank-scale genomic data},\n",
      "volume = {109},\n",
      "issn = {0002-9297},\n",
      "url = {https://www.sciencedirect.com/science/article/pii/S0002929722000660},\n",
      "doi = {10.1016/j.ajhg.2022.02.015},\n",
      "number = {4},\n",
      "journal = {The {A}merican {J}ournal of {H}uman {G}enetics},\n",
      "author = {Chiu, Alec M. and Molloy, Erin K. and Tan, Zilong and Talwalkar, Ameet and Sankararaman, Sriram},\n",
      "year = {2022},\n",
      "pages = {727--737}\n",
      "}\n",
      "\n",
      "@article{nagar_socioeconomic_2021,\n",
      "title = {Socioeconomic deprivation and genetic ancestry interact to modify type 2 diabetes ethnic disparities in the {United} {Kingdom}},\n",
      "volume = {37},\n",
      "issn = {25895370},\n",
      "url = {https://linkinghub.elsevier.com/retrieve/pii/S2589537021002406},\n",
      "doi = {10.1016/j.eclinm.2021.100960},\n",
      "journal = {eClinicalMedicine},\n",
      "author = {Nagar, Shashwat Deepali and Nápoles, Anna María and Jordan, I. King and Mariño-Ramírez, Leonardo},\n",
      "year = {2021},\n",
      "pages = {100960}\n",
      "}\n",
      "\n",
      "@misc{slavevoyages,\n",
      "  author = {{Slave Voyages: The Trans-Atlantic Slave Trade Database}},\n",
      "  year   = {2023},\n",
      "  title  = {Trans-Atlantic Slave Trade - Estimates},\n",
      "  note   = {\\url{http://www.slavevoyages.org/estimates/BeZD1wTh}},\n",
      "  urldate = {2023-03-27}\n",
      "}\n",
      "\n",
      "@article{fortes-lima_anthropological_2021,\n",
      "title = {Anthropological genetics perspectives on the transatlantic slave trade},\n",
      "volume = {30},\n",
      "issn = {0964-6906},\n",
      "url = {https://doi.org/10.1093/hmg/ddaa271},\n",
      "doi = {10.1093/hmg/ddaa271},\n",
      "number = {R1},\n",
      "journal = {Human Molecular Genetics},\n",
      "author = {Fortes-Lima, Cesar and Verdu, Paul},\n",
      "year = {2021},\n",
      "pages = {R79--R87}\n",
      "}\n",
      "\n",
      "@article{lewis_getting_2022,\n",
      "\tkeywords = {mainbib},\n",
      "title = {Getting genetic ancestry right for science and society},\n",
      "volume = {376},\n",
      "url = {https://www.science.org/doi/full/10.1126/science.abm7530},\n",
      "doi = {10.1126/science.abm7530},\n",
      "number = {6590},\n",
      "journal = {Science},\n",
      "author = {Lewis, Anna C. F. and Molina, Santiago J. and Appelbaum, Paul S. and Dauda, Bege and Di Rienzo, Anna and Fuentes, Agustin and Fullerton, Stephanie M. and Garrison, Nanibaa’ A. and Ghosh, Nayanika and Hammonds, Evelynn M. and Jones, David S. and Kenny, Eimear E. and Kraft, Peter and Lee, Sandra S.-J. and Mauro, Madelyn and Novembre, John and Panofsky, Aaron and Sohail, Mashaal and Neale, Benjamin M. and Allen, Danielle S.},\n",
      "year = {2022},\n",
      "pages = {250--252}\n",
      "}\n",
      "\n",
      "@article{jessa_stalled_2019,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Stalled developmental programs at the root of pediatric brain tumors},\n",
      "\tvolume = {51},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-019-0531-7},\n",
      "\tdoi = {10.1038/s41588-019-0531-7},\n",
      "\tnumber = {12},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Jessa, Selin and Blanchet-Cohen, Alexis and Krug, Brian and Vladoiu, Maria and Coutelier, Marie and Faury, Damien and Poreau, Brice and De Jay, Nicolas and Hébert, Steven and Monlong, Jean and Farmer, W. Todd and Donovan, Laura K. and Hu, Yixing and McConechy, Melissa K. and Cavalli, Florence M. G. and Mikael, Leonie G. and Ellezam, Benjamin and Richer, Maxime and Allaire, Andréa and Weil, Alexander G. and Atkinson, Jeffrey and Farmer, Jean-Pierre and Dudley, Roy W. R. and Larouche, Valerie and Crevier, Louis and Albrecht, Steffen and Filbin, Mariella G. and Sartelet, Hervé and Lutz, Pierre-Eric and Nagy, Corina and Turecki, Gustavo and Costantino, Santiago and Dirks, Peter B. and Murai, Keith K. and Bourque, Guillaume and Ragoussis, Jiannis and Garzia, Livia and Taylor, Michael D. and Jabado, Nada and Kleinman, Claudia L.},\n",
      "\tyear = {2019},\n",
      "\tpages = {1702--1713}\n",
      "}\n",
      "\n",
      "@article{hennig_what_2015,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {What are the true clusters?},\n",
      "\turl = {http://arxiv.org/abs/1502.02555},\n",
      "\tjournal = {arXiv:1502.02555 [stat]},\n",
      "\tauthor = {Hennig, Christian},\n",
      "\tyear = {2015}\n",
      "}\n",
      "\n",
      "@article{kamiza_transferability_2022,\n",
      "title = {Transferability of genetic risk scores in {African} populations},\n",
      "volume = {28},\n",
      "issn = {1546-170X},\n",
      "url = {https://www.nature.com/articles/s41591-022-01835-x},\n",
      "doi = {10.1038/s41591-022-01835-x},\n",
      "number = {6},\n",
      "journal = {Nature Medicine},\n",
      "author = {Kamiza, Abram B. and Toure, Sounkou M. and Vujkovic, Marijana and Machipisa, Tafadzwa and Soremekun, Opeyemi S. and Kintu, Christopher and Corpas, Manuel and Pirie, Fraser and Young, Elizabeth and Gill, Dipender and Sandhu, Manjinder S. and Kaleebu, Pontiano and Nyirenda, Moffat and Motala, Ayesha A. and Chikowore, Tinashe and Fatumo, Segun},\n",
      "year = {2022},\n",
      "pages = {1163--1166}\n",
      "}\n",
      "\n",
      "@article{zabad_fast_2023,\n",
      "title = {Fast and accurate {Bayesian} polygenic risk modeling with variational inference},\n",
      "issn = {0002-9297},\n",
      "url = {https://www.sciencedirect.com/science/article/pii/S0002929723000939},\n",
      "doi = {10.1016/j.ajhg.2023.03.009},\n",
      "journal = {{The American Journal of Human Genetics}},\n",
      "author = {Zabad, Shadi and Gravel, Simon and Li, Yue},\n",
      "year = {2023}\n",
      "}\n",
      "\n",
      "@article{bennet_pleiotropy_2010,\n",
      "title = {Pleiotropy in the {Presence} of {Allelic} {Heterogeneity}: {Alternative} {Genetic} {Models} for the {Influence} of {APOE} on {Serum} {LDL}, {CSF} {Amyloid}-$\\beta$ 42, and {Dementia}},\n",
      "volume = {22},\n",
      "issn = {1387-2877},\n",
      "shorttitle = {Pleiotropy in the {Presence} of {Allelic} {Heterogeneity}},\n",
      "url = {https://content.iospress.com/articles/journal-of-alzheimers-disease/jad100864},\n",
      "doi = {10.3233/JAD-2010-100864},\n",
      "number = {1},\n",
      "journal = {Journal of Alzheimer's Disease},\n",
      "author = {Bennet, Anna M. and Reynolds, Chandra A. and Gatz, Margaret and Blennow, Kaj and Pedersen, Nancy L. and Prince, Jonathan A.},\n",
      "year = {2010},\n",
      "pages = {129--134}\n",
      "}\n",
      "\n",
      "@article{rasmussen-torvik_high_2012,\n",
      "title = {High {Density} {GWAS} for {LDL} {Cholesterol} in {African} {Americans} {Using} {Electronic} {Medical} {Records} {Reveals} a {Strong} {Protective} {Variant} in {APOE}},\n",
      "volume = {5},\n",
      "issn = {1752-8062},\n",
      "url = {https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1752-8062.2012.00446.x},\n",
      "doi = {10.1111/j.1752-8062.2012.00446.x},\n",
      "number = {5},\n",
      "journal = {Clinical and Translational Science},\n",
      "author = {Rasmussen-Torvik, Laura J. and Pacheco, Jennifer A. and Wilke, Russell A. and Thompson, William K. and Ritchie, Marylyn D. and Kho, Abel N. and Muthalagu, Arun and Hayes, M. Geoff and Armstrong, Loren L. and Scheftner, Douglas A. and Wilkins, John T. and Zuvich, Rebecca L. and Crosslin, David and Roden, Dan M. and Denny, Joshua C. and Jarvik, Gail P. and Carlson, Christopher S. and Kullo, Iftikhar J. and Bielinski, Suzette J. and McCarty, Catherine A. and Li, Rongling and Manolio, Teri A. and Crawford, Dana C. and Chisholm, Rex L.},\n",
      "year = {2012},\n",
      "pages = {394--399}\n",
      "}\n",
      "\n",
      "@article{sanna_fine_2011,\n",
      "title = {Fine {Mapping} of {Five} {Loci} {Associated} with {Low}-{Density} {Lipoprotein} {Cholesterol} {Detects} {Variants} {That} {Double} the {Explained} {Heritability}},\n",
      "volume = {7},\n",
      "issn = {1553-7404},\n",
      "url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002198},\n",
      "doi = {10.1371/journal.pgen.1002198},\n",
      "number = {7},\n",
      "journal = {PLOS Genetics},\n",
      "author = {Sanna, Serena and Li, Bingshan and Mulas, Antonella and Sidore, Carlo and Kang, Hyun M. and Jackson, Anne U. and Piras, Maria Grazia and Usala, Gianluca and Maninchedda, Giuseppe and Sassu, Alessandro and Serra, Fabrizio and Palmas, Maria Antonietta and Iii, William H. Wood and Njølstad, Inger and Laakso, Markku and Hveem, Kristian and Tuomilehto, Jaakko and Lakka, Timo A. and Rauramaa, Rainer and Boehnke, Michael and Cucca, Francesco and Uda, Manuela and Schlessinger, David and Nagaraja, Ramaiah and Abecasis, Gonçalo R.},\n",
      "year = {2011},\n",
      "pages = {e1002198}\n",
      "}\n",
      "\n",
      "@article{breitling_genetic_2015,\n",
      "title = {Genetic {Contribution} of {Variants} near {SORT1} and {APOE} on {LDL} {Cholesterol} {Independent} of {Obesity} in {Children}},\n",
      "volume = {10},\n",
      "issn = {1932-6203},\n",
      "url = {https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0138064},\n",
      "doi = {10.1371/journal.pone.0138064},\n",
      "number = {9},\n",
      "journal = {PLOS ONE},\n",
      "author = {Breitling, Clara and Gross, Arnd and Büttner, Petra and Weise, Sebastian and Schleinitz, Dorit and Kiess, Wieland and Scholz, Markus and Kovacs, Peter and Körner, Antje},\n",
      "year = {2015},\n",
      "pages = {e0138064}\n",
      "}\n",
      "\n",
      "@article{wang_associations_2022,\n",
      "title = {Associations of genetic variants of lysophosphatidylcholine metabolic enzymes with levels of serum lipids},\n",
      "volume = {91},\n",
      "issn = {1530-0447},\n",
      "url = {https://www.nature.com/articles/s41390-021-01549-9},\n",
      "doi = {10.1038/s41390-021-01549-9},\n",
      "number = {6},\n",
      "journal = {Pediatric Research},\n",
      "author = {Wang, Hui and Wang, Yang and Song, Jie-Yun and Zhang, Ping-Ping and Song, Qi-Ying and Li, Chen-Xiong and Li, Li and Wang, Hai-Jun},\n",
      "year = {2022},\n",
      "pages = {1595--1599}\n",
      "}\n",
      "\n",
      "@article{macleod_principles_2009,\n",
      "title = {Some principles and practices of genetic biobanking studies},\n",
      "volume = {33},\n",
      "issn = {0903-1936, 1399-3003},\n",
      "url = {https://erj.ersjournals.com/content/33/2/419},\n",
      "doi = {10.1183/09031936.00043508},\n",
      "number = {2},\n",
      "journal = {European Respiratory Journal},\n",
      "author = {MacLeod, A. K. and Liewald, D. C. M. and McGilchrist, M. M. and Morris, A. D. and Kerr, S. M. and Porteous, D. J.},\n",
      "year = {2009},\n",
      "pmid = {19181915},\n",
      "pages = {419--425}\n",
      "}\n",
      "\n",
      "@article{diaz-papkovich_review_2021,\n",
      "title = {A review of {UMAP} in population genetics},\n",
      "volume = {66},\n",
      "issn = {1435-232X},\n",
      "url = {https://www.nature.com/articles/s10038-020-00851-4},\n",
      "doi = {10.1038/s10038-020-00851-4},\n",
      "number = {1},\n",
      "journal = {Journal of Human Genetics},\n",
      "author = {Diaz-Papkovich, Alex and Anderson-Trocmé, Luke and Gravel, Simon},\n",
      "year = {2021},\n",
      "pages = {85--91}\n",
      "}\n",
      "\n",
      "@article{mcmahon_substantial_2017,\n",
      "\ttitle = {The {Substantial} {Interdependence} of {Wikipedia} and {Google}: {A} {Case} {Study} on the {Relationship} {Between} {Peer} {Production} {Communities} and {Information} {Technologies}},\n",
      "\tvolume = {11},\n",
      "\tissn = {2334-0770},\n",
      "\tshorttitle = {The {Substantial} {Interdependence} of {Wikipedia} and {Google}},\n",
      "\turl = {https://ojs.aaai.org/index.php/ICWSM/article/view/14883},\n",
      "\tdoi = {10.1609/icwsm.v11i1.14883},\n",
      "\tnumber = {1},\n",
      "\tjournal = {Proceedings of the International AAAI Conference on Web and Social Media},\n",
      "\tauthor = {McMahon, Connor and Johnson, Isaac and Hecht, Brent},\n",
      "\tyear = {2017},\n",
      "\tpages = {142--151}\n",
      "}\n",
      "\n",
      "\n",
      "@article{ongaro_evaluating_2021,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Evaluating the {Impact} of {Sex}-{Biased} {Genetic} {Admixture} in the {Americas} through the {Analysis} of {Haplotype} {Data}},\n",
      "\tvolume = {12},\n",
      "\tissn = {2073-4425},\n",
      "\turl = {https://www.mdpi.com/2073-4425/12/10/1580},\n",
      "\tdoi = {10.3390/genes12101580},\n",
      "\tnumber = {10},\n",
      "\tjournal = {Genes},\n",
      "\tauthor = {Ongaro, Linda and Molinaro, Ludovica and Flores, Rodrigo and Marnetto, Davide and Capodiferro, Marco R. and Alarcón-Riquelme, Marta E. and Moreno-Estrada, Andrés and Mabunda, Nedio and Ventura, Mario and Tambets, Kristiina and Achilli, Alessandro and Capelli, Cristian and Metspalu, Mait and Pagani, Luca and Montinaro, Francesco},\n",
      "\tyear = {2021},\n",
      "\tpages = {1580}\n",
      "}\n",
      "\n",
      "@article{korunes_sex-biased_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Sex-biased admixture and assortative mating shape genetic variation and influence demographic inference in admixed {Cabo} {Verdeans}},\n",
      "\tvolume = {12},\n",
      "\tissn = {2160-1836},\n",
      "\turl = {https://doi.org/10.1093/g3journal/jkac183},\n",
      "\tdoi = {10.1093/g3journal/jkac183},\n",
      "\tnumber = {10},\n",
      "\tjournal = {G3 Genes{\\textbar}Genomes{\\textbar}Genetics},\n",
      "\tauthor = {Korunes, Katharine L and Soares-Souza, Giordano Bruno and Bobrek, Katherine and Tang, Hua and Araújo, Isabel Inês and Goldberg, Amy and Beleza, Sandra},\n",
      "\tyear = {2022},\n",
      "}\n",
      "\n",
      "@article{marcheco-teruel_cuba_2014,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Cuba: {Exploring} the {History} of {Admixture} and the {Genetic} {Basis} of {Pigmentation} {Using} {Autosomal} and {Uniparental} {Markers}},\n",
      "\tvolume = {10},\n",
      "\tissn = {1553-7404},\n",
      "\turl = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004488},\n",
      "\tdoi = {10.1371/journal.pgen.1004488},\n",
      "\tnumber = {7},\n",
      "\tjournal = {PLOS Genetics},\n",
      "\tauthor = {Marcheco-Teruel, Beatriz and Parra, Esteban J. and Fuentes-Smith, Evelyn and Salas, Antonio and Buttenschøn, Henriette N. and Demontis, Ditte and Torres-Español, María and Marín-Padrón, Lilia C. and Gómez-Cabezas, Enrique J. and Álvarez-Iglesias, Vanesa and Mosquera-Miguel, Ana and Martínez-Fuentes, Antonio and Carracedo, Ángel and Børglum, Anders D. and Mors, Ole},\n",
      "\tyear = {2014},\n",
      "\tnote = {Publisher: Public Library of Science},\n",
      "\tpages = {e1004488}\n",
      "}\n",
      "\n",
      "@article{loh_mixed-model_2018,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Mixed-model association for biobank-scale datasets},\n",
      "\tvolume = {50},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-018-0144-6},\n",
      "\tdoi = {10.1038/s41588-018-0144-6},\n",
      "\tnumber = {7},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Loh, Po-Ru and Kichaev, Gleb and Gazal, Steven and Schoech, Armin P. and Price, Alkes L.},\n",
      "\tyear = {2018},\n",
      "\tpages = {906--908}\n",
      "}\n",
      "\n",
      "@article{zhou_saige-gene_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {{SAIGE}-{GENE}+ improves the efficiency and accuracy of set-based rare variant association tests},\n",
      "\tvolume = {54},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-022-01178-w},\n",
      "\tdoi = {10.1038/s41588-022-01178-w},\n",
      "\tnumber = {10},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Zhou, Wei and Bi, Wenjian and Zhao, Zhangchen and Dey, Kushal K. and Jagadeesh, Karthik A. and Karczewski, Konrad J. and Daly, Mark J. and Neale, Benjamin M. and Lee, Seunggeun},\n",
      "\tyear = {2022},\n",
      "\tpages = {1466--1469}\n",
      "}\n",
      "\n",
      "@article{mbatchou_computationally_2021,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Computationally efficient whole-genome regression for quantitative and binary traits},\n",
      "\tvolume = {53},\n",
      "\tissn = {1546-1718},\n",
      "\turl = {https://www.nature.com/articles/s41588-021-00870-7},\n",
      "\tdoi = {10.1038/s41588-021-00870-7},\n",
      "\tnumber = {7},\n",
      "\tjournal = {Nature Genetics},\n",
      "\tauthor = {Mbatchou, Joelle and Barnard, Leland and Backman, Joshua and Marcketta, Anthony and Kosmicki, Jack A. and Ziyatdinov, Andrey and Benner, Christian and O’Dushlaine, Colm and Barber, Mathew and Boutkov, Boris and Habegger, Lukas and Ferreira, Manuel and Baras, Aris and Reid, Jeffrey and Abecasis, Goncalo and Maxwell, Evan and Marchini, Jonathan},\n",
      "\tyear = {2021},\n",
      "\tpages = {1097--1103}\n",
      "}\n",
      "\n",
      "@article{qian_efficient_2014,\n",
      "title = {Efficient clustering of identity-by-descent between multiple individuals},\n",
      "volume = {30},\n",
      "issn = {1367-4803},\n",
      "url = {https://doi.org/10.1093/bioinformatics/btt734},\n",
      "doi = {10.1093/bioinformatics/btt734},\n",
      "number = {7},\n",
      "journal = {Bioinformatics},\n",
      "author = {Qian, Yu and Browning, Brian L. and Browning, Sharon R.},\n",
      "year = {2014},\n",
      "pages = {915--922}\n",
      "}\n",
      "\n",
      "@article{han_clustering_2017,\n",
      "title = {Clustering of 770,000 genomes reveals post-colonial population structure of {North} {America}},\n",
      "volume = {8},\n",
      "issn = {2041-1723},\n",
      "url = {https://www.nature.com/articles/ncomms14238},\n",
      "doi = {10.1038/ncomms14238},\n",
      "number = {1},\n",
      "journal = {Nature Communications},\n",
      "author = {Han, Eunjung and Carbonetto, Peter and Curtis, Ross E. and Wang, Yong and Granka, Julie M. and Byrnes, Jake and Noto, Keith and Kermany, Amir R. and Myres, Natalie M. and Barber, Mathew J. and Rand, Kristin A. and Song, Shiya and Roman, Theodore and Battat, Erin and Elyashiv, Eyal and Guturu, Harendra and Hong, Eurie L. and Chahine, Kenneth G. and Ball, Catherine A.},\n",
      "year = {2017},\n",
      "pages = {14238}\n",
      "}\n",
      "\n",
      "@article{nait_saada_identity-by-descent_2020,\n",
      "title = {Identity-by-descent detection across 487,409 {British} samples reveals fine scale population structure and ultra-rare variant associations},\n",
      "volume = {11},\n",
      "issn = {2041-1723},\n",
      "url = {https://www.nature.com/articles/s41467-020-19588-x},\n",
      "doi = {10.1038/s41467-020-19588-x},\n",
      "number = {1},\n",
      "journal = {Nature Communications},\n",
      "author = {Nait Saada, Juba and Kalantzis, Georgios and Shyr, Derek and Cooper, Fergus and Robinson, Martin and Gusev, Alexander and Palamara, Pier Francesco},\n",
      "year = {2020},\n",
      "pages = {6130}\n",
      "}\n",
      "\n",
      "@article{caggiano_disease_2023,\n",
      "\ttitle = {Disease risk and healthcare utilization among ancestrally diverse groups in the {Los} {Angeles} region},\n",
      "\tvolume = {29},\n",
      "\tissn = {1546-170X},\n",
      "\turl = {https://www.nature.com/articles/s41591-023-02425-1},\n",
      "\tdoi = {10.1038/s41591-023-02425-1},\n",
      "\tnumber = {7},\n",
      "\tjournal = {Nature Medicine},\n",
      "\tauthor = {Caggiano, Christa and Boudaie, Arya and Shemirani, Ruhollah and Mefford, Joel and Petter, Ella and Chiu, Alec and Ercelen, Defne and He, Rosemary and Tward, Daniel and Paul, Kimberly C. and Chang, Timothy S. and Pasaniuc, Bogdan and Kenny, Eimear E. and Shortt, Jonathan A. and Gignoux, Christopher R. and Balliu, Brunilda and Arboleda, Valerie A. and Belbin, Gillian and Zaitlen, Noah},\n",
      "\tyear = {2023},\n",
      "\tpages = {1845--1856}\n",
      "}\n",
      "\n",
      "@article{lawson_inference_2012,\n",
      "title = {Inference of {Population} {Structure} using {Dense} {Haplotype} {Data}},\n",
      "volume = {8},\n",
      "issn = {1553-7404},\n",
      "url = {https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1002453},\n",
      "doi = {10.1371/journal.pgen.1002453},\n",
      "number = {1},\n",
      "journal = {PLOS Genetics},\n",
      "author = {Lawson, Daniel John and Hellenthal, Garrett and Myers, Simon and Falush, Daniel},\n",
      "year = {2012},\n",
      "pages = {e1002453}\n",
      "}\n",
      "\n",
      "@article{belbin_toward_2021,\n",
      "title = {Toward a fine-scale population health monitoring system},\n",
      "volume = {184},\n",
      "issn = {0092-8674},\n",
      "url = {https://www.sciencedirect.com/science/article/pii/S0092867421003652},\n",
      "doi = {10.1016/j.cell.2021.03.034},\n",
      "number = {8},\n",
      "journal = {Cell},\n",
      "author = {Belbin, Gillian M. and Cullina, Sinead and Wenric, Stephane and Soper, Emily R. and Glicksberg, Benjamin S. and Torre, Denis and Moscati, Arden and Wojcik, Genevieve L. and Shemirani, Ruhollah and Beckmann, Noam D. and Cohain, Ariella and Sorokin, Elena P. and Park, Danny S. and Ambite, Jose-Luis and Ellis, Steve and Auton, Adam and Bottinger, Erwin P. and Cho, Judy H. and Loos, Ruth J. F. and Abul-Husn, Noura S. and Zaitlen, Noah A. and Gignoux, Christopher R. and Kenny, Eimear E.},\n",
      "month = apr,\n",
      "year = {2021},\n",
      "pages = {2068--2083.e11}\n",
      "}\n",
      "\n",
      "@article{dummer_canadian_2018,\n",
      "title = {The {Canadian} {Partnership} for {Tomorrow} {Project}: a pan-{Canadian} platform for research on chronic disease prevention},\n",
      "volume = {190},\n",
      "issn = {0820-3946, 1488-2329},\n",
      "shorttitle = {The {Canadian} {Partnership} for {Tomorrow} {Project}},\n",
      "url = {https://www.cmaj.ca/content/190/23/E710},\n",
      "doi = {10.1503/cmaj.170292},\n",
      "number = {23},\n",
      "journal = {CMAJ},\n",
      "author = {Dummer, Trevor J. B. and Awadalla, Philip and Boileau, Catherine and Craig, Camille and Fortier, Isabel and Goel, Vivek and Hicks, Jason M. T. and Jacquemont, Sébastien and Knoppers, Bartha Maria and Le, Nhu and McDonald, Treena and McLaughlin, John and Mes-Masson, Anne-Marie and Nuyt, Anne-Monique and Palmer, Lyle J. and Parker, Louise and Purdue, Mark and Robson, Paula J. and Spinelli, John J. and Thompson, David and Vena, Jennifer and Zawati, Ma’n and Consortium, with the CPTP Regional Cohort},\n",
      "year = {2018},\n",
      "pmid = {29891475},\n",
      "pages = {E710--E717}\n",
      "}\n",
      "\n",
      "\n",
      "@article{wojcik_genetic_2019,\n",
      "\ttitle = {Genetic analyses of diverse populations improves discovery for complex traits},\n",
      "\tvolume = {570},\n",
      "\tissn = {1476-4687},\n",
      "\turl = {https://www.nature.com/articles/s41586-019-1310-4},\n",
      "\tdoi = {10.1038/s41586-019-1310-4},\n",
      "\tnumber = {7762},\n",
      "\tjournal = {Nature},\n",
      "\tauthor = {Wojcik, Genevieve L. and Graff, Mariaelisa and Nishimura, Katherine K. and Tao, Ran and Haessler, Jeffrey and Gignoux, Christopher R. and Highland, Heather M. and Patel, Yesha M. and Sorokin, Elena P. and Avery, Christy L. and Belbin, Gillian M. and Bien, Stephanie A. and Cheng, Iona and Cullina, Sinead and Hodonsky, Chani J. and Hu, Yao and Huckins, Laura M. and Jeff, Janina and Justice, Anne E. and Kocarnik, Jonathan M. and Lim, Unhee and Lin, Bridget M. and Lu, Yingchang and Nelson, Sarah C. and Park, Sung-Shim L. and Poisner, Hannah and Preuss, Michael H. and Richard, Melissa A. and Schurmann, Claudia and Setiawan, Veronica W. and Sockell, Alexandra and Vahi, Karan and Verbanck, Marie and Vishnu, Abhishek and Walker, Ryan W. and Young, Kristin L. and Zubair, Niha and Acuña-Alonso, Victor and Ambite, Jose Luis and Barnes, Kathleen C. and Boerwinkle, Eric and Bottinger, Erwin P. and Bustamante, Carlos D. and Caberto, Christian and Canizales-Quinteros, Samuel and Conomos, Matthew P. and Deelman, Ewa and Do, Ron and Doheny, Kimberly and Fernández-Rhodes, Lindsay and Fornage, Myriam and Hailu, Benyam and Heiss, Gerardo and Henn, Brenna M. and Hindorff, Lucia A. and Jackson, Rebecca D. and Laurie, Cecelia A. and Laurie, Cathy C. and Li, Yuqing and Lin, Dan-Yu and Moreno-Estrada, Andres and Nadkarni, Girish and Norman, Paul J. and Pooler, Loreall C. and Reiner, Alexander P. and Romm, Jane and Sabatti, Chiara and Sandoval, Karla and Sheng, Xin and Stahl, Eli A. and Stram, Daniel O. and Thornton, Timothy A. and Wassel, Christina L. and Wilkens, Lynne R. and Winkler, Cheryl A. and Yoneyama, Sachi and Buyske, Steven and Haiman, Christopher A. and Kooperberg, Charles and Le Marchand, Loic and Loos, Ruth J. F. and Matise, Tara C. and North, Kari E. and Peters, Ulrike and Kenny, Eimear E. and Carlson, Christopher S.},\n",
      "\tyear = {2019},\n",
      "\tpages = {514--518}\n",
      "}\n",
      "\n",
      "@article{lin_admixed_2021,\n",
      "title = {Admixed {Populations} {Improve} {Power} for {Variant} {Discovery} and {Portability} in {Genome}-{Wide} {Association} {Studies}},\n",
      "volume = {12},\n",
      "issn = {1664-8021},\n",
      "url = {https://www.frontiersin.org/articles/10.3389/fgene.2021.673167},\n",
      "journal = {Frontiers in Genetics},\n",
      "author = {Lin, Meng and Park, Danny S. and Zaitlen, Noah A. and Henn, Brenna M. and Gignoux, Christopher R.},\n",
      "year = {2021}\n",
      "}\n",
      "\n",
      "@article{fang_harmonizing_2019,\n",
      "title = {Harmonizing {Genetic} {Ancestry} and {Self}-identified {Race}/{Ethnicity} in {Genome}-wide {Association} {Studies}},\n",
      "volume = {105},\n",
      "issn = {0002-9297},\n",
      "url = {http://www.sciencedirect.com/science/article/pii/S0002929719303386},\n",
      "doi = {10.1016/j.ajhg.2019.08.012},\n",
      "number = {4},\n",
      "journal = {{The American Journal of Human Genetics}},\n",
      "author = {Fang, Huaying and Hui, Qin and Lynch, Julie and Honerlaw, Jacqueline and Assimes, Themistocles L. and Huang, Jie and Vujkovic, Marijana and Damrauer, Scott M. and Pyarajan, Saiju and Gaziano, J. Michael and DuVall, Scott L. and O’Donnell, Christopher J. and Cho, Kelly and Chang, Kyong-Mi and Wilson, Peter W. F. and Tsao, Philip S. and Gaziano, J. Michael and Ramoni, Rachel and Breeling, Jim and Chang, Kyong-Mi and Huang, Grant and Muralidhar, Sumitra and O’Donnell, Christopher J. and Tsao, Philip S. and Muralidhar, Sumitra and Moser, Jennifer and Whitbourne, Stacey B. and Brewer, Jessica V. and Concato, John and Warren, Stuart and Argyres, Dean P. and Stephens, Brady and Brophy, Mary T. and Humphries, Donald E. and Do, Nhan and Shayan, Shahpoor and Nguyen, Xuan-Mai T. and Pyarajan, Saiju and Cho, Kelly and Hauser, Elizabeth and Sun, Yan and Zhao, Hongyu and Wilson, Peter and McArdle, Rachel and Dellitalia, Louis and Harley, John and Whittle, Jeffrey and Beckham, Jean and Wells, John and Gutierrez, Salvador and Gibson, Gretchen and Kaminsky, Laurence and Villareal, Gerardo and Kinlay, Scott and Xu, Junzhe and Hamner, Mark and Haddock, Kathlyn Sue and Bhushan, Sujata and Iruvanti, Pran and Godschalk, Michael and Ballas, Zuhair and Buford, Malcolm and Mastorides, Stephen and Klein, Jon and Ratcliffe, Nora and Florez, Hermes and Swann, Alan and Murdoch, Maureen and Sriram, Peruvemba and Yeh, Shing Shing and Washburn, Ronald and Jhala, Darshana and Aguayo, Samuel and Cohen, David and Sharma, Satish and Callaghan, John and Oursler, Kris Ann and Whooley, Mary and Ahuja, Sunil and Gutierrez, Amparo and Schifman, Ronald and Greco, Jennifer and Rauchman, Michael and Servatius, Richard and Oehlert, Mary and Wallbom, Agnes and Fernando, Ronald and Morgan, Timothy and Stapley, Todd and Sherman, Scott and Anderson, Gwenevere and Sonel, Elif and Boyko, Edward and Meyer, Laurence and Gupta, Samir and Fayad, Joseph and Hung, Adriana and Lichy, Jack and Hurley, Robin and Robey, Brooks and Striker, Robert and Sun, Yan V. and Tang, Hua},\n",
      "year = {2019},\n",
      "pages = {763--772}\n",
      "}\n",
      "\n",
      "@misc{government_of_canada_technical_2019,\n",
      "title = {Technical report on changes in response related to the census ethnic origin question: {Focus} on {Jewish} origins, 2016 {Census} integrated with 2011 {National} {Household} {Survey}},\n",
      "shorttitle = {Technical report on changes in response related to the census ethnic origin question},\n",
      "url = {https://www150.statcan.gc.ca/n1/pub/89-657-x/89-657-x2019009-eng.htm},\n",
      "author = {Smith, Trevor and McLeish, Scott},\n",
      "publisher = {{Statistics Canada}, {Government of Canada}},\n",
      "year = {2019}\n",
      "}\n",
      "\n",
      "@article{carlson_counter_2022,\n",
      "title = {Counter the weaponization of genetics research by extremists},\n",
      "volume = {610},\n",
      "url = {https://www.nature.com/articles/d41586-022-03252-z},\n",
      "doi = {10.1038/d41586-022-03252-z},\n",
      "number = {7932},\n",
      "journal = {Nature},\n",
      "author = {Carlson, Jedidiah and Henn, Brenna M. and Al-Hindi, Dana R. and Ramachandran, Sohini},\n",
      "year = {2022},\n",
      "pages = {444--447}\n",
      "}\n",
      "\n",
      "@article{roth_multiple_2016,\n",
      "title = {The multiple dimensions of race},\n",
      "volume = {39},\n",
      "issn = {0141-9870},\n",
      "url = {https://doi.org/10.1080/01419870.2016.1140793},\n",
      "doi = {10.1080/01419870.2016.1140793},\n",
      "number = {8},\n",
      "journal = {Ethnic and Racial Studies},\n",
      "author = {Roth, Wendy D.},\n",
      "year = {2016},\n",
      "pages = {1310--1338}\n",
      "}\n",
      "\n",
      "@article{vyas_hidden_2020,\n",
      "title = {Hidden in {Plain} {Sight} — {Reconsidering} the {Use} of {Race} {Correction} in {Clinical} {Algorithms}},\n",
      "volume = {383},\n",
      "issn = {0028-4793},\n",
      "url = {https://doi.org/10.1056/NEJMms2004740},\n",
      "doi = {10.1056/NEJMms2004740},\n",
      "number = {9},\n",
      "journal = {New England Journal of Medicine},\n",
      "author = {Vyas, Darshali A. and Eisenstein, Leo G. and Jones, David S.},\n",
      "year = {2020},\n",
      "pages = {874--882}\n",
      "}\n",
      "\n",
      "@article{choudhury_high-depth_2020,\n",
      "title = {High-depth {African} genomes inform human migration and health},\n",
      "volume = {586},\n",
      "issn = {1476-4687},\n",
      "url = {https://www.nature.com/articles/s41586-020-2859-7},\n",
      "doi = {10.1038/s41586-020-2859-7},\n",
      "number = {7831},\n",
      "journal = {Nature},\n",
      "author = {Choudhury, Ananyo and Aron, Shaun and Botigué, Laura R. and Sengupta, Dhriti and Botha, Gerrit and Bensellak, Taoufik and Wells, Gordon and Kumuthini, Judit and Shriner, Daniel and Fakim, Yasmina J. and Ghoorah, Anisah W. and Dareng, Eileen and Odia, Trust and Falola, Oluwadamilare and Adebiyi, Ezekiel and Hazelhurst, Scott and Mazandu, Gaston and Nyangiri, Oscar A. and Mbiyavanga, Mamana and Benkahla, Alia and Kassim, Samar K. and Mulder, Nicola and Adebamowo, Sally N. and Chimusa, Emile R. and Muzny, Donna and Metcalf, Ginger and Gibbs, Richard A. and Rotimi, Charles and Ramsay, Michèle and Adeyemo, Adebowale A. and Lombard, Zané and Hanchard, Neil A.},\n",
      "year = {2020},\n",
      "pages = {741--748}\n",
      "}\n",
      "\n",
      "@Book{wickham_2016,\n",
      "  author = {Hadley Wickham},\n",
      "  title = {ggplot2: Elegant Graphics for Data Analysis},\n",
      "  publisher = {Springer-Verlag New York},\n",
      "  year = {2016},\n",
      "  isbn = {978-3-319-24277-4},\n",
      "  url = {http://ggplot2.org}\n",
      "}\n",
      "\n",
      "@MISC{Hlavac2018-fy,\n",
      "  title       = \"stargazer: {Well-Formatted} Regression and Summary Statistics Tables\",\n",
      "  author      = \"Hlavac, Marek\",\n",
      "  institution = \"Central European Labour Studies Institute (CELSI)\",\n",
      "  year        =  2018,\n",
      "  url         = \"https://CRAN.R-project.org/package=stargazer\",\n",
      "  address     = \"Bratislava, Slovakia\"\n",
      "}\n",
      "\n",
      "@article{purcell_plink_2007,\n",
      "title = {{PLINK}: {A} {Tool} {Set} for {Whole}-{Genome} {Association} and {Population}-{Based} {Linkage} {Analyses}},\n",
      "volume = {81},\n",
      "issn = {0002-9297},\n",
      "shorttitle = {{PLINK}},\n",
      "url = {https://www.sciencedirect.com/science/article/pii/S0002929707613524},\n",
      "doi = {10.1086/519795},\n",
      "number = {3},\n",
      "journal = {The American Journal of Human Genetics},\n",
      "author = {Purcell, Shaun and Neale, Benjamin and Todd-Brown, Kathe and Thomas, Lori and Ferreira, Manuel A. R. and Bender, David and Maller, Julian and Sklar, Pamela and de Bakker, Paul I. W. and Daly, Mark J. and Sham, Pak C.},\n",
      "year = {2007},\n",
      "pages = {559--575}\n",
      "}\n",
      "\n",
      "@Manual{r_2018,\n",
      "    title = {R: A Language and Environment for Statistical Computing},\n",
      "    author = {{R Core Team}},\n",
      "    organization = {R Foundation for Statistical Computing},\n",
      "    address = {Vienna, Austria},\n",
      "    year = {2018},\n",
      "    url = {https://www.R-project.org/},\n",
      "  }\n",
      "  \n",
      "@article{lewis_human_2012,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Human {Cluster} {Evaluation} and {Formal} {Quality} {Measures}: {A} {Comparative} {Study}},\n",
      "\tissn = {1069-7977},\n",
      "\turl = {https://escholarship.org/uc/item/8rr3t8dx},\n",
      "\tnumber = {34},\n",
      "\tjournal = {Proceedings of the Annual Meeting of the Cognitive Science Society},\n",
      "\tauthor = {Lewis, Joshua M and Ackerman, Margareta and de Sa, Virginia R},\n",
      "\tyear = {2012},\n",
      "\tpages = {1870--1875}\n",
      "}\n",
      "\n",
      "@article{fatumo_roadmap_2022,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {A roadmap to increase diversity in genomic studies},\n",
      "\tvolume = {28},\n",
      "\tissn = {1546-170X},\n",
      "\turl = {https://www.nature.com/articles/s41591-021-01672-4},\n",
      "\tdoi = {10.1038/s41591-021-01672-4},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Nature Medicine},\n",
      "\tauthor = {Fatumo, Segun and Chikowore, Tinashe and Choudhury, Ananyo and Ayub, Muhammad and Martin, Alicia R. and Kuchenbaecker, Karoline},\n",
      "\tyear = {2022},\n",
      "\tpages = {243--250}\n",
      "}\n",
      "\n",
      "@article{sun_analyses_2022,\n",
      "\ttitle = {Analyses of biomarker traits in diverse {UK} biobank participants identify associations missed by {European}-centric analysis strategies},\n",
      "\tvolume = {67},\n",
      "\tcopyright = {2021 The Author(s), under exclusive licence to The Japan Society of Human Genetics},\n",
      "\tissn = {1435-232X},\n",
      "\turl = {https://www.nature.com/articles/s10038-021-00968-0},\n",
      "\tdoi = {10.1038/s10038-021-00968-0},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Journal of Human Genetics},\n",
      "\tauthor = {Sun, Quan and Graff, Misa and Rowland, Bryce and Wen, Jia and Huang, Le and Miller-Fleming, Tyne W. and Haessler, Jeffrey and Preuss, Michael H. and Chai, Jin-Fang and Lee, Moa P. and Avery, Christy L. and Cheng, Ching-Yu and Franceschini, Nora and Sim, Xueling and Cox, Nancy J. and Kooperberg, Charles and North, Kari E. and Li, Yun and Raffield, Laura M.},\n",
      "\tyear = {2022},\n",
      "\tpages = {87--93}\n",
      "}\n",
      "\n",
      "\n",
      "@article{huang_representativeness_2021,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {Representativeness {Is} {Not} {Representative}: {Addressing} {Major} {Inferential} {Threats} in the {UK} {Biobank} and {Other} {Big} {Data} {Repositories}},\n",
      "\tvolume = {32},\n",
      "\tissn = {1044-3983},\n",
      "\tshorttitle = {Representativeness {Is} {Not} {Representative}},\n",
      "\tdoi = {10.1097/EDE.0000000000001317},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Epidemiology},\n",
      "\tauthor = {Huang, Jonathan Yinhao},\n",
      "\tyear = {2021},\n",
      "\tpages = {189},\n",
      "}\n",
      "\n",
      "@article{watson_philosophy_2023,\n",
      "\tkeywords = {mainbib},\n",
      "\ttitle = {On the {Philosophy} of {Unsupervised} {Learning}},\n",
      "\tvolume = {36},\n",
      "\tissn = {2210-5441},\n",
      "\turl = {https://doi.org/10.1007/s13347-023-00635-6},\n",
      "\tdoi = {10.1007/s13347-023-00635-6},\n",
      "\tnumber = {2},\n",
      "\tjournal = {Philosophy \\& Technology},\n",
      "\tauthor = {Watson, David S.},\n",
      "\tyear = {2023},\n",
      "\tpages = {28}\n",
      "}\n"
     ]
    }
   ],
   "source": [
    "for line in lines:\n",
    "    if \"@\" in line:\n",
    "        pos1 = line.find(\"{\")\n",
    "        pos2 = line.find(\",\")\n",
    "        \n",
    "        cite = line[pos1+1:pos2]\n",
    "\n",
    "        if cite in citations_all:\n",
    "            line = line + (\"\\tkeywords = {mainbib},\\n\")\n",
    "#            print(line)\n",
    "    print(line.strip(\"\\n\"))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "771eb302",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.13"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
