We have presented a method of nonlinear dimensionality reduction that is compatible with the scale and composition of modern biobanks. With UMAP and HDBSCAN($\hat{\epsilon}$) we have uncovered a wide variety of patterns of fine-scale population structure in every biobank studied. Since the publication of \hyperref[chap:chapter2]{Chapter~2} , UMAP has become a standard method for visualization and exploratory data analysis in population genetics. Given the demand for cluster extraction noted in \hyperref[chap:chapter3]{Chapters~3} and \hyperref[chap:chapter4]{4}, it is possible that HDBSCAN($\hat{\epsilon}$)---or some other density clustering method---could see similar adoption.

The timing of the development UMAP coincides fortuitously with the growth of biobanks and the genomic revolution. Writing in his review of TDA in 2018, Wasserman asked, ``is it possible to derive low dimensional embedding methods that explicitly preserve topological features of the data? This is an interesting open question.''\citep{wasserman_topological_2018} This review was published in the same year that UMAP, an explicitly topological method, was released. Topological analysis and density clustering seem especially well-suited to the task of understanding population structure in diverse biobanks.

\section{The value of visualization}

One may ask: why visualize data at all? Are these figures simply decorations for manuscripts? Though analyses, inferences, and models can shed light on the biological world, there is an inherent value to being able to literally see your data and understand its structure. In \hyperref[chap:chapter3]{Chapter~3} we describe the relationship between dimensionality reduction and biological data as analogous to that of microscopes and biological samples. This analogy has been made before with respect to mathematics in general\citep{cohen_mathematics_2004}. Seeing our data can compel us to study it more deeply and to crystallize theoretical concepts. In population genetics, one of the most cited papers is ``Genes mirror geography within Europe''\citep{novembre2008europe}, which visualizes the relationship between PCA and isolation-by-distance. Although isolation-by-distance had been characterized almost a century earlier, the figure of the top principal components superimposed over Europe was an elegant illustration of the geographical distribution of human genetic variation.

With UMAP, we are able to map complex genetic structure down to $2$ or $3$ dimensions for visualization, examine the fine-scale structure that a method like PCA compresses, while preserving interesting signal in the data. Using auxiliary data---geographic data, population labels, phenotypic information, environmental variables, etc.---we can visually scan for patterns and generate hypotheses. In \hyperref[chap:chapter2]{Chapter~2} we present a variety of visualization methods, including colouring points by admixture levels to uncover gradients in ancestry and converting 3D UMAP coordinates to RGB colour levels for colouring maps. One common method used in single-cell studies is to colour UMAP plots by gene expression levels (e.g. \citep{jessa_stalled_2019} Figure~4h). A similar approach could be used in exploring allele distributions by clusters to see whether they are relatively more common in different populations---one such method, called the Allele Dispersion Score, was proposed by Correard et al. in 2022\citep{correard_allele_2022}. Though UMAP figures are now standard in genetic studies, most are limited to a simple 2D scatterplot coloured by some population label---in every biobank, there is certainly value in exploring deeper and using other variables.

There is also value in going beyond static $2$D figures. Software libraries such as plotly\citep{plotly} can generate interactive figures, are available for Python and R, and are straightforward to implement. Interactive exploration can assess how individual points fit within larger projections, rapidly identifying patterns or using them for diagnostics. Working in $3$D is much easier when the figures are interactive, and we have, e.g., explored the relationships between individuals and populations in the UKB. Finally, as noted in \hyperref[chap:chapter4]{Chapter~4}, clustering works well in dimensions above $2$---while it can be tempting to delineate clusters from $2$D figures alone, the lower the dimensionality of our representation, the less faithful it is to the data. Despite UMAP's now-widespread use, its ability to work in $3$ or more dimensions is, in my view, underappreciated.

\section{Implications for biomedical and epidemiological research}

\begin{figure}[h]
\centering
\begin{subfigure}{0.45\linewidth}
    \includegraphics[width=\linewidth]{main_figures/discussion/populations_1.png}
    \caption{}
    \label{fig:statistical_populations1}
\end{subfigure}
\hfill
\begin{subfigure}{0.45\linewidth}
    \includegraphics[width=\linewidth]{main_figures/discussion/populations_2.png}
    \caption{}
    \label{fig:statistical_populations2}
\end{subfigure}
\caption[The relationship between target and study populations]{\textbf{The study population should match the target population.} In \textbf{(a)}, the study population is a subset of the target population, so inferences drawn from it will apply to the target population. In \textbf{(b)}, the study population is not fully a subset of the target population, so its inferences will be less reliable.}
\label{fig:statistical_populations}
\end{figure}

In inferential statistics, we require some definition of a ``population''. To establish the scope of study we define a target population (the group to whom the research applies) and the study population (the group which is covered by the study and is, ideally, drawn from the target population as illustrated in Figure~\ref{fig:statistical_populations1})\citep{statcan2003}. Given the results in \hyperref[chap:chapter4]{Chapter 4}, density clustering with UMAP could potentially be used to define populations for study. It has several advantages over methods based on, e.g., $K$-means clustering: it clusters all or almost all individuals, including those with recent admixture or from uncommon ancestries; it is does not assume a number of populations $K$; it does not require reference panels; its assumptions of the structure of genetic data are more realistic. It also has advantages over variables like self-identification or country of birth that, though sometimes used as proxies for population structure, are not actually genetic. 

Unlike methods that try to, e.g., harmonize labels with genetic data, the density clustering approach is label-agnostic. This is beneficial when there is differential response in identification, as we saw in \hyperref[chap:chapter4]{Chapter 4}  with many individuals from non-majority populations simply identifying as ``other''. There is currently a significant bias in genomic studies, with a tendency to discard data from non-European populations, often citing concerns over low sample sizes or issues with population structure\citep{ben-eghan_dont_2020}. Approximately $86\%$ of genomic studies by 2021 were based on individuals of European ancestry, an increase from $81\%$ in 2016\citep{fatumo_roadmap_2022,kaplan_polygenic_2022}. Using UMAP and density clustering presents an avenue to boost sample sizes and include a more diverse array of populations in studies by not depending on clustering that is, essentially, based on population labels.

Biobanks are as complex as they are rich. Across visualizations in \hyperref[chap:chapter2]{Chapters~2} and \hyperref[chap:chapter4]{4}, we see patterns. Some appear quickly at the population level, as in Figure~\ref{fig:fig6} where we see population-level differences in height and blood cell count; others require iterative smoothing, such as  Figure~\ref{fig:smoothed_phenos} where we see residual structure in phenotypes after removing the effects of the top $40$ PCs. These patterns also appear in behavioural measures, such as smoking in Figure~\ref{fig:supp_ukb_smoking}. Figures~\ref{fig:supp_montage_ukbb_ns} and \ref{fig:supp_montage_ukbb_ew} show how geographic structure is closely tied to genetic structure. Using variables like admixture proportions, country of birth, immigration history, environmental data, etc., we can see the difficulty of disentangling genetic and environmental effects---structure and potential confounding are omnipresent.

Biobanks are generally not random samples of a population and suffer from selection bias\citep{huang_representativeness_2021}; there are genetic biases in enrolment itself\citep{pirastu_genetic_2021,benonisdottir_studying_2023}. Though the target population for a study may be, e.g., all European-ancestry individuals, if there is significant bias in the composition of a biobank, the study population will not be aligned with the target population and the inference may not be reliable. As such, though we have a prospective methodology to define populations, we must remember that our data are not necessarily representative of the population at large.

If we do not understand the gene-environment interplay, we may also incorrectly credit genetic architecture with a phenotype or disease that is actually caused by environmental factors. In a well-known paper in the 1980s\citep{rose_sick_2001}, epidemiologist Geoffrey Rose posed a thought experiment: what if everyone smoked $20$ cigarettes per day? Taking a purely genetic approach, we would (wrongly) deduce that lung cancer is a genetic disease. Similarly, there are non-genetic factors, such as pollution or discriminatory policies, that may disproportionately or exclusively impact certain groups. Such factors may not be measured at all within biobanks. It is therefore crucial to understand as much as possible the broader context of a population's environment.

\section{Clustering in population genetics}

As we have seen throughout this work (particularly \hyperref[chap:chapter4]{Chapter~4}), clusters are useful abstractions. The structure they find is ``real'' in the sense that it is a discrete measure of relatedness between individuals, with the caveat that the structure identified is conditioned on the data and methods. When we require a genetic definition of a population, such as to calculate a MAF or train/test a PGS or to explore biobank data efficiently, the approach of using UMAP and HDBSCAN($\hat{\epsilon}$) shows great promise.

Unlike problems like classification, clustering does not have a well-defined ground truth, and even the most basic definition, such as ``similar points are grouped together and dissimilar points are grouped separately'', can be self-contradictory\citep{ben-david_clustering_2018}. In a genetic cluster, individuals may be closely related to their immediate neighbours, but not to those in a far end of a cluster; they are grouped by similarity in one sense, but not separated by dissimilarity. Though various quality metrics exist, even untrained humans excel at assessing $2$D clusterings\citep{lewis_human_2012}---there is an element of ``I know it when I see it'', and since clustering is a data-driven computational method, it carries an air of objectivity. %This has led to manual cluster identification being common, and when paired with variables like population labels or country of birth, they may carry an air of objectivity.%, often with an air of objectivity.

%Paired with auxiliary data, e.g. country of birth, we can explore population structure much more effectively than before.

\begin{figure}[h]
\centering
    \includegraphics[width=0.75\linewidth]{main_figures/discussion/magritte.png}
\caption[The treachery of clustering]{\textbf{A cluster is only an abstraction, albeit a useful one.} A cluster can represent a population, but it is not a population itself.}
\label{fig:magritte}
\end{figure}

However, it is erroneous to describe clusters as ``objective'' measures of human population structure, both for the aforementioned reasons, and also because clustering changes based on many subjective inputs: algorithms, hyperparameters, subsets of data, etc\citep{watson_philosophy_2023}. This myth of objectivity in machine learning has perpetuated systemic biases because it spares methods from critical examination\citep{gebru_race_2020}. The cost of epistemological ignorance here varies, but at its most extreme, it has led to justification of racism and violence\citep{panofsky_genetic_2019,panofsky_how_2021,carlson_counter_2022}. Dimensionality reduction, visualization, and clustering are powerful tools, and we must understand not only their strengths and limitations, but their uses and misuses in the population genetics. 

Population genetic clusters are a discrete categorical measure of genetic ancestry, which is a continuous and multidimensional variable. Clusters that are clearly separated in one sample will ultimately fall into a continuum among larger and more diverse samples\citep{lewis_getting_2022}. We have seen through this thesis that clusters can split, merge, reappear, or disappear; in the same vein, in microscopy objects may appear or disappear depending on the depth and focus and type of microscope used. It would be folly to look at, e.g., a $2$D slice of a single cell and its sub-cellular components at one single time point and to conclude that this image defines the ``real'' cell. It is, however, a useful representation of the structure of the cell---provided we keep in mind that everything we see is conditioned on the sample (e.g. the cell type and state, the health of the organism, the environment in which the cell is viewed, etc).

%These same conditions apply to data.
Data sets do not spring forth naturally, nor do they examine themselves; they are sampled, collected, processed, analysed, and are products of societies and institutions situated within their own contexts and with their own biases\citep{dignazio_data_2020}. Population labels are an excellent example, since we often use them as shorthand for clusters and populations. In the UKB, there are $20$ unique values for ``ethnic background'' (see Table~\ref{table:ukb_eb_options}) and they are selected in a nested manner (first you select from a list of ``ethnic groups'', then from a second list of ``ethnic backgrounds''). It is not possible to identify as, for example, ``Chinese British'' or ``Mixed Black and Asian'', though there are likely individuals who would select those labels. The term ``Asian'' itself also has multiple meanings depending on context, including: a cultural label for South Asians (in British English), a cultural label for East Asians (North American English), or a label of geographic origin. These details shape how we perceive populations in biobanks, how we talk about them, and---eventually---the perceptions of those who consume our research. With genetic data, we are cartographers of unknown territory.
% We are cartographers of unknown territory; we also draw the shadows on the wall of Plato's cave.
% Intricate complexity cannot be overstated.

\section{Limitations}

Dimensionality reduction necessarily sacrifices information. To help us navigate the physical world, we use two-dimensional Euclidean maps of a three-dimensional spherical planet. This transformation introduces artificial tearing and distortion into world maps---PCA, $t$-SNE, UMAP, and density clustering are no different, although with maps there is a more intuitive understanding of how we distort information. After transforming data, we may see patterns or clusters that are artefactual, arising from unidentified batch effects or the distribution of genetic variants particular to a biobank. The relationships between visual objects may be misleading. Specific combinations of parameters may generate unusual results because of something particular to our data. Patterns may also form from noise. It is crucial to carry out multiple runs of dimensionality reduction, to examine any results with a sceptical eye, and to test hypotheses with secondary analyses.

UMAP works within a particular paradigm: it represents topology by approximating the local structure of the data. By patching together many local topological representations, it achieves a semblance of the global structure of data; however, the distances themselves do not have meaningful interpretations. PCA, which explicitly models global variation in data, has been interpreted in connection to TMRCA and F-statistics\citep{mcvean2009genealogical,peter_geometric_2022}. Several other nonlinear dimensionality reduction methods aim to balance global and local structure (e.g. PHATE\citep{moon2019visualizing}, POPVAE\citep{battey_visualizing_2021}). Since UMAP captures topology and not geometry, it approximates the shapes of data, but not how the shapes are positioned relative to one another. Consider three nested hollow spheres in $3$D: each sphere forms one connected shape, but a $2$D UMAP would represent them as three disconnected clusters\citep{herrmann_enhancing_2022}. This correctly identifies the three connected components but not their relationships to each other. Thus it is important to use UMAP in combination with other methods, providing snapshots of our data from many angles and resolutions.

\section{Future directions}

Topological data analysis has proven a useful addition to the population geneticist's toolbox. The work done in this thesis has been on genotypic data, and usually using the top principal components for computational reasons. Since the publication of the manuscripts, multi-threaded versions of UMAP and HDBSCAN($\hat{\epsilon}$) have been released. One avenue for future research is to work directly with genotype data that has not been processed with PCA. We explored this briefly in \hyperref[chap:chapter2]{Chapter~2} using 1KGP data; the improvement in compute time with multi-threading should make deeper investigation more feasible. 

There are possible connections between genetic topology and IBD. As noted in \hyperref[chap:chapter4]{Chapter~4}, UMAP and HDBSCAN($\hat{\epsilon}$) extract population structure that appears similar in scale to IBD studies of diverse biobanks; the two methods may capture similar information via different routes. Clusters extracted by our approach likely contain substructure themselves, and an approach that breaks clusters down further may reveal finer details.

Studying different subsets of populations or genetic data may be illuminating. We have noted studies in \hyperref[chap:chapter3]{Chapter~3} that use, e.g., structural variants rather than genotype data. Sex-bias in admixture has been noted in other studies (e.g. \citep{ongaro_evaluating_2021,korunes_sex-biased_2022,marcheco-teruel_cuba_2014}), whereas all of the analyses presented here have been on autosomal data and combining the sexes. Each of these approaches seems likely to uncover interesting patterns.

Within the biomedical realm, the confounding of GWAS and transferability of PGS are perennial areas of research. We presented a method of visualization in \hyperref[chap:chapter4]{Chapter~4} that illustrates how PCA adjustment affects populations across an entire biobank, with an additional analysis that studied the transferability of PGS. These approaches could also extend to studying the interplay between environment, genetics, and biomedical traits.