\setcounter{section}{-1}

\section{Preface}

In Chapter 1, we apply UMAP to population genetic data for the first time. Until this time, dimensionality reduction in population genetics was largely limited to PCA, with the occasional foray into methods like t-SNE. We provide an in-depth analysis and comparison of PCA, t-SNE, and UMAP on genotype data from three biobanks: the 1KGP, the HRS, and the UKB.

We explore a variety of visualization methods and illustrate the relative strengths of UMAP as well as its limitations compared to other methods. We use UMAP to reduce our data to $2$ dimensions and uncover fine-scale population structure in each of our data sets and colour it with sociodemographic data, geographic coordinates, phenotype distributions, admixture estimates, and other variables to reveal intricate patterns. We use UMAP to reduce our data to $3$ dimensions and translate this from $(x,y,z)$ coordinates to $(R,G,B)$ values to show how to use topological data analysis to reveal spatial gradients in population structure.

This manuscript became the basis of several UMAP analyses by other researchers in a wide variety of contexts. It is now standard for new biobanks to publish a UMAP plot of their population structure. This manuscript was released as a preprint on \textit{BioRxiv} in 2018 and published in \textit{PLoS Genetics} in 2019.

\clearpage

\section{Abstract}

Human populations feature both discrete and continuous patterns of variation. Current analysis approaches struggle to jointly identify these patterns because of modelling assumptions, mathematical constraints, or numerical challenges. Here we apply uniform manifold approximation and projection (UMAP), a non-linear dimension reduction tool, to three well-studied genotype datasets and discover overlooked subpopulations within the American Hispanic population, fine-scale relationships between geography, genotypes, and phenotypes in the UK population, and cryptic structure in the Thousand Genomes Project data. This approach is well-suited to the influx of large and diverse data and opens new lines of inquiry in population-scale datasets.

\section{Author summary}

The demographic history of human populations features varying geographic and social barriers to mating. Over time, these barriers have led to varying levels of genetic relatedness among individuals.  This population structure is informative about human history, and can have a significant impact on studies of medical genetics. Because population structure depends on myriad demographic, ecological, and social forces, a priori visualization is useful to identify subtle patterns of population structure. We use a dimension reduction method---UMAP---to visualize population structure in three genomic datasets and find previously unobserved patterns, revealing fine-scale population structure and illustrating differences between groups in traits such as white blood cell count, height, and FEV1, a measure of lung function. Using UMAP is computationally efficient and can identify fine-scale population structure in large population datasets. We find it particularly useful to reveal phenotypic variation among genetically related populations, and recommend it is a complement  to principal component analysis in primary data visualization. 

\section{Introduction}
Questions in medicine, anthropology, and related fields hinge on interpreting the deluge of genomic data provided by modern high-throughput sequencing technologies. Because genomic datasets are high-dimensional, their interpretation requires statistical methods that can comprehensively condense information in a manner that is understandable to researchers and minimizes the amount of data that is sacrificed. Both model-based and model-agnostic approaches to summarize data have played important roles in shaping our understanding of the evolution of our species (e.g., \citep{lawson2012inference, novembre2016recent, spence2018inference, eigen2006, Hellenthal747}).

Here we will focus on nonparametric approaches to visualize relatedness patterns among individuals within populations. If we consider unphased single nucleotide polymorphism (SNP) data, an individual genome can be represented as a sequence of integers corresponding to the number of copies of the alleles carried by the individual at each of the $L$ SNPs for which genotypes are available, with $L$ ranging from hundreds of thousands to hundreds of millions. Since each individual is represented as an $L$-dimensional vector, dimension reduction methods are needed to visualize the data.

Principal component analysis (PCA) is often the first dimensional reduction tool used for genomic data. It identifies and ranks directions in genotype space that explain most-to-least variance among individuals. Positions of individuals along directions of highest variance can then be used to summarize individual genotypes. PCA coordinates have natural genealogical interpretations in terms of expected times to a most recent common ancestor (TMRCA) \citep{mcvean2009genealogical}, and are used empirically to reveal admixture \citep{brisbin2012pcadmix}, continuous isolation-by-distance \citep{novembre2008europe, nelson2008population}, as well as technical artefacts. PCA coordinates are particularly well-suited to correct for population structure in GWAS\citep{eigen2006}.

The amount of information encoded in the highest-variance PCs increases slowly with sample size, so researchers typically examine multiple two-dimensional projections to lower-variance PCs to explore data. In this process, finer features of the data may be hidden by the projections or hard to interpret. To display finer features of the data in a two dimensional figure, we can use non-linear transformations that emphasize the local structure of the data. A popular method for such visualization is t-distributed stochastic neighbour embedding (t-SNE)\citep{maaten_visualizing_2008}. t-SNE has been used before to visualize SNPs\citep{platzer2013visualization}. Using data from the 1000 Genomes Project (1KGP)\citep{10002015global}, it groups individuals corresponding roughly to their continent of origin, with smaller ethnic sub-groups visible within the larger continental clusters\citep{li2017tsne}. However, t-SNE struggles with very large datasets, when a large number of locally optimal configurations make convergence to a globally satisfying solution difficult.

Uniform Manifold Approximation and Projection (UMAP) is a dimension reduction technique designed to model and preserve the high-dimensional topology of data points in the low-dimensional space\citep{2018arXivUMAP}. With genotype data, UMAP creates a neighbourhood around each individual's genetic coordinates and identifies a pre-selected number of neighbours to build high-dimensional manifolds. The end result is a patchwork of low-dimensional representations of neighbourhoods that groups genetically similar individuals together on a local scale while better preserving long-range topological connections to more distantly related individuals. The method has been successfully applied to single-cell RNA sequencing datasets \citep{umap2018singlecell}.

Non-linear dimension reduction methods tend to be computationally intensive. A common practice to reduce this burden is to first apply PCA to data, and perform dimensional reduction on data projected to leading principal components (PCs). In addition to being computationally advantageous, this discards noise that can confound non-linear approaches: population structure arising from $n$ isolated randomly-mating demes can be described by the leading $n-1$ PCs, with the following PCs describing stochastic variation in relatedness~\citep{eigen2006}. Selecting the leading PCs therefore has potential to extract meaningful population structure while filtering out stochastic noise. We explore different strategies to pre-process the data and investigate discrete and continuous population structure patterns present in large datasets of human genotypes: the 1KGP, the Health and Retirement Study (HRS)\citep{juster1995overview}, and the UK BioBank (UKBB)\citep{sudlow2015uk}, and compare UMAP's performance to t-SNE.  

\clearpage

\begin{figure}[ht]
    \centering
    \includegraphics[width=0.7\textwidth]{placeholder.png}
    \caption[Four methods of dimension reduction of 1KGP genotype data]{\textbf{Four methods of dimension reduction of 1KGP genotype data with population labels.} \textbf{(A)} PCA maps individuals in a triangle with vertices corresponding to African, Asian, and European continental ancestry. Discarding lower-variance PCs leads to overlap of populations with no close affinity, such as Central and South American populations with South Asians. \textbf{(B)} t-SNE forms groups corresponding to continents, with some overlap between European and Central and South American people. Smaller subgroups are visible within continental clusters. The cloud of peripheral points results from the method's poor convergence. \textbf{(C)} UMAP forms distinct clusters related to continent with clearly defined subgroups. Japanese, Finnish, Luhya, and some Punjabi and Telugu populations form separate clusters consistent with their population history\citep{10002015global}. \textbf{(D)} UMAP on the first 15 principal components forms fine-scale clusters for individual populations. Groups closely related by ancestry or geography, such as African Caribbean/African American, Spanish/Italian, and Kinh/Dai populations cluster together. Results using t-SNE on principal components are presented in Figure~\ref{fig:supp_megamontage_pc2_9}. Axes in UMAP and t-SNE are arbitrary. Since the algorithms prioritize local distances, long distances between clusters are not meaningful.}
    \label{fig:fig1}
\end{figure}

\clearpage

\section{Results}
\subsection{Fine-scale visualization of the 1KGP dataset}

The 1KGP contains genotype data of 3,450 individuals from 26 relatively distinct labeled populations\citep{10002015global}. Figure~\ref{fig:fig1} shows visualizations using PCA, t-SNE, UMAP, and UMAP with PCA pre-processing. Using UMAP and t-SNE on the genotype data presents clusters that are roughly grouped by continent, with UMAP showing a clear hierarchy of population and continental clusters, whereas t-SNE fails to assign many individuals to population clusters. Using either method on the top principal components leads to distinct population clusters and less defined continental structure. Adding more components results in progressively finer clusters until approximately 20 populations appear using 15 components; adding further components converges to results similar to using the entire genotype data (see Figures~\ref{fig:supp_megamontage_pc2_9}, \ref{fig:supp_megamontage_pc10_50}, and \ref{fig:supp_montage_1kgp_converge}). To investigate the population information contained in low-variance PCs, we performed UMAP on data projected onto PCs 100 to 3450 (i.e., without information about the leading 99 PCs). Figure~\ref{fig:supp_1kgp_3350} shows that population structure is still clearly visible.

Focusing on UMAP with the leading 15 principal components (Figure~\ref{fig:fig1}D), several population clusters reflect shared ancestries. British individuals from England and Scotland form a cluster mixed with those from Utah who claim Northern and Western European ancestry. Toscani and Iberian individuals form a cluster reflecting their Mediterranean heritage. African Americans in the Southwest US, African Caribbean individuals in Barbados, and some Puerto Ricans also form a cluster. Three East Asian clusters appear: one is largely Han and Southern Han individuals, another is comprised of the Chinese Dai in southern China and the Kinh from Vietnam, and the third is the Japanese population. Other clusters are comprised of Colombians and Peruvians, the Esan and Yoruba populations of Nigeria, and several South Asian populations. 

Within population clusters, family members were projected near each other within broader population groups. When UMAP was parameterized to use only 5 nearest neighbours, however, families often formed distinct clusters (Figure~\ref{fig:supp_1kgp_families}): Using few neighbours to build a manifold emphasizes closer relatedness.

A few individuals cluster with populations different from their label: some Mexican individuals cluster with Spanish and Italian populations; some Puerto Rican individuals cluster with African American and Caribbean populations; and one Gambian individual clusters with the Mende of Sierra Leone. Two populations form multiple clusters: Gujarati Indians in Houston, Texas and Punjabi people in Lahore, Pakistan. This clustering effect is robust to the number of components considered (Figure~\ref{fig:supp_megamontage_pc10_50}). Differentiation in the 1KGP Gujarati population has been previously identified through a PCA restricted to the Gujarati \citep{reich2009india}. Following a preprint version of the present article, 23andme released a statement \citep{23andme} arguing that one of the two clusters could be traced, via 23andMe participant recontact, to individuals from a group in Western India with shared ancestry and patronym[D. Poznik, 23andMe, personal communication]\citep{23andme}.

\subsection{Admixed individuals fall along a genetic continuum}
The 1KGP sampled individuals from relatively distinct populations, so the data are more likely to form clusters. Most medical cohorts, however, comprise larger numbers of individuals sampled across extended geographical areas. The HRS contains genotype data of 12,454 Americans from a variety of backgrounds. Using UMAP on the first 10 principal components, we demonstrate projections that present a collection of sub-populations and a continuum of genetic variation.

The HRS forms several large clusters, reflecting both ethnicity (Figure~\ref{fig:supp_umap_hrs_eth}) and admixture proportions (Figure~\ref{fig:umap_hrs_admix}). Gradients in admixture proportion are visible within the predominantly Hispanic cluster, but not within the predominantly Black cluster, perhaps because the variance in ancestry proportions is greater among Hispanics. The ``White Not Hispanic'' (WNH) group forms several interconnected clusters, and these do not correspond to broad geographical areas (Figure~\ref{fig:supp_hrs_born}). By generating the PC axes and UMAP embedding for the HRS data in Figure~\ref{fig:supp_umap_hrs_eth}, and projecting the 1KGP data onto it, we reveal substructure within the Hispanic cluster, groupings of Finnish individuals within the WNH groups, as well as Italian and Spanish individuals grouping near the White Hispanic population (Figure~\ref{fig:supp_hrs_1kgp_projected}). One group of WNH individuals regularly appears at the periphery of the main cluster and does not cluster with any 1KGP populations.

\subsection{Regional patterns in the Hispanic subpopulation}
Applying UMAP to self-identified Hispanic individuals in the HRS reveals clear groupings related to birth region in Figure~\ref{fig:fig2}A. The highlighted cluster consists almost entirely of individuals born in the Mountain Region of the United States. This cluster is not apparent when looking at a grid of pairwise plots of the first 8 principal components, provided in Figure~\ref{fig:supp_hrs_hisp_grid}, as the signal is distributed along PCs 3, 4, and 6. Even though continental admixture patterns do correlate with UMAP position (Figure~\ref{fig:supp_umap_hrs_hisp_admix}), these do not explain the Mountain Region cluster. Individuals from 1KGP populations do not appear in the cluster when projected to the UMAP embedding. The cluster possibly comprises the Hispano/Nuevomexicano population of the Southwest US, who have been present in the Mountain Region area long before the more recent immigrants from Latin America, and whose ancestry is expected to reflect both distinct Native ancestry and population-specific drift relative to other Hispanic populations. Such a cluster has been previously identified in AncestryDNA data using network-based clustering on identity-by-descent connections\citep{han2017clustering}; a recent preprint discusses the Mountain Region Hispanics with a more detailed historical description\citep{Jordan333609}. 

\clearpage

\begin{figure}[ht]
    \centering
    \begin{subfigure}{0.49\columnwidth}
    %\includegraphics[width=\columnwidth]{images/HRS_1000G_NP1_UMAP_PC7_NC2_NN15_MD05_pca_hrshisp_added1kgp_2018115153245_1kgp_hisp.jpeg}
    \includegraphics[width=\columnwidth]{placeholder.png}
    \caption{}
    \label{fig:supp_umap_hrs_hisp_1kgp_hisp}
    \end{subfigure}\hfill
    \unskip\ \vrule\
        \begin{subfigure}{0.49\columnwidth}
%    \includegraphics[width=\columnwidth]{images/Asthma_FullAsian_PC_QC_new_UMAP_PC5_NC2_NN15_MD05_201931173256_ethcob.jpeg}
    \includegraphics[width=\columnwidth]{placeholder.png}
    \caption{}
        \label{fig:umap_ukbb_indian}
    \end{subfigure}
    \caption[Applying UMAP to subsets of data can reveal deep population structure]{\textbf{Applying UMAP to subsets of data can reveal deep population structure.} \textbf{(a)} UMAP on the top $7$ principal components of the self-identified Hispanic population of the HRS reveals a cluster. Colouring the points by birthplace shows they were born almost entirely in the Mountain region (in green) of the United States (New Mexico, Arizona, Colorado, Utah, Nevada, Wyoming, Idaho, and Montana). When populations from the 1KGP are projected onto the UMAP embedding they do not map to the cluster. Six 1KGP populations are presented:
    CLM, Colombian in Medellin, Colombia;
    IBS, Iberian in Spain;
    MXL, Mexican in Los Angeles, California;
    PEL, Peruvian;
    PUR, Puerto Rican;
    TSI, Toscani in Italy. Figures~\ref{fig:supp_umap_hrs_hisp_admix} and \ref{fig:supp_umap_hrs_hisp_birth} present the same projection of individuals from the HRS coloured by estimated admixture proportions census region of birth, respectively. \textbf{(b)} UMAP on the top $8$ principal components of the self-identified Asian populations of the UKBB creates clusters. Indian individuals born in Kenya (in purple) form one such cluster. A version coloured by self-identified ethnicity is presented in Figure~\ref{fig:supp_umap_ukbb_asian_eth}.}
    \label{fig:fig2}
\end{figure}

\clearpage

\begin{figure}%
\centering
\begin{subfigure}{.5\columnwidth}
%\includegraphics[width=\columnwidth]{images/UKBB_PC0_PC1_eth_crop.jpeg}
    \includegraphics[width=\columnwidth]{placeholder.png}
\caption{}%
\label{fig:pc_ukbb}%
\end{subfigure}\hfill%
\begin{subfigure}{.5\columnwidth}
%\includegraphics[width=\columnwidth]{images/UKBB_UMAP_PC10_NN15_MD05_eth_labels.jpeg}%
    \includegraphics[width=\columnwidth]{placeholder.png}
\caption{}
\label{fig:umap_ukbb_pc10}
\end{subfigure}%
\caption[The UKBB coloured by self-reported ethnic background]{\textbf{The UKBB coloured by self-reported ethnic background.} \textbf{(a)} The first two principal components, showing the usual triangle with vertices corresponding to African, Asian, and European ancestries, and intermediate values indicating admixture or lack of relationship to the vertex populations. \textbf{(b)} UMAP on the first 10 principal components. The cluster of White British and White Irish individuals is greatly expanded, with the Irish forming a distinct sub cluster mixed with the White British population. South Asian and East Asian individuals form their separate clusters, as do individuals of African or Caribbean backgrounds. Population clusters are connected by "trails" comprised of large proportions of individuals with mixed backgrounds.
BA, Black African; 
BC, Black Caribbean; 
BG, Bangladeshi; 
CHN, Chinese; 
IND, Indian;
PK, Pakistani;
WB, White British;
WI, White Irish;
WBC, White and Black Caribbean; 
WBA, White and Black African; 
WAA, White and Asian;
AAB, Any other Asian Background; 
ABB, Any other Black Background;
AWB, Any other White Background;
AMB, Any other Mixed Background;
OEG, Other ethnic group.
}
\label{fig:fig3}
\end{figure}

\clearpage

\subsection{Population structure in the UKBB reflects local and global genetic variation}

The UKBB contains data on 488,377 individuals including genotypes, phenotypic measures and self-identified ethnic backgrounds. Figure~\ref{fig:fig3} compares UMAP to PCA applied to the UKBB. As expected, PCA captures major axes of variation emphasizing continental ancestry, whereas UMAP reveals finer structure. UMAP on the top 10 principal components reveals continuous and discrete population structure (Figure~\ref{fig:fig3}B): the patchwork of local topologies identifies multiple sub-populations, as well as continuous structure within populations and admixture gradients between populations. The result is a succinct illustration of the complex structure and population relationships in a large and multi-ethnic dataset.

The largest cluster in Figure~\ref{fig:fig3}B consists of the White British and Irish populations. The Irish population forms a sub-cluster, but many individuals are also scattered throughout the British-identifying population. Individuals identifying as Black African and Black Caribbean partially overlap, but admixed individuals form distinct trails leading to Asian and European clusters. Chinese individuals form a cluster, within a broader East Asian population; Indian, Pakistani, and Bangladeshi populations form a closely bound cluster as well. The East Asian and South Asian populations each have large clusters of individuals who identify as having an ``other Asian background'' or belonging to an ``other ethnic group''. The patchwork of genetic neighbourhoods is connected by trails of admixed individuals, which converge at a nexus of individuals with a variety of ethnicities. Many claim mixed ancestry, and there are clusters of individuals who belong to an ``other ethnic group''. Using data on countries of birth, we identified many finer groups in Figure~\ref{fig:supp_ukbb_cob}, and confirmed they appeared in intuitive areas with, e.g., Japanese and Filipino clusters being projected near Chinese clusters.

Figure~\ref{fig:fig4} presents the UMAP projection from Figure~\ref{fig:fig3}B coloured instead by geographical coordinates from the Ordnance Survey National Grid (OSGB1936), with distances defined as a north or east position relative to the Isles of Scilly. Geographic clusters form in the large White British grouping, reflecting the relationship between genetic and geographic distance, as has been observed in Europe and British-wide data\citep{novembre2008europe,leslie2015fine}. Figure~\ref{fig:fig3}B shows that the admixed individuals have UMAP coordinates next to White British individuals residing in the South East of the UK, where London is located (see also Figure~\ref{fig:supp_london_distance}, where individuals are coloured by distance from London). This likely reflects the high migration levels to the city and surrounding area: the UMAP projection attempted to preserve both the genetic similarity among admixed individuals and the relatedness with White British individuals in cosmopolitan areas. 

The detailed shape of extended clusters is not stable as we vary the number of PCs included, but the patterns mentioned above are preserved. Figures~\ref{fig:supp_montage_ukbb_eth}, \ref{fig:supp_montage_ukbb_ns}, and \ref{fig:supp_montage_ukbb_ew} show UMAP plots using the top $40$ PCs from the UKBB.

\clearpage

\begin{figure}
    \centering
    \begin{subfigure}{\columnwidth}
%    \includegraphics[width=0.8\columnwidth]{images/UKBB_UMAP_PC10_NN15_MD05_2018328174511_geo_permuted_10nn_sd50000_base50000_201883118582.pdf}
    \includegraphics[width=0.8\columnwidth]{placeholder.png}
    \caption{}
    \label{fig:umap_ukbb_geo_osgb}
    \end{subfigure}
      \caption[UMAP captures relationships between population structure and geography]{\textbf{UMAP captures relationships between population structure and geography.} Each individual is coloured by their geographical coordinates of residence. Coordinates follow the UKBB's OSGB1936 geographic grid system and represent distance from the Isles of Scilly, which lie southwest of Great Britain. The left image colours individuals by their north-south (``northing'') coordinates, and the right image colours them by their east-west (``easting'') coordinates. Adding more components creates finer clusters  (Figures~\ref{fig:supp_montage_ukbb_ns} and \ref{fig:supp_montage_ukbb_ew}). Northing values were truncated between $100$km and $700$km, and easting values were truncated between $200$km and $600$km.}
\label{fig:fig4}
\end{figure}

\clearpage

\begin{figure}[h!]
  \centering
  \begin{subfigure}[b]{0.4\linewidth}
    %\includegraphics[width=\linewidth]{images/UKBB_mapsInMaps_2D3_permute50.jpg}
        \includegraphics[width=\columnwidth]{placeholder.png}
    \caption{}
    \label{fig:umap_ukbb_geo_map}
  \end{subfigure}
  \begin{subfigure}[b]{0.4\linewidth}
%    \includegraphics[width=\linewidth]{images/meanWorldMap.jpg}
    \includegraphics[width=\columnwidth]{placeholder.png}
    \caption{}
  \label{fig:umap_world_map}
  \end{subfigure}
  \caption[Maps coloured by 3D UMAP projections]{\textbf{Maps coloured by 3D UMAP projections of the top 20 principal components of the UKBB.} Each individual is assigned a 3D RGB vector based on 3D UMAP coordinates (a flattened projection is in the top right of panel A). Individuals who are closer to each other in the projection will be closer in colour in the maps. More details on colouring, as well as randomization of points to protect participant privacy, are available in the materials and methods. \textbf{(a)} Each point is an individual placed based on where they live. Patterns in genetic similarity are visible in Scotland, South England, North and South Wales, the East and West Midlands, and major urban centres. \textbf{(b)} Geographic distribution of UMAP coordinates. Using the country of birth of individuals in the UKBB, we colour countries by the closeness in 3D UMAP space of those born there. Broad patterns of similarity appear in East Asia, South Asia, North African and the Middle East, West Africa, and South America. Differences between neighbouring countries can reflect both ancient population structure and recent differences in migration history. Evidence of migrations related to colonialism are visible with, e.g., European ancestry in South Africa and South Asian ancestry in Kenya and Tanzania. Because of the large number of White British individuals born abroad, to avoid skewing the colour scale they were not included unless they were born in the UK, Europe, Australia, Canada, or the United States, where UKBB participants already tended to have European ancestry. Zoomed maps of East Asia, the Caribbean, and Europe are available in Figures~\ref{fig:supp_umap_ukbb_asia}, \ref{fig:supp_umap_ukbb_car}, and \ref{fig:supp_umap_ukbb_eur}, respectively.}
  \label{fig:fig5}
\end{figure}

\clearpage

As an alternate visualization of geography and genetic diversity, we performed a 3D UMAP projection and converted the UMAP coordinates into RGB values, allowing us to plot individuals on a map of Great Britain, emphasizing both spatial gradients of genetic relatedness and increased diversity in urban centers (Figure~\ref{fig:fig5}A). The geographical patterns outside major urban centers are similar to those reported in~\citep{leslie2015fine} using the haplotype-based CHROMOPAINTER on British individuals whose grandparents lived nearby. Using data about country of birth, we performed a similar analysis of a world map in Figure~\ref{fig:fig5}B, revealing subtle regional variation around the world. 

Similarly to UMAP, t-SNE applied to the UKBB data both displays diversity within the "White British" population and identifies clusters among other groups. However, it has three drawbacks: it is much slower, requiring 2.26 hours for its first thousand iterations alone on 10 principal components against UMAP's 14 minutes; it fails to find a global optimum, which results in a scattering of individuals and groups that are not stable across independent runs; and it does not identify continuity between different continental groups resulting from admixture (Figure~\ref{fig:supp_ukbb_tsne}). 

\subsection{Identifying patterns in phenotype variation related to genetic population structure} 

\begin{figure}
    \centering
    \begin{subfigure}{\columnwidth}
%    \includegraphics[width=0.8\columnwidth]{images/UKBB_UMAP_PC10_NN15_MD05_2018328174511_geo_permuted_10nn_sd50000_base50000_201883118582.pdf}
    \includegraphics[width=0.8\columnwidth]{placeholder.png}
    \caption{}
    \label{fig:umap_height_female}
    \end{subfigure}
      \caption[UMAP captures relationships between population structure and phenotype heterogeneity.]{\textbf{UMAP captures relationships between population structure and phenotype heterogeneity.} Females from the UMAP projection in Figure~\ref{fig:fig3}B, coloured by age-adjusted difference from mean population height (left) and leukocyte counts (right). Individuals with missing data were excluded. To protect participant privacy, data in these images has been randomized as explained in the materials and methods section.}
\label{fig:fig6}
\end{figure}

Covariates such as height and leukocyte count (Figure~\ref{fig:fig6}) and autoimmune and asthma-related measures (Figures~\ref{fig:supp_ukbb_basophill_f} to \ref{fig:supp_ukbb_neutrophill_m}) correlate strongly with both discrete and continuous population structure. Several populations in Figure~\ref{fig:fig6}, including South Asian, East Asian, African, and others have noticeably lower-than-average heights. More subtle patterns are also visible: the area of the projection in Figure~\ref{fig:fig3}B with the cluster of White Irish people appears more blue than the main body of White British individuals. To quantify and statistically test these qualitative observations, we performed an unpaired two sample t-test of self-identified White Irish and White British individuals and found British males taller on average by 0.846cm (p-value $2.10\times 10^{-23}$) and British females by 0.763cm (p-value $3.65\times 10^{-23}$) (see Figures~\ref{fig:supp_box_height_f} and \ref{fig:supp_box_height_m} for boxplots). Height differences between Irish and British populations have been previously observed but the direction of the difference is not consistent\citep{robinson2015population,komlos1994stature}.

In clinical settings, baseline Forced Expiratory Volume in 1 second (FEV1) is determined via equations that include ethnicity or race\citep{quanjer2012multi}, but studies in several populations have shown that there is considerable variation based on ancestry, even within self-defined ethnicity\citep{ortega2015effect}. Figures~\ref{fig:supp_ukbb_fev_f} and \ref{fig:supp_ukbb_fev_m} show strong correlations with genetic clustering: certain populations---South Asian, African, and Caribbean---have considerably lower measurements on average (see Figures~\ref{fig:supp_box_fev_f} and \ref{fig:supp_box_fev_m} for boxplots and p-values). 
 
Notably, there appears to be a juncture in the admixed population, highlighted in Figure~\ref{fig:supp_comparison_fev_afr}, where the distribution of FEV1 changes. This roughly corresponds to the transition from Black African/Caribbean individuals to those who identified having mixed backgrounds. Boxplots and statistical testing suggest that relative to White British populations, FEV1 values are significantly lower for Black African and Black Caribbean populations, but not for White and Black Caribbean and White and Black African populations (Figures~\ref{fig:supp_box_fev_f} and \ref{fig:supp_box_fev_m}).

Figure~\ref{fig:supp_comparison_fev_chi_eur} further suggests a difference in FEV1 between those who self-identified as Chinese and a nearby cluster enriched in individuals born in Japan; to our knowledge there have not been studies into differences in FEV1 between these populations. To focus on individuals of Asian ancestry (rather than, e.g., individuals born in Japan but who have European ancestry), we first selected the individuals whose UMAP coordinates were near the Chinese cluster. We then focused on individuals born in Japan, Malaysia, and the Philippines as well as the self-identified Chinese population. These four groups are mutually exclusive and are shown in Figure~\ref{fig:supp_fev_test_pops}. After adjusting for age, age$^2$, height, and sex, an unpaired two-sample t-test shows those born in Japan have a higher mean FEV1 than Chinese individuals by 0.224 (p-value $2.787\times 10^{-15}$). By sex, there is a difference of 0.213 (p-value $5.4\times 10^{-13}$) among females and 0.317 (p-value $5.1\times 10^{-4}$) among males, though there are considerably fewer males in the sample (distributions presented in Figure~\ref{fig:supp_fev_ridgeplots}). For comparison, the adjusted difference between self-identified African and British individuals in the UKBB is 0.762 (p-value $2.2\times 10^{-16}$).

\subsection{Comparing t-SNE and UMAP}
Identifying the best dimension reduction technique is challenging, both because the ``best'' representation depends on context, and because convergence issues may mean that a good theoretical model for dimensional reduction might perform poorly because of challenges in numerical optimization. To assess whether the relatively poor performance of t-SNE could be due to convergence rather than a flawed model, we used UMAP to preprocess the UKBB data and provide a starting point to a standard t-SNE implementation. This led to representations that were objectively better (according to the t-SNE metric) than the default t-SNE implementation (Figure~\ref{fig:supp_tsne_umap_compare_ukbb_graph}). Yet, these representations were much less detailed than the UMAP embedding provided as a starting point (Figure~\ref{fig:supp_tsne_umap_compare_ukbb}).  Given these results, we recommend UMAP over t-SNE for large and diverse genomic datasets.

\section{Discussion}
Methods such as UMAP and t-SNE focus on preserving local distances to reveal fine-scale structure in populations, and in the process may preserve aspects of global structure as well. In contrast, PCA preserves long range distances but hides finer-scale details. Hierarchical clustering of networks has also successfully detected fine-scale population structure using identity-by-descent similarity by attempting to preserve relations between global networks and smaller local ones (e.g., \citep{han2017clustering}). We speculate that the addition of weak constraints favouring the preservation of longer distances in UMAP-like approaches has the potential of preserving the desirable local properties while encouraging more intuitive positioning of clusters on a global scale.

UMAP comprehensively illustrates genotypic information at fine scales and within the context of global population structure. It is easy to use and fast: given PCA data and a desktop computer, UMAP can be performed in 15 to 25 minutes on a sample of hundreds of thousands of individuals over tens of dimensions. It excels with larger datasets containing individuals with admixed backgrounds, which present discrete and continuous population structure.

Using UMAP reveals clusters that would have been difficult to identify via pairwise PCA plots or Admixture analysis, such as the geographically restricted cluster within the Hispanic population of the HRS, or the splits within the Gujarati and Punjabi population samples in the 1KGP. More importantly, UMAP helps reveal patterns of covariation between geography, phenotypes and genotypes. Traits such as height showed continuous variation across admixture edges and geographic gradients, as expected from genetically controlled complex traits, and others, such as leukocyte counts or FEV1, showed sharper boundaries and non-linear behavior consistent with the existence of strong regional environmental influences. 

We found that pre-processing the data with PCA allowed for time savings, but identifying an optimal number of PCs to use is challenging. Groupings on ethnicity formed slowly as PCs were added until reaching a stable number around 10 to 15 PCs. Geographical patterns in the UKBB continued to appear even up to 40 components, as visible in Figures~\ref{fig:supp_montage_ukbb_ns} and \ref{fig:supp_montage_ukbb_ew}. 

\subsection{Caveats}
In contrast to PCA, UMAP has more adjustable parameters. Changing the PC cutoff, minimum distance, and number of neighbours can change characteristics of the visualizations. Using a minimal number of neighbours (e.g. 5 rather than the default 15) can result in the formation of disjoint clusters comprised of related family members (Figure~\ref{fig:supp_1kgp_families}), and using a low minimum distance (e.g. 0.001 rather than the default 0.1) can result in clusters becoming more compact, losing visual detail. We used a minimum distance of 0.5 and 15 neighbours; however, default suggested parameters in UMAP generally perform well across datasets. 

In the absence of clear theoretical rationales, we suggest to use as many PCs as are available and computationally feasible, even though we sometimes found that a lower number of PCs led to a simpler shape that facilitated discussion (e.g. Figure~\ref{fig:fig3}B). Overall, we recommend reporting on a range of parameter values and following up on observations with statistical testing. 

Like most non-linear methods, UMAP lacks direct interpretability. It emphasizes local distances over global distances; while points that are very close in UMAP space are likely close in the original data, points that are distant in UMAP space are not necessarily very different in the original data. Disconnected clusters may also change their positions relative to other clusters over the course of multiple projections, as in Figure~\ref{fig:supp_megamontage_pc10_50}. For these reasons, UMAP coordinates should not be used as GWAS covariates or for quantifying distances between populations. UMAP is sensitive to sample sizes and spends more visual space on populations with larger sample sizes. This is useful to identify significant patterns in a cohort, but it makes comparing visualization across cohorts difficult and may appear to exaggerate the genetic variation within the most sampled populations, such as the White British population in the UKBB. We did not assign meaning to wiggles in UMAP figures, which occurred consistently in the UKBB but may be an artefact of the dimensional reduction strategy rather than a meaningful feature of the data. Hand-waving interpretations of pretty plots have a history of getting population geneticists in trouble (as pointed out, e.g., in \citep{novembre2008interpreting}): visualization is not a replacement for statistical testing.

With these caveats in mind, a priori data visualization plays a central role in quality control, hypothesis generation, and confounder identification for a wide range of genomic applications. Non-linear approaches, despite their limitations, become increasingly useful as the size of datasets increases. UMAP, in particular, reveals a wide range of features that would not be apparent using linear maps. Given its ease of use, broad applicability, and low computational cost, we propose that UMAP should become a default companion to PCA and other population structure visualization and inference methods in large genomic cohorts.

\section{Materials and methods}
We used genotype data from 12,454 individuals from the Health and Retirement Study (HRS), genotyped on the Illumina Human Omni 2.5M platform\citep{juster1995overview}. Principal components were computed in PLINK v1.90b5.2 64-bit\citep{purcell2007plink} using variants with a minor allele frequency greater than 0.05, Hardy-Weinberg p-value of more than $1\times 10^{-6}$, and genotype missing rate of less than $0.1$, and sample with genotype missing rate of less than $0.1$. We used the principal components of genotype data from 488,377 individuals in the UK BioBank (UKBB) as computed by the cohort \citep{sudlow2015uk}. We used genotype data from 3,450 individuals from the 1KGP project using Affy 6.0 genotyping\citep{10002015global}.

The HRS contains genotype data of 12,454 American individuals across all 50 states who have provided racial identity (10,434 White, 1,652 Black, 368 Other) as well as whether they identify as Hispanic (1,203 total) and, if so, whether they identify as Mexican-American (705 total)\citep{juster1995overview}. We crossed these three variables to form a composite self-reported ethnicity resulting in 10 categories (e.g. White Hispanic Mexican-American), and considered birth regions based on the 10 census regions and divisions used by the US Census Bureau. Admixture proportions for each individual were estimated in \citep{baharian2016great} by assuming ancestral African, Asian, and European populations using RFMIX \citep{Maples:2013fia}. We have scaled each of the three proportions to values between 0 and 255 (with 100\% corresponding to 255), to colour individual points by their estimated admixture represented by RGB where red, green, and blue respectively correspond to African, European, and Asian/Native American ancestry. To project 1KGP data on HRS embeddings, as in Figures~\ref{fig:fig2}A and \ref{fig:supp_hrs_1kgp_projected}, we created the PC axes and UMAP embedding for the HRS data and then projected the 1KGP data onto them.

The UKBB provides genotype data on 488,377 individuals along with self-identified ethnic background in a hierarchical tree-structured dictionary. Participants provided ethnic background on two occasions. We used the initial ethnicity after finding minimal differences between the two. The dataset is majority White (88.3\% British, 2.6\% Irish, 3.4\% other), with large populations identifying as Black (1.6\% either African, Caribbean, or other), Asian (1.9\% either Indian, Pakistani, Bangladeshi, or other), Chinese (0.3\%), an other ethnic group (0.8\%), mixed ethnicity (0.6\%), or an unavailable response (0.5\%). 

Scripts for all tests and plotting functions can be found on \url{https://github.com/diazale/gt-dimred} with a command line script for UMAP available at \url{https://github.com/diazale/gt-dimred/scripts/general_umap_script.py}. A demo version using freely available 1KGP data is available at \url{https://github.com/diazale/1KGP_dimred}. PCA and standard t-SNE were done with Scikit-learn\citep{scikit-learn}. UMAP was performed using a Python implementation\citep{2018arXivUMAP}. Statistical testing was done in SciPy\citep{scipy}, StatsModels\citep{seabold2010statsmodels}, and R\citep{r2013}. Visualizations were created with Matplotlib\citep{Hunter2007} and ggplot2\citep{ggplot2016}, and maps were made with Natural Earth.

Both UMAP and t-SNE feature a number of adjustable parameters. Among the parameters that we varied, the number of PCs used in pre-processing of the data has the largest effect for both methods (see Figures~\ref{fig:supp_megamontage_pc2_9} and \ref{fig:supp_megamontage_pc10_50}). With UMAP, there are other parameters, such as the learning rate and the distance metric; these were left to the default values.

We tested different choices for perplexity in t-SNE. The default value of 30 provided comparable performance to other parameter choices.  Similarly, we tested different parameter choices for UMAP, with the clearest results generated by specifying 15 nearest neighbours (the default value) and a ``minimum distance'' between points in low dimensions of 0.5. UMAP developers described ``sensible'' values for nearest neighbours as between 5 and 50 and minimum distance between 0.5 and 0.001. Tuning these parameters will not change qualitative results much but may make patterns easier to identify. Increasing the number of neighbours will increase the computational load, and a smaller minimum distance can break the connectivity between clusters, though the same individuals will continue to group together.

UMAP and t-SNE projections were carried out on an iMac with a 3.5GhZ Intel Core i7 processor, 32 GB 1600 MHz DDR3 of RAM, and an NVIDIA GeForce GTX 775M 2048 MB graphics card.

Colours for maps in Figures~\ref{fig:fig5}A, \ref{fig:fig5}B, \ref{fig:supp_umap_ukbb_asia}, \ref{fig:supp_umap_ukbb_car}, and \ref{fig:supp_umap_ukbb_eur} were determined by projecting data to 3D and using each 3D coordinate as an RGB coordinate. For the world map, countries were determined using the country of birth variable, with a country's colour being determined by the mean $x$, $y$, and $z$ values of all individuals born in that country. Because many self-identified White British individuals were born abroad, including them everywhere would skew the colour scheme; they were included only if they were born in the UK, Europe, Australia, Canada, or the United States. This approach to colouring is sensitive to sample sizes as UMAP will give more space to larger populations. 

To reduce the potential risks for re-identification from results in this publication, data has been randomly permuted so that the population characteristics are preserved but individual-level data is not presented directly in the figures. We rounded each attribute to an attribute-specific number of bins, and then permuted the data in the following way: 
For each point (i.e. each individual) in UMAP visualizations, and each attribute, we identified the 9 nearest neighbouring points, and copied the attribute from a randomly selected neighbour (thus allowing for the possibility of one value being printed twice).  Because this process is done independently for each visualization, a given point shown on the figure will copy values from different randomly selected individuals. Projections coloured by participants' spatial coordinates have random noise added (normally distributed about 0 with a standard deviation of 50km) before binning to the nearest 50km. Projections coloured by participants' distance from London have random noise added (normally distributed about 0 with a standard deviation of 5km) before binning to the nearest km. For each point in Figure~\ref{fig:fig5}A we identified the nearest 50 neighbouring individuals and copied the colour value from a randomly selected neighbour.

\subsection{Ethics statement}
HRS data was accessed under IRB Study No. A11-E91-13B - The apportionment of genetic diversity within the United States. UKBB data was accessed under accession number 6728.

\section{Acknowledgments}
We thank all participants in the HRS, UKBB, and 1KGP for providing their genetic data as well as the teams who generated and assembled the datasets. We also thank Audrey Grant, Ryan Hernandez, Jose Sergio Hleap, Mark Lathrop, Dominic Nelson, Markus Munter, Stephen Sawcer, Melissa Spear, and Dara Torgerson for useful discussions about science, programming, and data access; David Poznik, Liz Babalola, and Adam Auton from 23andMe for discussing findings in the 1KGP; and Selin Jessa for introducing us to UMAP.


\clearpage

\section{Supporting information}

\input{base/chapter1/chapter1_supp.tex}

\clearpage